> ![](media/image1.jpeg){width="0.8359284776902887in"
> height="0.913332239720035in"}![](media/image2.jpeg)[Clinical and
> Translational Radiation Oncology 1 (2016)
> 19--26](http://dx.doi.org/10.1016/j.ctro.2016.10.002)

## ![](media/image3.png)Independent validation of the prognostic value of cancer stem cell marker expression and hypoxia-induced gene expression for patients with locally advanced HNSCC after postoperative radiotherapy {#independent-validation-of-the-prognostic-value-of-cancer-stem-cell-marker-expression-and-hypoxia-induced-gene-expression-for-patients-with-locally-advanced-hnscc-after-postoperative-radiotherapy .unnumbered}

> Annett Linge
> ^[a](#_bookmark0),[b](#_bookmark1),[c](#_bookmark2),[d](#_bookmark3),[1](#_bookmark10),^[⇑](#_bookmark11),
> Steffen Löck
> ^[a](#_bookmark0),[b](#_bookmark1),[d](#_bookmark3),[1](#_bookmark10)^,
> Constanze Krenn [^b^](#_bookmark1), Steffen Appold
> ^[c](#_bookmark2),[d](#_bookmark3)^, Fabian Lohaus
> ^[a](#_bookmark0),[b](#_bookmark1),[c](#_bookmark2),[d](#_bookmark3)^,
> Alexander Nowak ^[d](#_bookmark3),[e](#_bookmark4)^, Volker Gudziol
> ^[d](#_bookmark3),[f](#_bookmark5)^, Gustavo B. Baretton
> ^[a](#_bookmark0),[d](#_bookmark3),[g](#_bookmark6),[h](#_bookmark7)^,
> Frank Buchholz ^[a](#_bookmark0),[i](#_bookmark8)^,
>
> Michael Baumann
> ^[a](#_bookmark0),[b](#_bookmark1),[c](#_bookmark2),[d](#_bookmark3),[j](#_bookmark9)^,
> Mechthild Krause
> ^[a](#_bookmark0),[b](#_bookmark1),[c](#_bookmark2),[d](#_bookmark3),[j](#_bookmark9)^
>
> [[]{#_bookmark1 .anchor}]{#_bookmark0 .anchor}^a^ *German Cancer
> Research Center (DKFZ), Heidelberg and German Cancer Consortium
> (DKTK), Partner Site Dresden, Germany*
>
> []{#_bookmark2 .anchor}^b^ *OncoRay -- National Center for Radiation
> Research in Oncology, Faculty of Medicine and University Hospital Carl
> Gustav Carus, Technische Universität Dresden, Germany*
>
> []{#_bookmark3 .anchor}^c^ *Department of Radiation Oncology, Faculty
> of Medicine and University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark4 .anchor}^d^ *National Center for Tumor Diseases (NCT),
> Dresden, Germany*
>
> []{#_bookmark5 .anchor}^e^ *Department of Oral and Maxillofacial
> Surgery, Faculty of Medicine and University Hospital Carl Gustav
> Carus, Technische Universität Dresden, Germany*
>
> []{#_bookmark6 .anchor}^f^ *Department of Otorhinolaryngology, Faculty
> of Medicine and University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark7 .anchor}^g^ *Institute of Pathology, Faculty of
> Medicine and University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark8 .anchor}^h^ *Tumour- and Normal Tissue Bank, University
> Cancer Center (UCC), University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark9 .anchor}^i^ *University Cancer Center (UCC), Medical
> Systems Biology, University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> ^j^ *Helmholtz-Zentrum Dresden -- Rossendorf, Institute of
> Radiooncology, Germany*
>
> a r t i c l e i n f o
>
> *Article history:*
>
> Received 21 September 2016
>
> []{#_bookmark10 .anchor}Revised 21 October 2016
>
> Accepted 24 October 2016
>
> Available online 22 December 2016
>
> *Keywords:*
>
> Biomarker Cancer stem cells HNSCC
>
> HPV
>
> Hypoxia
>
> Postoperative radiochemotherapy Validation
>
> a b s t r a c t
>
> *Objective:* To validate the impact of HPV status, cancer stem cell
> (CSC) marker expression and tumour hypoxia status in patients with
> locally advanced head and neck squamous cell carcinoma (HNSCC), who
> received postoperative radiotherapy. The results of the exploration
> cohort have previously been reported by the German Cancer Consortium
> Radiation Oncology Group (DKTK-ROG; Lohaus et al., 2014; Linge et al.,
> 2016).
>
> *Materials and methods:* For 152 patients with locally advanced HNSCC
> the impact of HPV16 DNA status, CSC marker expression and
> hypoxia-associated gene signatures on outcome of postoperative
> radiother- apy were retrospectively analysed. Out of them, 40 patients
> received postoperative radiochemotherapy. Cox models presented in a
> previous study were validated using the concordance index as a
> performance measure. The primary endpoint of this study was
> loco-regional control. Results were compared to those previously
> reported by DKTK-ROG.
>
> *Results:* Loco-regional control, freedom from distant metastases and
> overall survival were inferior to the previously reported cohort.
> Despite of this, the prognostic value of the combination of HPV
> infection sta- tus, CSC marker expression (*SLC3A2*) and tumour
> hypoxia status could be validated in univariate analyses using an
> independent validation cohort. For multivariate models, the
> concordance index was between
>
> 0.58 and 0.69 in validation, indicating a good prognostic performance
> of the models. The inclusion of *CD44* and the 15-gene hypoxia
> signature moderately improved the performance compared to a baseline
> model without CSC markers or hypoxia classifiers.
>
> *Conclusions:* The HPV status, CSC marker expression of *CD44* and
> *SLC3A2* as well as hypoxia status are potential prognostic biomarkers
> for patients with locally advanced HNSCC treated by postoperative
> radiotherapy.
>
> © 2016 The Authors. Published by Elsevier Ireland Ltd on behalf of
> European Society for Radiotherapy and Oncology. This is an open access
> article under the CC BY-NC-ND license
> ([http://creativecommons.org/](http://creativecommons.org/licenses/by-nc-nd/4.0/)

[licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/)).

> []{#_bookmark11 .anchor}⇑ Corresponding author at: Department of
> Radiation Oncology and OncoRay - National Center for Radiation
> Research in Oncology, Faculty of Medicine and University Hospital Carl
> Gustav Carus, Technische Universität Dresden, Fetscherstraße 74, 01307
> Dresden, Germany.
>
> *E-mail address:* <Annett.Linge@uniklinikum-dresden.de> (A. Linge).
>
> ^1^ Both authors contributed equally to this study.
>
> Introduction
>
> The 5-year overall survival rate of patients with head and neck
> squamous cell carcinoma (HNSCC) is about 50%, although treatment
> efficacy has been therapeutically improved in the last
>
> <http://dx.doi.org/10.1016/j.ctro.2016.10.002>
>
> 2405-6308/© 2016 The Authors. Published by Elsevier Ireland Ltd on
> behalf of European Society for Radiotherapy and Oncology. This is an
> open access article under the CC BY-NC-ND license
> (<http://creativecommons.org/licenses/by-nc-nd/4.0/>).
>
> decade [\[1--3\]](#_bookmark20). Furthermore, the number of
> oropharyngeal cancers has increased worldwide over the last years
> [\[4--6\]](#_bookmark20). For patients with resected locally advanced
> HNSCC it has been shown in three randomized clinical trials
> [\[7--9\]](#_bookmark21) that loco-regional control can be improved by
> postoperative radiochemotherapy (PORT-C) com- pared to postoperative
> radiotherapy (PORT) alone. However, due to their biological
> characteristics tumours are responding very heterogeneously to this
> treatment. In addition to established clin- ical parameters, novel
> biomarkers are needed to identify patient groups, which require
> escalated or de-escalated treatment schedules.
>
> Independent of exogenous carcinogens, e.g. tobacco and alcohol
> consumption, infection with high-risk human papilloma virus (HPV) has
> become a major risk factor for the development of HNSCC over the past
> decade [\[10\]](#_bookmark27). It has been shown that patients with
> HPV-related HNSCC are a distinct subgroup with better out- come of
> primary radiochemotherapy [\[11--14\]](#_bookmark29). Recently, in a
> multi- centre retrospective trial conducted by the German Cancer
> Consortium Radiation Oncology Group (DKTK-ROG) it was shown, that
> patients with HPV16 DNA positive oropharyngeal tumours have a high
> rate of loco-regional tumour control and overall sur- vival also after
> PORT-C compared to patients with HPV16 DNA neg- ative tumours
> [\[15\]](#_bookmark22). Furthermore, in a recent study it was shown
> that high expression of previously developed tumour hypoxia-
> associated gene signatures [\[16,17\]](#_bookmark22) and cancer stem
> cell (CSC) markers within the surgically removed primary tumour were
> asso- ciated with impaired loco-regional tumour control after PORT-C
> in HNSCC [\[18\]](#_bookmark22). These biomarkers were used to
> stratify patients with HPV16 DNA negative tumours into smaller
> subgroups with low and high probability of loco-regional recurrence.
> In order to apply this patient stratification scheme in future
> clinical trials regarding treatment de-escalation or intensification,
> these results have to be validated. Therefore, the present publication
> reports on the valida- tion results of the Cox models presented in
> [\[18\]](#_bookmark22) on HPV16 DNA sta- tus, CSC marker expression
> and tumour hypoxia-associated gene signatures, using an independent
> validation cohort of HNSCC patients treated by PORT or PORT-C.
>
> Material and methods
>
> *Patients*
>
> In this publication, two patient cohorts are being considered: a
> training cohort previously reported by DKTK-ROG and a monocen- tric
> validation cohort. For the training cohort, inclusion criteria, data
> collection, handling and analyses of biomaterial were previ- ously
> described in detail [\[15,18\]](#_bookmark22). Briefly, 221 patients,
> who received PORT-C between 2004 and 2012 at nine partner sites of the
> DKTK-ROG, were included to identify potential prognosticators for
> loco-regional control after PORT-C, such as CSCs and tumour hypoxia
> [\[18\]](#_bookmark22). In the retrospective validation cohort,
> patients trea- ted at the Department of Radiation Oncology of the
> University Hospital Dresden meeting the following criteria were
> included: not included in the previous DKTK-ROG cohort, histologically
> pro- ven squamous cell carcinoma arising from the oral cavity,
> orophar- ynx, hypopharynx or larynx; treatment between 1999 and 2006
> with PORT or PORT-C in curative intention according to standard
> radiotherapy protocols (2 Gy/fraction, 5 fractions/week), covering the
> former tumour region and regional lymph nodes (50 Gy) and a boost
> (10--16 Gy) to the former tumour region and to the regions of involved
> lymph nodes. Before surgery, all patients had under- gone staging
> (computed tomography or magnetic resonance imag- ing, chest X-ray and
> abdominal ultrasound), and only patients without evidence of distant
> metastases were included. Formalin- fixed paraffin-embedded (FFPE)
> tumour material as well as
>
> follow-up data of patients had to be available. Finally, 152 patients
> meeting all these criteria were included in the validation cohort.
> Follow-up data of patients were collected using the RadPlanBio
> Platform at the DKTK partner site Dresden [\[19\]](#_bookmark23).
> Ethical approval for retrospective analyses of clinical and biological
> data was obtained from the local ethics committee.
>
> *Preparation of biomaterials for biomarker analyses*
>
> FFPE blocks of the primary tumour specimens (removed by sur- gery)
> were first subjected to haematoxylin and eosin staining to
> histologically confirm the presence of squamous cell carcinoma.
> Afterwards, they were processed under standardised procedures for
> biomarker investigations described below.
>
> *Immunohistochemical staining of p16 and CD44*
>
> Immunohistochemical (IHC) staining of p16 was performed using the
> CINtec Histology kit (Roche mtm laboratories AG, Basel, CH) according
> to the manufacturer's instructions. Overexpression of p16 was defined
> as P70% intense tumour staining. For p16 staining, 148 samples were
> evaluable. For CD44 protein expression, 145 patients of the validation
> cohort could be analysed. Immuno- histochemical staining was performed
> as described in [\[18\]](#_bookmark22). CD44 staining intensity was
> scored (0, +, ++, +++) and tumours with a minimum of 10% staining were
> considered as positive. Blinded samples were scored by two independent
> observers (AL and CK) with an inter-observer variability of \<5%.
>
> *DNA extraction and PCR-array based analyses of HPV status*
>
> DNA extraction and PCR-array based analyses of HPV status were
> performed as described previously [\[15\]](#_bookmark22). Briefly,
> genomic DNA was extracted from 5-lm FFPE sections using the QIAamp DNA
> FFPE tissue kit (Qiagen). HPV DNA analyses including geno-
>
> typing were performed using the LCD-Array HPV 3.5 kit (CHIPRON GmbH,
> Berlin, DE) according to the instructions of the manufac- turer. In
> total, 148 samples were evaluable for HPV DNA analyses.
>
> *nanoString RNA analyses*
>
> For the validation cohort, gene expression analyses have been
> performed using nanoString technology as described previously
> [\[18\]](#_bookmark22), including two hypoxia-associated gene
> signatures (see Sup- plementary Table 1) as well as potential CSC
> markers *CD44*, *SLC3A2* and *MET*. Briefly, total RNA as well as
> reporter and capture probes specific to the genes of interest were
> mixed and incubated at 62 °C for 22 h. Samples were then kept at 4 °C
> for a maximum of 18 h and subjected to the nCounter system. Raw counts
> were logarithmised and then normalised to the mean of the internal
> level of reference genes *ACTR3*, *B2M*, *GNB2L1*, *NDFIP1*, *POLR2A*,
> *RPL11, RPL37A*, or to
>
> the reference genes of the corresponding hypoxia-gene signatures
> (Supplementary Table 1), respectively [\[16,17\]](#_bookmark22). Seven
> samples had to be omitted from nanoString analyses due to insufficient
> tumour material or due to too low RNA yield, thus 145 samples were
> evalu- able. The *CD44* probe design, which was incorrectly designed
> for the training cohort [\[18\]](#_bookmark22), has been corrected and
> included in the validation cohort.
>
> *Clinical endpoints and statistical analysis*
>
> The primary endpoint was loco-regional tumour control (LRC) and
> secondary endpoints were freedom from distant metastases (DM) and
> overall survival (OS). The endpoints were calculated from the first
> day of radiotherapy to the date of event or censoring and survival
> curves were estimated by the Kaplan-Meier method. To
>
> compare patient groups stratified by HPV16 DNA status, CSC marker
> expression and hypoxia classification, Log-rank tests were used. For
> the stratification of the validation cohort with respect to CSC marker
> expression, the cut-offs from the training cohort reported in
> [\[18\]](#_bookmark22) were applied. Hypoxia classification on the
> valida- tion cohort was performed by k-means clustering (Euclidian
> distance) according to the cluster centres of the training cohort
> (Supplementary Table 2). The impact of potential prognostic variables
> on the endpoints was evaluated using univariate Cox-regression for
> both cohorts. To evaluate the prognostic perfor- mance of the
> multivariate Cox models presented in [\[18\]](#_bookmark22) for the
> validation cohort, the concordance index (ci) was calculated, which is
> equal to 0.5 for non-prognostic models and equals 1 for perfectly
> predicting models [\[20\]](#_bookmark24). Bootstrap resampling with
> 10,000 samples was used to estimate the confidence intervals (CI) of
> the concor- dance index. From these bootstrap samples a *p*-value for
> the hypothesis ci = 0.5 was calculated. To evaluate differences
> between the two cohorts Mann-Whitney-*U* tests were used for
> continuous variables and chi-squared tests for categorical variables.
> The boot- strapping procedure and ci calculation were implemented in
> Python. For all of the other analyses, SPSS 23 software (IBM Corpo-
> ration, Armonk, NY, USA) was used. In this study, two-sided tests were
> performed and *p*-values \< 0.05 were considered statistically
> significant.
>
> Results
>
> In this retrospective study, the prognostic value of HPV16 DNA status,
> CSC markers and hypoxia classifiers presented in
>
> [\[18\]](#_bookmark22) for 221 patients (training cohort) should be
> validated on an independent validation cohort of 152 patients. The
> patient data, treatment parameters and tumour characteristics of the
> training cohort have been described previously
> [\[15,18\]](#_bookmark22). They are summarised in [Table
> 1](#_bookmark12) together with the values of the vali- dation cohort
> and the comparison between both cohorts. Patients in the validation
> cohort were treated between 1999 and 2006 with PORT (*N* = 112) or
> PORT-C (*N* = 40), while all patients of the training cohort received
> PORT-C as the standard treatment (*p* \< 0.001). Furthermore, patients
> in the retrospective validation cohort had a significantly lower age
> (*p* = 0.016), deliv- ered dose (*p* = 0.001), N stage (*p* \< 0.001),
> R status (*p* \< 0.001), extracapsular extension (ECE) status (*p* \<
> 0.001) and also showed a different distribution in their tumour
> localisations (*p* \< 0.001). The fraction of oropharyngeal tumours
> and tumours of the oral cavity were almost reverse in training and
> validation cohort (oropharynx: 57% vs 30%, oral cavity: 27% vs 55%).
> In the valida- tion cohort HPV infection occurred less frequently than
> in the (more recently treated) training cohort (HPV16 DNA positivity:
> 14% vs. 33%, *p* \< 0.001). For the considered endpoints, patients in
> the validation cohort showed lower LRC and OS, while the occurrence of
> DM was not significantly different, see [Fig. 1](#_bookmark13). Actu-
> arial rates of LRC, freedom from DM and OS for the training and
> validation cohort were 90% vs 78% (*p* = 0.002), 85% vs 84%
>
> (*p* = 0.21) and 83% vs 68% (*p* \< 0.001), after two years,
>
> respectively.
>
> [Table 2](#_bookmark14) shows the results of univariate Cox regression
> with clin- ical parameters and HPV status for the considered endpoints
> of the validation cohort. Significantly lower LRC and OS was found for
> higher T stage (LRC: HR 2.46, *p* = 0.008; OS: HR 2.14, *p* = 0.001)
>
> and tumours of the oral cavity (LRC: HR 2.68, *p* = 0.005; OS: HR
> 2.22, *p* \< 0.001), while oropharyngeal tumours (LRC: HR 0.38, *p* =
> 0.020; OS: HR 0.40, *p* = 0.001) and HPV16 DNA positive tumours (LRC:
> HR 0.25, *p* = 0.060; OS: HR 0.35, *p* = 0.015) showed
>
> improved LRC and OS. These results are consistent with the find- ings
> of the training cohort [\[18\]](#_bookmark22). [Fig. 2](#_bookmark16)
> A-B shows Kaplan-Meier
>
> curves of the validation cohort stratified for HVP16 DNA status and
> use of chemotherapy. Patients with HPV16 DNA positive tumours showed
> significantly higher loco-regional control rates compared to patients
> with HPV16 DNA negative tumours using the Log-rank test; a statistical
> trend was obtained by Cox regres- sion. The higher loco-regional
> control rates in the small patient group receiving PORT-C did not
> reach statistical significance.
>
> In [\[18\]](#_bookmark22) a significant impact of the cancer stem-cell
> markers *CD44*, *SLC3A2* and *MET* on LRC was found using their
> expression values dichotomised at the cut-offs 0.2, -3.135 and -4.135,
> respectively. These cut-offs were used for patient stratification of
> the validation cohort. The resulting Kaplan-Meier curves are depicted
> in [Fig. 2](#_bookmark16)D--F. In univariate analyses, the impact of
> *CD44* (HR 2.14, *p* = 0.049) and *SLC3A2* (HR 2.45, *p* = 0.047) on
> LRC could
>
> be confirmed with lower hazard ratios, while *MET* expression showed
> no correlation with LRC ([Table 3](#_bookmark17)). Also the
> significant impact of CD44 protein on LRC could be confirmed (*p* =
> 0.027, [Fig. 2](#_bookmark16)C). For the secondary endpoints only
> *CD44* showed a signifi- cant correlation with freedom of DM. The
> prognostic value of *MET* and *SLC3A2* for OS and DM could not be
> confirmed.
>
> For hypoxia-related gene classification based on the 15- and 26-gene
> signature [\[16,17\]](#_bookmark22), k-means clustering was used in
> [\[18\]](#_bookmark22). Both classifiers were able to stratify
> patients into a group with high hypoxia-related gene expression and
> low LRC and a group with low hypoxia-related gene expression and
> better LRC. For the validation cohort, the hypoxia classification was
> performed using the cluster centres of the training cohort. Based on
> this classification, [Fig. 2](#_bookmark16)G and [2](#_bookmark16)H
> present Kaplan-Meier curves of tumours with low and high
> hypoxia-related gene expression for the validation cohort. A
> statistical trend for differences in LRC between both groups was
> obtained for both signatures (15-gene signature: HR 2.32, *p* = 0.061;
> 26-gene signature: HR 3.36, *p* = 0.097; [Table 3](#_bookmark17)).
> Considering only patients with HPV16 DNA negative tumours, this trend
> remained for the 15-gene sig- nature (HR 2.13, *p* = 0.093) but not
> for the 26-gene signature. In
>
> [\[18\]](#_bookmark22) low-risk patients (HPV16 DNA positive or
> *SLC3A2* negative or low hypoxia) showed favourable LRC compared to
> the remaining high-risk patients. Here, this classification also
> yields a significant difference in LRC (*p* = 0.010, [Fig.
> 2](#_bookmark16)I).
>
> In [\[18\]](#_bookmark22) several multivariate Cox models were
> presented, which included HPV16 DNA status, one CSC marker and one
> hypoxia-associated gene classifier as well as the clinical param-
> eters ECE status, tumour localisation oropharynx and hypophar- ynx.
> For the primary endpoint LRC, these models were applied to the
> validation cohort without adapting the model parameters. The
> prognostic performance of these models was then evaluated by the
> concordance index (ci). To reveal an additional impact of the CSC
> markers or hypoxia classifiers, the model performance was compared to
> a baseline model, which did not contain any CSC marker or hypoxia
> classifier. In [Table 4](#_bookmark19) the results of the training
> cohort and the validation cohort are presented for all patients and
> for patients with HPV16 DNA negative tumours. For the training cohort,
> the baseline model showed a ci of 0.76 for all patients and of 0.66
> for patients with HPV16 DNA nega- tive tumours. Using additional CSC
> markers and hypoxia classi- fiers improved the ci up to 0.81 and 0.76,
> respectively, revealing their significant impact. For the validation
> cohort the baseline model showed ci = 0.66 for all patients and ci =
> 0.63 for patients with HVP16 DNA negative tumours. Including addi-
> tional CSC markers or hypoxia classifiers improved these models only
> for *CD44* and the 15-gene hypoxia classifier up to ci = 0.69 and ci =
> 0.65, respectively. Validation on the patient subgroup which received
> PORT-C led to higher ci values for all CSC mark- ers and hypoxia
> classifiers, however, due to the low patient number with large
> confidence intervals.
>
> []{#_bookmark12 .anchor}Table 1
>
> Comparison of the patient and tumour characteristics of the training
> and validation cohort.

+---------------------+----------+---------+----------+--------+-----+
| > Characteristics   | >        |         | > Va     |        | >   |
|                     | Training |         | lidation |        | *p* |
|                     | > cohort |         | > cohort |        | -Va |
|                     | > (200   |         | > (199   |        | lue |
|                     | 4--2012) |         | 9--2006) |        |     |
+=====================+==========+=========+==========+========+=====+
|                     | > Median |         | > Median |        |     |
|                     | >        |         | >        |        |     |
|                     |  (range) |         |  (range) |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Follow-up         | > 46.2   |         | > 43.1   |        | > 0 |
| > (months)          | > (2.5   |         | > (1     |        | .29 |
|                     | --100.1) |         | .8--153) |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Age (years)       | > 57.0   |         | > 52.7   |        | >   |
|                     | > (24.   |         | > (36.   |        |  0. |
|                     | 0--75.2) |         | 3--71.0) |        | 016 |
+---------------------+----------+---------+----------+--------+-----+
| > Dose (Gy)         | > 64     |         | > 64     |        | >   |
|                     | >        |         | >        |        |  0. |
|                     | (56--68) |         | (60--66) |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
|                     | > Number | > (%)   | > Number | > (%)  |     |
|                     | > of pts |         | > of pts |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Gender            |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Male/Female       | > 180/41 | >       | > 127/25 | > (    | > 0 |
|                     |          | (81/19) |          | 84/16) | .60 |
+---------------------+----------+---------+----------+--------+-----+
| > Clinical tumour   | > 41/    | >       | > 40/    | > (    | > 0 |
| > (T) stage         | 98/47/35 |  (19/44 | 69/24/19 | 26/45/ | .20 |
| >                   |          | /21/16) |          | 16/13) |     |
| > cT1/cT2/cT3/cT4   |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Clinical nodal    |          |         |          |        |     |
| > (N) stage         |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > cN0/cN1/cN2/cN3   | > 22/3   | >       | > 15/    | >      | > \ |
|                     | 1/138/30 |  (10/14 | 31/104/2 | (10/20 | <0. |
|                     |          | /62/14) |          | /68/1) | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > Stage (UICC 7th   |          |         |          |        |     |
| > edition)          |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > I/II/III/IV       | > 0/     | > (0/4  | > 2/     | >      | >   |
|                     | 8/33/180 | /15/81) | 3/34/113 |  (1/2/ |  0. |
|                     |          |         |          | 23/74) | 069 |
+---------------------+----------+---------+----------+--------+-----+
| > R status          |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 0/1/unknown       | >        | > (5    | > 1      | > (72  | > \ |
|                     | 125/94/2 | 7/42/1) | 09/30/13 | /20/8) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > ECE status        |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 0/1/unknown       | > 1      | > (4    | >        | > (68  | > \ |
|                     | 03/118/0 | 7/53/0) | 104/47/1 | /31/1) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > Localisation      |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > Oropharynx/oral   | > 126    | > (57/2 | > 46     | >      | > \ |
| > cavity            | /60/35/0 | 7/16/0) | /83/15/8 | (30/55 | <0. |
| /hypopharynx/larynx |          |         |          | /10/5) | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > Grading           |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 1/2/3/unknown     | > 5/     | > (2/5  | > 3      | >      | > 0 |
|                     | 123/89/4 | 6/40/2) | /78/71/0 |  (2/51 | .55 |
|                     |          |         |          | /47/0) |     |
+---------------------+----------+---------+----------+--------+-----+
| > Chemotherapy      |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > yes/no            | > 221/0  | >       | > 40/112 | > (    | > \ |
|                     |          | (100/0) |          | 26/74) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > Smoking during    |          |         |          |        |     |
| > therapy           |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > yes/no/unknown    | > 1      | > (     | > 1      | > (68/ | > 0 |
|                     | 86/20/15 | 84/9/7) | 03/16/33 | 11/22) | .30 |
+---------------------+----------+---------+----------+--------+-----+
| > Alcohol during    |          |         |          |        |     |
| > therapy           |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > yes/no/unknown    | > 1      | > (45   | >        | > (65/ | > 0 |
|                     | 00/30/91 | /14/41) | 99/19/34 | 13/22) | .17 |
+---------------------+----------+---------+----------+--------+-----+
| > p16 status        |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > negati            | >        | > (6    | >        | > (84  | > \ |
| ve/positive/unknown | 135/79/7 | 1/36/3) | 128/20/4 | /13/3) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > HPV16 DNA status  |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > negati            | >        | > (6    | >        | > (83  | > \ |
| ve/positive/unknown | 143/72/6 | 5/33/2) | 126/22/4 | /14/3) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > CD44 protein      |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > negati            | > 4      | > (20   | >        | > (10  | >   |
| ve/positive/unknown | 4/151/26 | /68/12) | 15/130/7 | /85/5) |  0. |
|                     |          |         |          |        | 003 |
+---------------------+----------+---------+----------+--------+-----+
| > *CD44*            |          |         |          |        |     |
| > (log2-normalised  |          |         |          |        |     |
| > expression)       |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 60.2 / \> 0.2 /   | > 7      | > (35   | >        | > (36  | > 0 |
| > unknown           | 8/118/25 | /54/11) |  55/90/7 | /59/5) | .73 |
+---------------------+----------+---------+----------+--------+-----+
| > []{#_bookmark13   |          |         |          |        |     |
| > .anchor}*SLC3A2*  |          |         |          |        |     |
| > (log2-normalised  |          |         |          |        |     |
| > expression)       |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 6-3.135 /\>-3.135 | > 7      | > (35   | >        | > (28  | >   |
| > / unknown         | 7/119/25 | /54/11) | 43/102/7 | /67/5) |  0. |
|                     |          |         |          |        | 066 |
+---------------------+----------+---------+----------+--------+-----+
| > *MET*             |          |         |          |        |     |
| > (log2-normalised  |          |         |          |        |     |
| > expression)       |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > 6-4.135 /\>-4.135 | > 9      | > (43   | >        | > (31  | > \ |
| > / unknown         | 4/102/25 | /46/11) |  47/98/7 | /64/5) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > 15-gene hypoxia   |          |         |          |        |     |
| > signature         |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > low               | > 7      | > (36   | >        | > (27  | >   |
| > hypoxic/highly    | 9/117/25 | /53/11) | 41/104/7 | /68/5) |  0. |
| > hypoxic/unknown   |          |         |          |        | 021 |
+---------------------+----------+---------+----------+--------+-----+
| > 26-gene hypoxia   |          |         |          |        |     |
| > signature         |          |         |          |        |     |
+---------------------+----------+---------+----------+--------+-----+
| > low               | > 7      | > (33   | >        | > (14  | > \ |
| > hypoxic/highly    | 2/124/25 | /56/11) | 22/123/7 | /81/5) | <0. |
| > hypoxic/unknown   |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+
| > Loco-regional     | > 29     | >       | > 38     | >      | >   |
| > recurrences       |          |  \(13\) |          | \(25\) |  0. |
|                     |          |         |          |        | 003 |
+---------------------+----------+---------+----------+--------+-----+
| > Distant           | > 42     | >       | > 24     | >      | > 0 |
| > metastases        |          |  \(19\) |          | \(22\) | .43 |
+---------------------+----------+---------+----------+--------+-----+
| > Deaths            | > 70     | >       | > 86     | >      | > \ |
|                     |          |  \(32\) |          | \(57\) | <0. |
|                     |          |         |          |        | 001 |
+---------------------+----------+---------+----------+--------+-----+

> Bold numbers indicate significant *p*-Values with *p* \< 0.05.

![](media/image5.png)**100**

**80**

**60**

**100**

**80**

**60**

B

> ![](media/image7.png)Training cohort
>
> Validation cohort
>
> **100**
>
> ![](media/image9.png)**80**
>
> **60**
>
> **Pts. at risk**
>
> **0**
>
> ![](media/image10.png)**0 12 24 36 48 60**
>
> Months after start of treatment
>
> **Pts. at risk**
>
> **0**
>
> ![](media/image12.png)**0 12 24 36 48 60**
>
> Months after start of treatment
>
> **Pts. at risk**
>
> **0**
>
> ![](media/image14.png)**0 12 24 36 48 60**
>
> Months after start of treatment

<table>
<colgroup>
<col style="width: 14%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 11%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 11%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th><blockquote>
<p>Training 221 191</p>
</blockquote></th>
<th><blockquote>
<p>173</p>
</blockquote></th>
<th><blockquote>
<p>148</p>
</blockquote></th>
<th><blockquote>
<p>107</p>
</blockquote></th>
<th><blockquote>
<p>63</p>
</blockquote></th>
<th><blockquote>
<p>Training 221</p>
</blockquote></th>
<th><blockquote>
<p>187</p>
</blockquote></th>
<th><blockquote>
<p>167</p>
</blockquote></th>
<th><blockquote>
<p>144</p>
</blockquote></th>
<th><blockquote>
<p>103</p>
</blockquote></th>
<th><blockquote>
<p>61</p>
</blockquote></th>
<th>Training 221</th>
<th><blockquote>
<p>198</p>
</blockquote></th>
<th><blockquote>
<p>178</p>
</blockquote></th>
<th><blockquote>
<p>153</p>
</blockquote></th>
<th><blockquote>
<p>111 64</p>
</blockquote></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><blockquote>
<p>Validation 152 109</p>
</blockquote></td>
<td><blockquote>
<p>92</p>
</blockquote></td>
<td><blockquote>
<p>76</p>
</blockquote></td>
<td><blockquote>
<p>64</p>
</blockquote></td>
<td><blockquote>
<p>44</p>
</blockquote></td>
<td><blockquote>
<p>Validation 152</p>
</blockquote></td>
<td><blockquote>
<p>110</p>
</blockquote></td>
<td><blockquote>
<p>95</p>
</blockquote></td>
<td><blockquote>
<p>77</p>
</blockquote></td>
<td><blockquote>
<p>64</p>
</blockquote></td>
<td><blockquote>
<p>45</p>
</blockquote></td>
<td>Validation 152</td>
<td><blockquote>
<p>120</p>
</blockquote></td>
<td><blockquote>
<p>102</p>
</blockquote></td>
<td><blockquote>
<p>88</p>
</blockquote></td>
<td><blockquote>
<p>70 50</p>
</blockquote></td>
</tr>
</tbody>
</table>

> Fig. 1. (A--C) Kaplan-Meier estimates of (A) loco-regional control
> (LRC), (B) freedom of distant metastases (DM) and (C) overall survival
> (OS) for the training and validation cohort.
>
> [[]{#_bookmark15 .anchor}]{#_bookmark14 .anchor}Table 2
>
> Univariate Cox regression with clinical variables as well as HPV
> status for the endpoints loco-regional control, freedom of distant
> metastases and overall survival for the validation cohort. Shown is
> the hazard ratio (HR) with 95% confidence interval (CI) and the
> *p*-value testing the hypothesis HR = 1.

<table style="width:100%;">
<colgroup>
<col style="width: 21%" />
<col style="width: 14%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 13%" />
<col style="width: 7%" />
<col style="width: 6%" />
<col style="width: 13%" />
<col style="width: 7%" />
<col style="width: 1%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>Loco-regional control</th>
<th></th>
<th></th>
<th>Distant metastases</th>
<th></th>
<th></th>
<th>Overall survival</th>
<th></th>
<th rowspan="2"></th>
</tr>
<tr class="odd">
<th><blockquote>
<p>Variable</p>
</blockquote></th>
<th>HR (95% CI)</th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
<th></th>
<th>HR (95% CI)</th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
<th></th>
<th>HR (95% CI)</th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><blockquote>
<p>Age (years)</p>
</blockquote></td>
<td>0.97 (0.94–1.01)</td>
<td><blockquote>
<p>0.16<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.99 (0.95–1.03)</td>
<td><blockquote>
<p>0.56</p>
</blockquote></td>
<td></td>
<td>1.01 (0.99–1.04)</td>
<td><blockquote>
<p>0.31</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Dose (Gy)</p>
</blockquote></td>
<td>1.07 (0.94–1.24)</td>
<td><blockquote>
<p>0.31</p>
</blockquote></td>
<td></td>
<td>1.21 (1.02–1.42)</td>
<td><blockquote>
<p>0.022</p>
</blockquote></td>
<td></td>
<td>1.07 (0.98–1.18)</td>
<td><blockquote>
<p>0.15</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Gender</p>
</blockquote></td>
<td>0.36 (0.11–1.16)</td>
<td><blockquote>
<p>0.087</p>
</blockquote></td>
<td></td>
<td>0.04 (0.00–1.33)</td>
<td><blockquote>
<p>0.071</p>
</blockquote></td>
<td></td>
<td>0.59 (0.31–1.15)</td>
<td><blockquote>
<p>0.12</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>T stage (1,2 vs 3,4)</p>
</blockquote></td>
<td>2.46 (1.26–4.79)</td>
<td><blockquote>
<p>0.008<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>2.50 (1.24–5.05)</td>
<td><blockquote>
<p>0.010<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>2.14 (1.35–3.38)</td>
<td><blockquote>
<p>0.001<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>N stage (0,1 vs 2,3)</p>
</blockquote></td>
<td>0.87 (0.44–1.74)</td>
<td><blockquote>
<p>0.69</p>
</blockquote></td>
<td></td>
<td>0.99 (0.47–2.09)</td>
<td><blockquote>
<p>0.98<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.85 (0.54–1.34)</td>
<td><blockquote>
<p>0.48</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>UICC stage (I–III vs IV)</p>
</blockquote></td>
<td>1.13 (0.53–2.40)</td>
<td><blockquote>
<p>0.75</p>
</blockquote></td>
<td></td>
<td>1.03 (0.48–2.21)</td>
<td><blockquote>
<p>0.95</p>
</blockquote></td>
<td></td>
<td>0.99 (0.62–1.60)</td>
<td><blockquote>
<p>0.97</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>R status</p>
</blockquote></td>
<td>0.86 (0.39–1.89)</td>
<td><blockquote>
<p>0.71</p>
</blockquote></td>
<td></td>
<td>1.38 (0.64–3.00)</td>
<td><blockquote>
<p>0.41</p>
</blockquote></td>
<td></td>
<td>1.29 (0.78–2.11)</td>
<td><blockquote>
<p>0.32</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>ECE status</p>
</blockquote></td>
<td>1.60 (0.82–3.11)</td>
<td><blockquote>
<p>0.17</p>
</blockquote></td>
<td></td>
<td>1.75 (0.87–3.49)</td>
<td><blockquote>
<p>0.11<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>1.41 (0.90–2.21)</td>
<td><blockquote>
<p>0.13<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Oropharynx</p>
</blockquote></td>
<td>0.38 (0.17–0.86)</td>
<td><blockquote>
<p>0.020<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.41 (0.18–0.94)</td>
<td><blockquote>
<p>0.034<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.40 (0.23–0.68)</td>
<td><blockquote>
<p>0.001<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Oral cavity</p>
</blockquote></td>
<td>2.68 (1.35–5.33)</td>
<td><blockquote>
<p>0.005<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>1.55 (0.78–3.08)</td>
<td><blockquote>
<p>0.21<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>2.22 (1.43–3.46)</td>
<td><blockquote>
<p>&lt;0.001<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Hypopharynx</p>
</blockquote></td>
<td>0.91 (0.32–2.60)</td>
<td><blockquote>
<p>0.86</p>
</blockquote></td>
<td></td>
<td>1.87 (0.77–4.53)</td>
<td><blockquote>
<p>0.17</p>
</blockquote></td>
<td></td>
<td>0.89 (0.43–1.84)</td>
<td><blockquote>
<p>0.74</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Larynx</p>
</blockquote></td>
<td>0.40 (0.05–2.89)</td>
<td><blockquote>
<p>0.36</p>
</blockquote></td>
<td></td>
<td>1.05 (0.31–3.59)</td>
<td><blockquote>
<p>0.94</p>
</blockquote></td>
<td></td>
<td>1.02 (0.44–2.35)</td>
<td><blockquote>
<p>0.96</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Grading (1,2 vs 3)</p>
</blockquote></td>
<td>0.97 (0.50–1.86)</td>
<td><blockquote>
<p>0.92</p>
</blockquote></td>
<td></td>
<td>1.81 (0.91–3.61)</td>
<td><blockquote>
<p>0.094</p>
</blockquote></td>
<td></td>
<td>1.12 (0.74–1.72)</td>
<td><blockquote>
<p>0.59</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Chemotherapy</p>
</blockquote></td>
<td>0.55 (0.23–1.32)</td>
<td><blockquote>
<p>0.18</p>
</blockquote></td>
<td></td>
<td>1.42 (0.69–2.93)</td>
<td><blockquote>
<p>0.34</p>
</blockquote></td>
<td></td>
<td>0.77 (0.45–1.31)</td>
<td><blockquote>
<p>0.33</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Smoking</p>
</blockquote></td>
<td>1.19 (0.36–3.98)</td>
<td><blockquote>
<p>0.78</p>
</blockquote></td>
<td></td>
<td>2.20 (0.52–9.30)</td>
<td><blockquote>
<p>0.28</p>
</blockquote></td>
<td></td>
<td>1.22 (0.55–2.68)</td>
<td><blockquote>
<p>0.63</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Alcohol</p>
</blockquote></td>
<td>0.87 (0.33–2.31)</td>
<td><blockquote>
<p>0.78</p>
</blockquote></td>
<td></td>
<td>2.83 (0.67–12.0)</td>
<td><blockquote>
<p>0.16</p>
</blockquote></td>
<td></td>
<td>1.26 (0.62–2.55)</td>
<td><blockquote>
<p>0.53</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>p16</p>
</blockquote></td>
<td>0.46 (0.14–1.49)</td>
<td><blockquote>
<p>0.19<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.46 (0.14–1.50)</td>
<td><blockquote>
<p>0.20<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.40 (0.17–0.92)</td>
<td><blockquote>
<p>0.031<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>HPV16 DNA</p>
</blockquote></td>
<td>0.25 (0.06–1.06)</td>
<td><blockquote>
<p>0.060<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.40 (0.12–1.33)</td>
<td><blockquote>
<p>0.13<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
<td>0.35 (0.15–0.82)</td>
<td><blockquote>
<p>0.015<a href="#_bookmark15"><sup>*</sup></a></p>
</blockquote></td>
<td></td>
</tr>
</tbody>
</table>

> Bold numbers indicate significant *p*-Values with *p* \< 0.05.
>
> ^\*^ *p* \< 0.05 in published training cohort.

![](media/image16.png)**100**

**100**

> **100**
>
> **80 80 80**
>
> **60 60 60**
>
> ![](media/image18.png)![](media/image19.png)![](media/image21.png)![](media/image24.png)**0
> 0**
>
> []{#_bookmark16 .anchor}**0 12 24 36 48 60 0 12 24 36 48 60**
>
> **0**
>
> ![](media/image26.png)**0 12 24 36 48 60**
>
> **Pts. at risk** HPV16 DNA - HPV16 DNA +
>
> 126
>
> 22
>
> Months after start of treatment
>
> **Pts. at risk**
>
> Months after start of treatment
>
> **Pts. at risk**
>
> Months after start of treatment

![](media/image27.png)**100**

**80**

**100**

![](media/image36.png)**80**

> **100**
>
> ![](media/image37.png)**80**
>
> **60 60 60**
>
> ![](media/image38.png)![](media/image41.png)**0 0**
>
> **60 0 12 24 36 48 60**
>
> **.** Months after start of treatment **.**
>
> **0**
>
> ![](media/image42.png)**0 12 24 36 48 60**
>
> Months after start of treatment
>
> 18 *SLC3A2*≤-3.135 43 35
>
> ![](media/image43.png)24 *SLC3A2*\>-3.135 102 69
>
> 28 20 16
>
> 61 53 45
>
> 11 *MET*≤-4.135 47 36 31
>
> 31 *MET*\>-4.135 98 68 58
>
> 24 20 12
>
> 49 41 30

![](media/image45.png)**100**

> **100 100**

I

> ![](media/image46.png){width="1.6355413385826771in"
> height="0.8804582239720035in"}HPV16 DNA + or *SLC3A2 -*
>
> or Hypoxia 15 low
>
> **80 80 80**
>
> **60 60**
>
> **60** HPV16 DNA - and *SLC3A2 +*
>
> and Hypoxia 15 high
>
> ![](media/image47.png)**0**
>
> **0 12 24 36 48 60**
>
> **0**
>
> ![](media/image48.png)**0 12 24 36 48 60**
>
> **0**
>
> ![](media/image49.png)**0 12 24 36 48 60**
>
> **Pts. at risk** Months after start of treatment
>
> **Pts. at risk**
>
> Months after start of treatment
>
> **Pts. at risk**
>
> Months after start of treatment

<table>
<colgroup>
<col style="width: 16%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 19%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 3%" />
<col style="width: 6%" />
<col style="width: 9%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 4%" />
<col style="width: 3%" />
</colgroup>
<thead>
<tr class="header">
<th><blockquote>
<p>Hypoxia15 low 41 35</p>
</blockquote></th>
<th><blockquote>
<p>29</p>
</blockquote></th>
<th><blockquote>
<p>21</p>
</blockquote></th>
<th>16</th>
<th><blockquote>
<p>11 Hypoxia26 low 22 20</p>
</blockquote></th>
<th><blockquote>
<p>17</p>
</blockquote></th>
<th><blockquote>
<p>11</p>
</blockquote></th>
<th><blockquote>
<p>9</p>
</blockquote></th>
<th>4</th>
<th><blockquote>
<p>Risk low</p>
</blockquote></th>
<th><blockquote>
<p>78 61</p>
</blockquote></th>
<th>52</th>
<th><blockquote>
<p>43</p>
</blockquote></th>
<th><blockquote>
<p>38</p>
</blockquote></th>
<th>26</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><blockquote>
<p>Hypoxia15 high 104 69</p>
</blockquote></td>
<td><blockquote>
<p>60</p>
</blockquote></td>
<td><blockquote>
<p>52</p>
</blockquote></td>
<td>45</td>
<td><blockquote>
<p>31 Hypoxia26 high 123 84</p>
</blockquote></td>
<td><blockquote>
<p>72</p>
</blockquote></td>
<td><blockquote>
<p>62</p>
</blockquote></td>
<td><blockquote>
<p>52</p>
</blockquote></td>
<td>38</td>
<td><blockquote>
<p>Risk high</p>
</blockquote></td>
<td><blockquote>
<p>66 43</p>
</blockquote></td>
<td>37</td>
<td><blockquote>
<p>30</p>
</blockquote></td>
<td><blockquote>
<p>23</p>
</blockquote></td>
<td>16</td>
</tr>
</tbody>
</table>

> Fig. 2. (A--I) Kaplan-Maier estimates of loco-regional control (LRC)
> for the validation cohort stratified for (A) HPV16 DNA status, (B)
> chemotherapy, (C--F) cancer stem cell (CSC) marker expression, (G and
> H) hypoxia-related gene classification and (I) combined risk. *Gene
> names are indicated using italics*.
>
> []{#_bookmark17 .anchor}Table 3
>
> Training (cohort 1) and validation (cohort 2) of univariate Cox models
> containing cancer stem cell marker expression or hypoxia classifiers
> for the endpoints loco-regional control, freedom of distant metastases
> and overall survival. Shown is the hazard ratio (HR) with 95%
> confidence interval (CI) and the *p*-value testing the hypothesis HR =
> 1.
>
> []{#_bookmark18 .anchor}Loco-regional control Distant metastases
> Overall survival
>
> HR (95% CI) *p*-Value HR (95% CI) *p*-Value HR (95% CI) *p*-Value
>
> Cohort

+---------+----+-----------+-----+----------+-----+----------+-----+
| > [     |    |           |     |          |     |          |     |
| ]{#_boo |    |           |     |          |     |          |     |
| kmark19 |    |           |     |          |     |          |     |
| > .anch |    |           |     |          |     |          |     |
| or}*All |    |           |     |          |     |          |     |
| > pa    |    |           |     |          |     |          |     |
| tients* |    |           |     |          |     |          |     |
+=========+====+===========+=====+==========+=====+==========+=====+
| > CD44  | >  | 9.09      | >   | 2.29     | >   | > 1.78   | >   |
| >       |  1 | (1.       |  0. | (0.9     |  0. | > (0.9   |  0. |
| protein |    | 24--66.8) | 030 | 0--5.86) | 084 | 1--3.50) | 093 |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  |           | >   | 1.80     | > 0 | > 1.16   | > 0 |
|         |  2 |           | [\* | (0.4     | .42 | > (0.5   | .71 |
|         |    |           | ](# | 3--7.54) |     | 3--2.53) |     |
|         |    |           | _bo |          |     |          |     |
|         |    |           | okm |          |     |          |     |
|         |    |           | ark |          |     |          |     |
|         |    |           | 18) |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 3.56      | >   | 1.69     | > 0 | > 1.57   | >   |
|  *CD44* |  1 | (1.       |  0. | (0.8     | .15 | > (0.9   |  0. |
| > \>    |    | 36--9.35) | 010 | 3--3.43) |     | 2--2.67) | 098 |
| > 0.2   |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 2.14      | >   | 2.30     | >   | > 1.20   | > 0 |
|         |  2 | (1.       |  0. | (1.0     |  0. | > (0.7   | .44 |
|         |    | 00--4.56) | 049 | 4--5.08) | 040 | 6--1.89) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| > *     | >  | 6.54      | >   | 4.17     | >   | > 2.37   | >   |
| SLC3A2* |  1 | (1.       |  0. | (1.7     |  0. | > (1.3   |  0. |
| > \>    |    | 97--21.7) | 002 | 4--10.0) | 001 | 6--4.13) | 002 |
| >       |    |           |     |          |     |          |     |
|  -3.135 |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 2.45      | >   | 1.78     | > 0 | > 1.19   | > 0 |
|         |  2 | (1.       |  0. | (0.7     | .18 | > (0.7   | .48 |
|         |    | 01--5.91) | 047 | 7--4.10) |     | 3--1.93) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| > *MET* | >  | 5.19      | >   | 3.41     | >   | > 2.74   | > \ |
| > \>    |  1 | (1.       |  0. | (1.6     |  0. | > (1.6   | <0. |
| >       |    | 97--13.7) | 001 | 1--7.23) | 001 | 0--4.70) | 001 |
|  -4.135 |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 1.43      | > 0 | 1.18     | > 0 | > 0.93   | > 0 |
|         |  2 | (0.       | .35 | (0.5     | .67 | > (0.5   | .74 |
|         |    | 67--3.07) |     | 6--2.47) |     | 8--1.47) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 3.55      | >   | 1.27     | > 0 | > 1.47   | > 0 |
| 15-gene |  1 | (1.       |  0. | (0.6     | .49 | > (0.8   | .14 |
| > si    |    | 35--9.35) | 010 | 5--2.46) |     | 8--2.47) |     |
| gnature |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 2.32      | >   | 1.06     | > 0 | > 1.20   | > 0 |
|         |  2 | (0.       |  0. | (0.5     | .89 | > (0.7   | .46 |
|         |    | 96--5.59) | 061 | 0--2.22) |     | 4--1.96) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 9.37      | >   | 2.15     | >   | > 2.48   | >   |
| 26-gene |  1 | (2.       |  0. | (1.0     |  0. | > (1.3   |  0. |
| > si    |    | 22--39.5) | 002 | 1--4.55) | 046 | 9--4.42) | 002 |
| gnature |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 3.36      | >   | 2.16     | > 0 | > 1.26   | > 0 |
|         |  2 | (0.       |  0. | (0.6     | .21 | > (0.6   | .48 |
|         |    | 81--14.0) | 097 | 6--7.07) |     | 6--2.38) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| > *P    |    |           |     |          |     |          |     |
| atients |    |           |     |          |     |          |     |
| > with  |    |           |     |          |     |          |     |
| > HPV16 |    |           |     |          |     |          |     |
| > DNA-n |    |           |     |          |     |          |     |
| egative |    |           |     |          |     |          |     |
| > t     |    |           |     |          |     |          |     |
| umours* |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+

> CD44 protein 1 [^\*^](#_bookmark18)
>
> 2 [\*](#_bookmark18)

+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 4.15      | >   | 1.69     | > 0 | > 1.40   | > 0 |
|  *CD44* |  1 | (1.       |  0. | (0.7     | .23 | > (0.7   | .28 |
| > \>    |    | 25--13.8) | 020 | 2--3.94) |     | 6--2.58) |     |
| > 0.2   |    |           |     |          |     |          |     |
+=========+====+===========+=====+==========+=====+==========+=====+
|         | >  | 1.85      | > 0 | 2.54     | >   | > 1.17   | > 0 |
|         |  2 | (0.       | .11 | (1.0     |  0. | > (0.7   | .52 |
|         |    | 86--3.96) |     | 4--6.22) | 041 | 2--1.90) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| > *     | >  | 9.23      | >   | 3.56     | >   | > 1.53   | > 0 |
| SLC3A2* |  1 | (1.       |  0. | (1.0     |  0. | > (0.7   | .21 |
| > \>    |    | 25--68.1) | 029 | 8--11.7) | 037 | 9--2.98) |     |
| >       |    |           |     |          |     |          |     |
|  -3.135 |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 1.87      | > 0 | 1.80     | > 0 | > 1.13   | > 0 |
|         |  2 | (0.       | .17 | (0.6     | .23 | > (0.6   | .64 |
|         |    | 77--4.55) |     | 9--4.73) |     | 7--1.93) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| > *MET* | >  | 3.74      | >   | 2.82     | >   | > 2.42   | >   |
| > \>    |  1 | (1.       |  0. | (1.1     |  0. | > (1.2   |  0. |
| >       |    | 29--10.9) | 015 | 6--6.87) | 023 | 7--4.61) | 007 |
|  -4.135 |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 0.97      | > 0 | 0.75     | > 0 | > 0.71   | > 0 |
|         |  2 | (0.       | .94 | (0.3     | .48 | > (0.4   | .17 |
|         |    | 45--2.08) |     | 4--1.66) |     | 3--1.16) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 4.66      | >   | 1.54     | > 0 | > 1.88   | >   |
| 15-gene |  1 | (1.       |  0. | (0.7     | .26 | > (1.0   |  0. |
| > si    |    | 60--13.5) | 005 | 3--3.21) |     | 6--3.34) | 031 |
| gnature |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 2.13      | >   | 1.37     | > 0 | > 1.38   | > 0 |
|         |  2 | (0.       |  0. | (0.5     | .47 | > (0.8   | .25 |
|         |    | 88--5.15) | 093 | 9--3.21) |     | 0--2.37) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
| >       | >  | 11.3      | >   | 1.51     | > 0 | > 2.05   | >   |
| 26-gene |  1 | (1.       |  0. | (0.6     | .34 | > (1.0   |  0. |
| > si    |    | 53--83.7) | 017 | 5--3.51) |     | 3--4.08) | 040 |
| gnature |    |           |     |          |     |          |     |
+---------+----+-----------+-----+----------+-----+----------+-----+
|         | >  | 1.62      | > 0 | 1.36     | > 0 | > 0.91   | > 0 |
|         |  2 | (0.       | .51 | (0.3     | .68 | > (0.4   | .81 |
|         |    | 39--6.78) |     | 2--5.73) |     | 2--1.99) |     |
+---------+----+-----------+-----+----------+-----+----------+-----+

> Bold numbers indicate significant *p*-Values with *p* \< 0.05.
>
> ^\*^ No convergence due to no events in the CD44 protein negative
> group.
>
> Table 4
>
> Training and validation of multivariate Cox models for the endpoint
> loco-regional control containing cancer stem cell marker expression
> and hypoxia classifiers in addition to HPV16 DNA status and the
> clinical parameters ECE status, localisation oropharynx and
> hypopharynx. Shown is the concordance index (ci) with 95% confidence
> interval (CI) and the *p*-value testing the hypothesis ci = 0.5.
> Validation was also performed for the subgroup of patients who
> received radiochemotherapy (RCT).

<table>
<colgroup>
<col style="width: 34%" />
<col style="width: 11%" />
<col style="width: 6%" />
<col style="width: 4%" />
<col style="width: 11%" />
<col style="width: 6%" />
<col style="width: 4%" />
<col style="width: 13%" />
<col style="width: 5%" />
<col style="width: 1%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th>Training cohort</th>
<th></th>
<th></th>
<th>Validation cohort</th>
<th></th>
<th></th>
<th><blockquote>
<p>Validation cohort, RCT</p>
</blockquote></th>
<th><blockquote>
<p>only</p>
</blockquote></th>
<th rowspan="2"></th>
</tr>
<tr class="odd">
<th><blockquote>
<p>Included variables</p>
</blockquote></th>
<th>ci (95% CI)</th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
<th></th>
<th>ci (95% CI)</th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
<th></th>
<th><blockquote>
<p>ci (95% CI)</p>
</blockquote></th>
<th><blockquote>
<p><em>p</em>-Value</p>
</blockquote></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><blockquote>
<p><em>All patients</em></p>
</blockquote></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Baseline (clinical parameters, HPV16 DNA status)</p>
</blockquote></td>
<td>0.76 (0.67–0.83)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.66 (0.58–0.75)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.69 (0.40–0.91)</p>
</blockquote></td>
<td><blockquote>
<p>0.15</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>CD44</em> &gt; 0.2</p>
</blockquote></td>
<td>0.76 (0.68–0.83)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.69 (0.60–0.77)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.77 (0.55–0.93)</p>
</blockquote></td>
<td><blockquote>
<p>0.02</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>SLC3A2</em> &gt; -3.135</p>
</blockquote></td>
<td>0.77 (0.68–0.84)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.66 (0.56–0.76)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.72 (0.44–0.92)</p>
</blockquote></td>
<td><blockquote>
<p>0.10</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>MET</em> &gt; -4.135</p>
</blockquote></td>
<td>0.78 (0.69–0.85)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.64 (0.54–0.74)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.75 (0.50–0.92)</p>
</blockquote></td>
<td><blockquote>
<p>0.04</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>15-gene signature</p>
</blockquote></td>
<td>0.80 (0.73–0.87)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.68 (0.58–0.77)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.74 (0.44–0.93)</p>
</blockquote></td>
<td><blockquote>
<p>0.10</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>26-gene signature</p>
</blockquote></td>
<td>0.79 (0.71–0.85)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.67 (0.58–0.75)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.71 (0.43–0.91)</p>
</blockquote></td>
<td><blockquote>
<p>0.11</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>CD44</em> &gt; 0.2, 15-gene signature</p>
</blockquote></td>
<td>0.80 (0.72–0.87)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.69 (0.60–0.78)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.78 (0.53–0.94)</p>
</blockquote></td>
<td><blockquote>
<p>0.04</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>SLC3A2</em> &gt; -3.135, 15-gene signature</p>
</blockquote></td>
<td>0.81 (0.73–0.87)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.67 (0.58–0.75)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.76 (0.48–0.95)</p>
</blockquote></td>
<td><blockquote>
<p>0.07</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>MET</em> &gt; -4.135, 15-gene signature</p>
</blockquote></td>
<td>0.81 (0.74–0.88)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.66 (0.56–0.75)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.75 (0.47–0.93)</p>
</blockquote></td>
<td><blockquote>
<p>0.07</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>Patients with HPV16 DNA negative tumours</em></p>
</blockquote></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Baseline (clinical parameters)</p>
</blockquote></td>
<td>0.66 (0.55–0.77)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td>0.63 (0.54–0.71)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.60 (0.21–0.87)</p>
</blockquote></td>
<td><blockquote>
<p>0.49</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>CD44</em> &gt; 0.2</p>
</blockquote></td>
<td>0.68 (0.58–0.77)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.63 (0.53–0.73)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.70 (0.41–0.90)</p>
</blockquote></td>
<td><blockquote>
<p>0.13</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>SLC3A2</em> &gt; -3.135</p>
</blockquote></td>
<td>0.69 (0.58–0.79)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.62 (0.51–0.71)</td>
<td><blockquote>
<p>0.03</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.63 (0.26–0.89)</p>
</blockquote></td>
<td><blockquote>
<p>0.38</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>MET</em> &gt; -4.135</p>
</blockquote></td>
<td>0.71 (0.60–0.81)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.58 (0.47–0.70)</td>
<td><blockquote>
<p>0.16</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.66 (0.31–0.90)</p>
</blockquote></td>
<td><blockquote>
<p>0.25</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>15-gene signature</p>
</blockquote></td>
<td>0.74 (0.65–0.82)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.64 (0.54–0.74)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.64 (0.25–0.91)</p>
</blockquote></td>
<td><blockquote>
<p>0.36</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>26-gene signature</p>
</blockquote></td>
<td>0.71 (0.62–0.80)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.62 (0.53–0.71)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.61 (0.26–0.88)</p>
</blockquote></td>
<td><blockquote>
<p>0.42</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>CD44</em> &gt; 0.2, 15-gene signature</p>
</blockquote></td>
<td>0.74 (0.65–0.82)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.65 (0.54–0.74)</td>
<td><blockquote>
<p>0.01</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.69 (0.36–0.91)</p>
</blockquote></td>
<td><blockquote>
<p>0.19</p>
</blockquote></td>
<td></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>SLC3A2</em> &gt; -3.135, 15-gene signature</p>
</blockquote></td>
<td>0.76 (0.68–0.84)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.62 (0.52–0.72)</td>
<td><blockquote>
<p>0.03</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.68 (0.33–0.92)</p>
</blockquote></td>
<td><blockquote>
<p>0.24</p>
</blockquote></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p><em>MET</em> &gt; -4.135, 15-gene signature</p>
</blockquote></td>
<td>0.76 (0.68–0.84)</td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td></td>
<td>0.61 (0.50–0.71)</td>
<td><blockquote>
<p>0.05</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>0.66 (0.29–0.91)</p>
</blockquote></td>
<td><blockquote>
<p>0.29</p>
</blockquote></td>
<td></td>
</tr>
</tbody>
</table>

> Discussion
>
> In our previous multicentre retrospective study, we showed that HPV
> status, tumour expression of CSC markers and hypoxia-related genes
> play a prognostic role in patients with locally advanced HNSCC, who
> were treated by PORT-C [\[15,18\]](#_bookmark22). In the current
> study, these results were validated on an earlier,
>
> independent cohort with patients who received PORT or PORT-C between
> 1999 and 2006. Despite significant differences in patient
> characteristics and treatment outcome between the two cohorts, the
> prognostic ability of the CSC markers CD44 protein, *CD44* and
> *SLC3A2* could be confirmed in univariate analyses; for the 15-gene
> hypoxia classifier a statistical trend was obtained. This underlines
> the robustness of the evaluated marker set.
>
> Furthermore, it could be shown that *CD44* and the 15-gene hypoxia
> classifier moderately improved the ci of the published multivariate
> Cox models. However, this improvement did not reach statistical
> significance.
>
> The validation cohort consisted of patients treated between 1999 and
> 2006. These patients showed favourable clinical param- eters with less
> R1-resected tumours and less lymphnodes showing extracapsular
> extension. However, LRC and OS were significantly lower compared to
> the later treated training cohort [\[15\]](#_bookmark22). This may be
> explained by the lower number of oropharyngeal cancers in this
> validation cohort, which show a significantly improved out- come due
> to a high radiosensitivity associated with positive HPV infection
> status [\[21,22\]](#_bookmark25) and the higher number of tumours in
> the oral cavity that generally show poorer outcome. From the litera-
> ture, there is also conflicting data on the value of HPV as a prognos-
> tic factor for non-oropharyngeal cancers. Lassen et al. showed that
> the radiotherapy outcome in the subgroup of non-oropharyngeal tumours
> does not differ by their p16 status [\[12\]](#_bookmark31). In
> contrast, Chung et al. demonstrated, that HPV infection may also play
> a role in a subset of non-oropharyngeal cancers
> [\[23\]](#_bookmark26), suggesting that this needs further
> exploration. Furthermore, only 40 of the 152 patients received
> simultaneous radiochemotherapy since this had not been a standard
> treatment at that time. Technological advances in radio- therapy
> [\[3\]](#_bookmark20) and diagnostics may also have contributed to a
> better outcome of the patients in the training cohort. Despite these
> signif- icant differences a statistical trend for improved LRC for
> patients with HPV16 DNA positive tumours could be obtained, which is
> in agreement with our previous study [\[15\]](#_bookmark22).
>
> Validation of the multivariate Cox models published in
> [\[18\]](#_bookmark22)
>
> showed a generally lower prognostic performance for the valida- tion
> cohort than for the training cohort (training: ci 0.66--0.81, val-
> idation: ci 0.58--0.69). This difference is expected, as the models
> were adjusted to the training data. Furthermore, cut-offs for the
> continuous CSC marker expressions as well as the clustering proce-
> dure for hypoxia classification were optimised for the training
> cohort. However, with an average ci of 0.67 for the total validation
> cohort and of 0.62 for patients with HPV16 DNA negative tumours, the
> multivariate models were still prognostic with an acceptable
> performance. The large difference in ci between all patients and those
> patients with HPV16 DNA negative tumours shows that the HPV16 DNA
> status is a strong prognosticator for LRC. Still, *CD44* and the
> 15-gene hypoxia classifier were able to further improve the models,
> also in the validation cohort. At first glance, it is sur- prising
> that tumour hypoxia-associated gene expression of the resected tumour
> is associated with the outcome of postoperative radiotherapy. As
> discussed before [\[18\]](#_bookmark22), it is very unlikely that the
> low number of any remaining tumour cells after surgery differs in
> their hypoxia status. However, this suggests that hypoxia may have the
> potential to impact the outcome after postoperative radiotherapy also
> by other radiobiological mechanisms than by direct biochemical or
> radiobiological effects. In previous data from our laboratory, it was
> shown that pre-treatment hypoxia has an impact on local tumour control
> after radiotherapy also when radio- therapy was applied under
> homogenous anoxic conditions [\[24\]](#_bookmark28).
>
> The prognostic ability of the putative CSC marker *MET* could
>
> not be confirmed by the validation cohort. However, the inclusion of
> *MET* resulted in multivariate models with higher ci values if only
> patients of the validation cohort who received PORT-C were considered.
> These models were not significant, which could be due to (1) the lower
> incidence of HPV positive tumours, which has shown to be associated
> with a lower CSC frequency [\[25\]](#_bookmark30) and (2) the low
> number of patients receiving PORT-C in the vali- dation cohort.
> Furthermore, in the validation cohort, patients only received PORT-C
> within clinical trials, i.e. a selection of patients with more
> favourable clinical characteristics is likely, leading to a more
> homogeneous patient group compared to the total cohort.
>
> In addition, diagnostics and radiotherapy treatment have been
> technologically improved over the last years, such that the con-
> sideration of biological aspects might become even more important.
>
> It should be noted that in the previously reported DKTK-ROG cohort
> [\[18\]](#_bookmark22) *CD44* expression was determined by real-time
> PCR (RT-PCR) analysis because the nanoString *CD44* probe design was
> invalid. In the present validation cohort, *CD44* expression was mea-
> sured by valid nanoString probes. Therefore, the validation of *CD44*
> is challenging, as its expression in the two cohorts was determined by
> different methods and the cut-off value 0.2 obtained for RT-PCR might
> be different for nanoString. In general, many genes of the validation
> cohort showed significantly different mean expression values between
> the cohorts, i.e. an average shift. This is problem- atic if the
> expressions are dichotomized at cut-off values, as these cut-offs may
> lead to imbalanced patient groups for the validation cohort. The same
> holds for the hypoxia classifiers built by k- means clustering. In the
> validation cohort, this shift in gene expres- sions caused the hypoxia
> classifiers to identify significantly more hypoxic tumours compared to
> the training cohort. Renormalizing the validation data to the training
> data, as described in [\[26\]](#_bookmark32), gives the same fraction
> of more and less hypoxic tumours for both cohorts and even leads to a
> significant impact of hypoxia status on LRC, in contrast to the
> statistical trend shown in [Table 3](#_bookmark17). How- ever, this
> method is not applicable for individual patient prognosis, as required
> in clinical trials on treatment adaptation, and may war- rant the
> inclusion of reference samples in future analyses.
>
> The model validation showed a better performance for the sub-
>
> group of patients, which received PORT-C than for all patients. This
> indicates that the prognostic ability of CSC markers and hypoxia
> classifiers might be stronger for a patient cohort, which is more
> similar to the cohort originally used in [\[18\]](#_bookmark22).
> However, at the time of treatment of the validation cohort
> simultaneous radiochemotherapy was applied only within clinical
> studies [\[7--](#_bookmark21) [9\]](#_bookmark21). This potentially
> led to a higher homogeneity with better clinical performance of the
> patient subgroup treated with PORT-C, which is important for the
> interpretation of the biological heterogeneity of the tumours.
> Currently, a homogeneous patient cohort with locally advanced HNSCC
> treated with PORT-C is being recruited within a prospective clinical
> trial of the DKTK-ROG, which will allow for fur- ther validation of
> the models.
>
> Taken together, this validation study confirmed the prognostic value
> of the HPV infection status, CSC marker expression of CD44 protein,
> *CD44* and *SLC3A2* and tumour hypoxia status presented in
> [\[18\]](#_bookmark22) for patients with locally advanced HNSCC
> receiving postop- erative radiotherapy or radiochemotherapy. While a
> lower perfor- mance of the prognostic models is expected due to the
> older validation dataset and several differences between the cohorts,
> the significant validation results indicate the robustness of these
> biomarkers. After further validation on a currently recruiting
> prospective clinical trial of the DKTK-ROG these models may help to
> stratify patients for individualised treatment de-escalation or
> intensification strategies.
>
> Conflict of interest
>
> The authors declare no conflict of interest regarding the present
> manuscript.
>
> Acknowledgements
>
> The authors gratefully acknowledge the excellent technical assistance
> by Mrs. Liane Stolz-Kieslich, Mrs. Sigrid Balschukat and Mrs. Daniela
> Friede. FFPE specimens were kindly provided by the Tumour- and Normal
> Tissue Bank (TNTB; Dr. Daniela E. Aust)
>
> [[[[[[]{#_bookmark25 .anchor}]{#_bookmark24 .anchor}]{#_bookmark23
> .anchor}]{#_bookmark22 .anchor}]{#_bookmark21 .anchor}]{#_bookmark20
> .anchor}of the University Cancer Center (UCC) of the University
> Hospital Dresden.
>
> Appendix A. Supplementary material
>
> []{#_bookmark26 .anchor}Supplementary data associated with this
> article can be found, in the online version, at
> <http://dx.doi.org/10.1016/j.ctro.2016.10.002>.
>
> [[]{#_bookmark28 .anchor}]{#_bookmark27 .anchor}References

1.  []{#_bookmark29 .anchor}Leemans CR, Braakhuis BJM, Brakenhoff RH.
    The molecular biology of head and neck cancer. Nat Rev Cancer
    2011;11:9--22. doi:
    [[http://dx.doi.org/10.1038/]{.underline}](http://dx.doi.org/10.1038/nrc2982)
    []{#_bookmark30
    .anchor}[[nrc2982]{.underline}](http://dx.doi.org/10.1038/nrc2982).

2.  []{#_bookmark31 .anchor}Ferlay J, Soerjomataram I, Dikshit R, Eser
    S, Mathers C, Rebelo M, et al. Cancer incidence and mortality
    worldwide: sources, methods and major patterns in GLOBOCAN 2012. Int
    J Cancer 2015;136:E359--86. doi:
    [[http://dx.doi.org/]{.underline}](http://dx.doi.org/10.1002/ijc.29210)
    []{#_bookmark32
    .anchor}[[10.1002/ijc.29210]{.underline}](http://dx.doi.org/10.1002/ijc.29210).

3.  Baumann M, Krause M, Overgaard J, Debus J, Bentzen SM, Daartz J, et
    al. Radiation oncology in the era of precision medicine. Nat Rev
    Cancer 2016;16:234--49. doi:
    [[http://dx.doi.org/10.1038/nrc.2016.18]{.underline}](http://dx.doi.org/10.1038/nrc.2016.18).

4.  Chaturvedi AK, Anderson WF, Lortet-Tieulent J, Curado MP, Ferlay J,
    Franceschi S, et al. Worldwide trends in incidence rates for oral
    cavity and oropharyngeal cancers. J Clin Oncol Off J Am Soc Clin
    Oncol 2013;31:4550--9. doi:
    [[http://dx]{.underline}.](http://dx.doi.org/10.1200/JCO.2013.50.3870)
    [[doi.org/10.1200/JCO.2013.50.3870]{.underline}](http://dx.doi.org/10.1200/JCO.2013.50.3870).

5.  Sturgis EM, Cinciripini PM. Trends in head and neck cancer incidence
    in relation to smoking prevalence: an emerging epidemic of human
    papillomavirus-associated cancers? Cancer 2007;110:1429--35. doi:
    [[http://]{.underline}](http://dx.doi.org/10.1002/cncr.22963)
    [[dx.doi.org/10.1002/cncr.22963]{.underline}](http://dx.doi.org/10.1002/cncr.22963).

6.  Habbous S, Chu KP, Qiu X, La Delfa A, Harland LTG, Fadhel E, et al.
    The changing incidence of human papillomavirus-associated
    oropharyngeal cancer using multiple imputation from 2000 to 2010 at
    a Comprehensive Cancer Centre. Cancer Epidemiol 2013;37:820--9. doi:
    [[http://dx.doi.org/10.1016/]{.underline}](http://dx.doi.org/10.1016/j.canep.2013.09.011)
    [[j.canep.2013.09.011]{.underline}](http://dx.doi.org/10.1016/j.canep.2013.09.011).

7.  Bernier J, Domenge C, Ozsahin M, Matuszewska K, Lefèbvre J-L,
    Greiner RH, et al. Postoperative irradiation with or without
    concomitant chemotherapy for locally advanced head and neck cancer.
    N Engl J Med 2004;350:1945--52. doi:
    [[http://dx.doi.org/10.1056/NEJMoa032641]{.underline}](http://dx.doi.org/10.1056/NEJMoa032641).

8.  Cooper JS, Pajak TF, Forastiere AA, Jacobs J, Campbell BH, Saxman
    SB, et al. Postoperative concurrent radiotherapy and chemotherapy
    for high-risk squamous-cell carcinoma of the head and neck. N Engl J
    Med 2004;350:1937--44. doi:
    [[http://dx.doi.org/10.1056/NEJMoa032646]{.underline}](http://dx.doi.org/10.1056/NEJMoa032646).

9.  [Fietkau R, Lautenschläger C, Sauer R, Dunst J, Becker A, Baumann
    M.](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0045)
    [Postoperative concurrent radiochemotherapy versus radiotherapy in
    high-](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0045) [risk
    SCCA of the head and neck: Results of the German phase III trial ARO
    96-3.](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0045) [J
    Clin Oncol
    2006;24:5507](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0045).

10. Fakhry C, Gillison ML. Clinical implications of human papillomavirus
    in head and neck cancers. J Clin Oncol Off J Am Soc Clin Oncol
    2006;24:2606--11. doi:
    [[http://dx.doi.org/10.1200/JCO.2006.06.1291]{.underline}](http://dx.doi.org/10.1200/JCO.2006.06.1291).

11. Ang KK, Harris J, Wheeler R, Weber R, Rosenthal DI, Nguyen-Tân PF,
    et al. Human papillomavirus and survival of patients with
    oropharyngeal cancer. N Engl J Med 2010;363:24--35. doi:
    [[http://dx.doi.org/10.1056/NEJMoa0912217]{.underline}](http://dx.doi.org/10.1056/NEJMoa0912217).

12. Lassen P, Primdahl H, Johansen J, Kristensen CA, Andersen E,
    Andersen LJ, et al. Impact of HPV-associated p16-expression on
    radiotherapy outcome in advanced oropharynx and non-oropharynx
    cancer. Radiother Oncol J Eur Soc Ther Radiol Oncol 2014;113:310--6.
    doi:
    [[http://dx.doi.org/10.1016/j.radonc]{.underline}.](http://dx.doi.org/10.1016/j.radonc.2014.11.032)
    [[2014.11.032]{.underline}](http://dx.doi.org/10.1016/j.radonc.2014.11.032).

13. Swick AD, Chatterjee A, Costa A-MA, Kimple RJ. Modulation of
    therapeutic sensitivity by human papillomavirus. Radiother Oncol J
    Eur Soc Ther Radiol Oncol 2015;116:342--5. doi:
    [[http://dx.doi.org/10.1016/j.radonc.2015.09.002]{.underline}](http://dx.doi.org/10.1016/j.radonc.2015.09.002).

14. Hong A, Zhang X, Jones D, Veillard A-S, Zhang M, Martin A, et al.
    Relationships between p53 mutation, HPV status and outcome in
    oropharyngeal squamous cell carcinoma. Radiother Oncol J Eur Soc
    Ther Radiol Oncol 2016;118:342--9. doi:
    [[http://dx.doi.org/10.1016/j.radonc.2016.02.009]{.underline}](http://dx.doi.org/10.1016/j.radonc.2016.02.009).

15. Lohaus F, Linge A, Tinhofer I, Budach V, Gkika E, Stuschke M, et al.
    HPV16 DNA status is a strong prognosticator of loco-regional control
    after postoperative radiochemotherapy of locally advanced
    oropharyngeal carcinoma: results from a multicentre explorative
    study of the German Cancer Consortium Radiation Oncology Group
    (DKTK-ROG). Radiother Oncol J Eur Soc Ther Radiol Oncol
    2014;113:317--23. doi:
    [[http://dx.doi.org/10.1016/j.radonc.2014.11.011]{.underline}](http://dx.doi.org/10.1016/j.radonc.2014.11.011).

16. Toustrup K, Sørensen BS, Nordsmark M, Busk M, Wiuf C, Alsner J, et
    al. Development of a hypoxia gene expression classifier with
    predictive impact for hypoxic modification of radiotherapy in head
    and neck cancer. Cancer Res 2011;71:5923--31. doi:
    [[http://dx.doi.org/10.1158/0008-5472.CAN-11-1182]{.underline}](http://dx.doi.org/10.1158/0008-5472.CAN-11-1182).

17. Eustace A, Mani N, Span PN, Irlam JJ, Taylor J, Betts GNJ, et al. A
    26-gene hypoxia signature predicts benefit from hypoxia-modifying
    therapy in laryngeal cancer but not bladder cancer. Clin Cancer Res
    An Off J Am Assoc Cancer Res 2013;19:4879--88. doi:
    [[http://dx.doi.org/10.1158/1078-0432.CCR-]{.underline}](http://dx.doi.org/10.1158/1078-0432.CCR-13-0542)
    [[13-0542]{.underline}](http://dx.doi.org/10.1158/1078-0432.CCR-13-0542).

18. Linge A, Löck S, Gudziol V, Nowak A, Lohaus F, von Neubeck C, et al.
    Low cancer stem cell marker expression and low hypoxia identify good
    prognosis subgroups in HPV(-) HNSCC after Postoperative
    radiochemotherapy: a multicenter study of the DKTK-ROG. Clin Cancer
    Res An Off J Am Assoc Cancer Res 2016;22:2639--49. doi:
    [[http://dx.doi.org/10.1158/1078-0432.CCR-]{.underline}](http://dx.doi.org/10.1158/1078-0432.CCR-15-1990)
    [[15-1990]{.underline}](http://dx.doi.org/10.1158/1078-0432.CCR-15-1990).

19. Skripcak T, Belka C, Bosch W, Brink C, Brunner T, Budach V, et al.
    Creating a data exchange strategy for radiotherapy research: towards
    federated databases and anonymised public datasets. Radiother Oncol
    J Eur Soc Ther Radiol Oncol 2014;113:303--9. doi:
    [[http://dx.doi.org/10.1016/j]{.underline}.](http://dx.doi.org/10.1016/j.radonc.2014.10.001)
    [[radonc.2014.10.001]{.underline}](http://dx.doi.org/10.1016/j.radonc.2014.10.001).

20. [Harrell F. Regression modeling strategies: with applications to
    linear
    models,](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0100)
    [logistic regression, and survival analysis. Springer;
    2001](http://refhub.elsevier.com/S2405-6308(16)30006-4/h0100).

21. Kawakami H, Okamoto I, Terao K, Sakai K, Suzuki M, Ueda S, et al.
    Human papillomavirus DNA and p16 expression in Japanese patients
    with oropharyngeal squamous cell carcinoma. Cancer Med
    2013;2:933--41. doi:
    [[http://dx.doi.org/10.1002/cam4.151]{.underline}](http://dx.doi.org/10.1002/cam4.151).

22. Salazar CR, Smith RV, Garg MK, Haigentz M, Schiff BA, Kawachi N, et
    al. Human papillomavirus-associated head and neck squamous cell
    carcinoma survival: a comparison by tumor site and initial
    treatment. Head Neck Pathol 2014;8:77--87. doi:
    [[http://dx.doi.org/10.1007/s12105-013-0486-4]{.underline}](http://dx.doi.org/10.1007/s12105-013-0486-4).

23. Chung CH, Zhang Q, Kong CS, Harris J, Fertig EJ, Harari PM, et al.
    P16 protein expression and human papillomavirus status as prognostic
    biomarkers of nonoropharyngeal head and neck squamous cell
    carcinoma. J Clin Oncol Off J Am Soc Clin Oncol 2014;32:3930--8.
    doi:
    [[http://dx.doi.org/10.1200/]{.underline}](http://dx.doi.org/10.1200/JCO.2013.54.5228)
    [[JCO.2013.54.5228]{.underline}](http://dx.doi.org/10.1200/JCO.2013.54.5228).

24. Yaromina A, Thames H, Zhou X, Hering S, Eicheler W, Dörfler A, et
    al. Radiobiological hypoxia, histological parameters of tumour
    microenvironment and local tumour control after fractionated
    irradiation. Radiother Oncol J Eur Soc Ther Radiol Oncol
    2010;96:116--22. doi:
    [[http://dx.doi.org/10.1016/j]{.underline}.](http://dx.doi.org/10.1016/j.radonc.2010.04.020)
    [[radonc.2010.04.020]{.underline}](http://dx.doi.org/10.1016/j.radonc.2010.04.020).

25. Vlashi E, Chen AM, Boyrie S, Yu G, Nguyen A, Brower PA, et al.
    Radiation- induced dedifferentiation of head and neck cancer cells
    into cancer stem cells depends on human papillomavirus status. Int J
    Radiat Oncol Biol Phys 2016;94:1198--206. doi:
    [[http://dx.doi.org/10.1016/j.ijrobp.2016.01.005]{.underline}](http://dx.doi.org/10.1016/j.ijrobp.2016.01.005).

26. Toustrup K, Sørensen BS, Metwally MAH, Tramm T, Mortensen LS,
    Overgaard J, et al. Validation of a 15-gene hypoxia classifier in
    head and neck cancer for prospective use in clinical trials. Acta
    Oncol 2016:1--8. doi:
    [[http://dx.doi.org/]{.underline}](http://dx.doi.org/10.3109/0284186X.2016.1167959)
    [[10.3109/0284186X.2016.1167959]{.underline}](http://dx.doi.org/10.3109/0284186X.2016.1167959).

> ![](media/image1.jpeg){width="0.8359284776902887in"
> height="0.913332239720035in"}![](media/image50.jpeg)[Clinical and
> Translational Radiation Oncology 16 (2019)
> 40--47](https://doi.org/10.1016/j.ctro.2019.03.002)
>
> Original Research Article

## ![](media/image51.png){width="0.39697178477690287in" height="0.39697178477690287in"}Independent validation of tumour volume, cancer stem cell markers and hypoxia-associated gene expressions for HNSCC after primary radiochemotherapy {#independent-validation-of-tumour-volume-cancer-stem-cell-markers-and-hypoxia-associated-gene-expressions-for-hnscc-after-primary-radiochemotherapy .unnumbered}

> Annett Linge
> ^[a](#_bookmark33),[b](#_bookmark34),[c](#_bookmark35),[d](#_bookmark36),^[⇑](#_bookmark47),[1](#_bookmark45),
> Stefan Schmidt
> ^[a](#_bookmark33),[b](#_bookmark34),[e](#_bookmark37),[1](#_bookmark45)^,
> Fabian Lohaus
> ^[a](#_bookmark33),[b](#_bookmark34),[c](#_bookmark35),[d](#_bookmark36)^,
> Constanze Krenn [^b^](#_bookmark34), Anna Bandurska-Luque
> ^[b](#_bookmark34),[c](#_bookmark35)^, Ivan Platzek
> [^f^](#_bookmark38), Cläre von Neubeck
> ^[a](#_bookmark33),[b](#_bookmark34)^, Steffen Appold
> ^[c](#_bookmark35),[d](#_bookmark36)^, Alexander Nowak
> ^[d](#_bookmark36),[g](#_bookmark39)^, Volker Gudziol
> ^[d](#_bookmark36),[h](#_bookmark40)^, Frank Buchholz
> ^[a](#_bookmark33),[i](#_bookmark41)^, Gustavo B. Baretton
> ^[a](#_bookmark33),[d](#_bookmark36),[j](#_bookmark42),[k](#_bookmark43)^,
>
> Michael Baumann
> ^[a](#_bookmark33),[b](#_bookmark34),[c](#_bookmark35),[l](#_bookmark44)^,
> Steffen Löck
> ^[a](#_bookmark33),[b](#_bookmark34),[c](#_bookmark35),[2](#_bookmark46)^,
> Mechthild Krause
> ^[a](#_bookmark33),[b](#_bookmark34),[c](#_bookmark35),[d](#_bookmark36),[e](#_bookmark37),[2](#_bookmark46)^
>
> [[]{#_bookmark34 .anchor}]{#_bookmark33 .anchor}^a^ *German Cancer
> Consortium (DKTK) Partner Site Dresden, Germany and German Cancer
> Research Center (DKFZ), Heidelberg, Germany*
>
> ^b^ *OncoRay -- National Center for Radiation Research in Oncology,
> Faculty of Medicine and University Hospital Carl Gustav Carus,
> Technische Universität Dresden,*
>
> []{#_bookmark35 .anchor}*Helmholtz-Zentrum Dresden -- Rossendorf,
> Dresden, Germany*
>
> []{#_bookmark36 .anchor}^c^ *Department of Radiotherapy and Radiation
> Oncology, Faculty of Medicine and University Hospital Carl Gustav
> Carus, Technische Universität Dresden, Germany*
>
> ^d^ *National Center for Tumor Diseases (NCT), Partner Site Dresden,
> Germany: German Cancer Research Center (DKFZ), Heidelberg, Germany;
> Faculty of Medicine and University Hospital* []{#_bookmark37
> .anchor}*Carl Gustav Carus, Technische Universität Dresden, Dresden,
> Germany, and; Helmholtz Association / Helmholtz-Zentrum Dresden --
> Rossendorf (HZDR), Dresden, Germany*
>
> []{#_bookmark38 .anchor}^e^ *Helmholtz-Zentrum Dresden -- Rossendorf,
> Institute of Radiooncology -- OncoRay, Germany*
>
> []{#_bookmark39 .anchor}^f^ *Department of Radiology, Faculty of
> Medicine and University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark40 .anchor}^g^ *Department of Oral and Maxillofacial
> Surgery, Faculty of Medicine and University Hospital Carl Gustav
> Carus, Technische Universität Dresden, Germany*
>
> []{#_bookmark41 .anchor}^h^ *Department of Otorhinolaryngology,
> Faculty of Medicine and University Hospital Carl Gustav Carus,
> Technische Universiät Dresden, Germany*
>
> []{#_bookmark42 .anchor}^i^ *University Cancer Centre (UCC), Medical
> Systems Biology, University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark43 .anchor}^j^ *Institute of Pathology, Faculty of
> Medicine and University Hospital Carl Gustav Carus, Technische
> Universität Dresden, Germany*
>
> []{#_bookmark44 .anchor}^k^ *Tumour- and Normal Tissue Bank,
> University Cancer Centre (UCC), University Hospital Carl Gustav Carus,
> Technische Universität Dresden, Germany*
>
> ^l^ *German Cancer Research Center (DKFZ), Heidelberg, Germany*
>
> [[]{#_bookmark46 .anchor}]{#_bookmark45 .anchor}a r t i c l e i n f o
>
> *Article history:*
>
> Received 23 November 2018
>
> Revised 13 March 2019
>
> Accepted 17 March 2019
>
> Available online 18 March 2019
>
> *Keywords:* Biomarker HNSCC
>
> Cancer stem cells HPV
>
> Hypoxia
>
> Primary radiochemotherapy
>
> a b s t r a c t
>
> *Objective:* To independently validate the impact of tumour volume,
> p16 status, cancer stem cell (CSC) mar- ker expression and
> hypoxia-associated gene signatures as potential prognostic biomarkers
> for patients with locally advanced head and neck squamous cell
> carcinoma (HNSCC), who underwent primary radio- therapy or
> radiochemotherapy (RCTx). These markers have previously been reported
> in a study of the German Cancer Consortium Radiation Oncology Group
> (DKTK-ROG) (Linge et al., 2016).
>
> *Materials and methods:* In this retrospective monocentric study, 92
> patients with locally advanced HNSCC were included. Univariable and
> multivariable logistic regressions and Cox models presented in the
> study of the DKTK-ROG were validated using the area under the curve
> (AUC) and the concordance index (ci), respec- tively. The primary
> endpoint of this study was loco-regional tumour control (LRC) after
> primary RCTx.
>
> *Results:* Although both cohorts significantly differed in the
> proportion of the tumour subsites, the param- eters tumour volume, p16
> status and N stage could be validated regarding LRC and overall
> survival (OS) using multivariable Cox regression (LRC ci: 0.59, OS ci:
> 0.63). These models were slightly improved by com- bination with the
> putative CSC marker *CD44* (LRC ci: 0.61, OS ci: 0.69). The logistic
> regression model for 2- year LRC based on tumour volume, p16 status
> and CD44 protein was validated with an AUC of 0.64. The patient
> stratification based on hypoxia-associated gene signatures status was
> similar to the original study but without significant differences in
> LRC and OS.
>
> *Conclusions:* In this validation study, the inclusion of the putative
> CSC marker CD44 slightly improved the prognostic performance of the
> baseline parameters tumour volume, p16 status and N stage. No
>
> []{#_bookmark47 .anchor}⇑ Corresponding author at: Department of
> Radiation Oncology and OncoRay -- National Center for Radiation
> Research in Oncology, Faculty of Medicine and University Hospital Carl
> Gustav Carus, Technische Universität Dresden, Fetscherstraße 74, 01307
> Dresden, Germany.
>
> *E-mail address:* <annett.linge@uniklinikum-dresden.de> (A. Linge).
>
> ^1^ Shared first authorship.
>
> ^2^ Shared senior authorship.
>
> <https://doi.org/10.1016/j.ctro.2019.03.002>
>
> 2405-6308/© 2019 The Authors. Published by Elsevier B.V. on behalf of
> European Society for Radiotherapy and Oncology. This is an open access
> article under the CC BY-NC-ND license
> (<http://creativecommons.org/licenses/by-nc-nd/4.0/>).
>
> improvement was observed when including expressions of the
> hypoxia-associated gene signatures. Prospective validation on a larger
> cohort is warranted to assess the clinical relevance of these markers.
>
> © 2019 The Authors. Published by Elsevier B.V. on behalf of European
> Society for Radiotherapy and Oncology. This is an open access article
> under the CC BY-NC-ND license
> ([http://creativecommons.org/](http://creativecommons.org/licenses/by-nc-nd/4.0/)

[licenses/by-nc-nd/4.0/](http://creativecommons.org/licenses/by-nc-nd/4.0/)).

1.  Introduction

> Head and neck squamous cell carcinoma (HNSCC) are repre- senting one
> of the ten most frequent tumours worldwide [\[2\]](#_bookmark54). Pri-
> mary radiochemotherapy (RCTx) is currently the considered treatment
> standard for patients with locally-advanced and func- tionally
> inoperable tumours, after several clinical trials showed a benefit
> over radiotherapy alone [\[3--8\]](#_bookmark55). However despite
> treatment escalation by simultaneous radiochemotherapy, the outcome of
> radio(chemo)therapy is still unsatisfying with only about half of the
> patients being alive after 5 years [\[9\]](#_bookmark62). Thus, the
> identification of patients who are very likely to have a poor
> treatment response is necessary and may be achieved by the
> complementation of well accepted clinical parameters by molecular
> biomarkers of the indi- vidual tumour. This may allow for inclusion of
> patients in treat- ment intensification trials, such as dose
> escalation or combination with novel systemic therapeutics
> [\[10,11\]](#_bookmark63).
>
> Over the last one to two decades, the human papilloma virus (HPV)
> infection status has become one of the major risk factors for the
> development of HNSCC besides alcohol abuse and tobacco
> [\[12--14\]](#_bookmark66). Several studies showed that patients with
> HPV-driven HNSCC show a favourable prognosis after primary or
> postoperative radiochemotherapy compared to those with HPV-negative
> tumours [\[1,15--18\]](#_bookmark56).
>
> In a recent multicentre study of the German Cancer Consortium
> Radiation Oncology Group (DKTK-ROG), previous studies were confirmed
> showing that patients with an overexpression of the HPV-surrogate
> marker p16 have a better LRC and OS compared to patients with
> HPV-negative tumours. Importantly, patient strat- ification regarding
> LRC and OS could be further improved, when the expression of the
> putative cancer stem cell markers CD44 or *SLC3A2* was also
> considered. Further refinement of for the predic- tion of prognosis
> was achieved by the tumour volume, which was suggested by others
> before [\[10,19--22\]](#_bookmark63). For patients with small tumours,
> tumour hypoxia as assessed by hypoxia-associated gene signatures was
> also found to be a prognostic biomarker.
>
> The aim of the present study was to independently validate these
> results obtained within the DKTK-ROG [\[23\]](#_bookmark57) using a
> retro- spective monocentric cohort of 92 patients with locally
> advanced HNSCC, who were treated by primary RCTx.

2.  Materials and methods

    1.  *Patients and study design*

> In the current publication, two independent patient cohorts who
> received curatively intended primary radio(chemo)therapy are being
> evaluated.
>
> The retrospective primary HNSCC cohort of the DKTK-ROG served as the
> training cohort and included 158 patients with locally advanced and
> histologically proven HNSCC. Inclusion crite- ria have previously been
> described in detail [\[1\]](#_bookmark56). All patients received
> primary RCTx with a median dose of 72.0 Gy (range 68.4--74.0 Gy) based
> on cisplatinum or mitomycin-C between 2005 and 2011 at one of six
> partner sites of the DKTK-ROG. The relation of the primary tumour
> volume, p16 status, the CSC mark- ers CD44 and *SLC3A2* and
> hypoxia-associated gene signatures to LRC and OS were investigated in
> this cohort before [\[1\]](#_bookmark56).
>
> To validate these results, an independent cohort of 92 patients with
> locally advanced HNSCC treated by curatively intended pri- mary RCTx
> was considered in this manuscript. Out of these 92 patients, 43
> patients were presented earlier in a prospective mono-centre
> single-arm non-randomised observational imaging trial, which was
> registered ([www.clinicaltrials.gov](http://www.clinicaltrials.gov/),
> NCT00180180) and approved by the German Federal Radiation Protection
> Authority (Bundesamt für Strahlenschutz, Z5 -- 22461/2 -- 2004-061)
> and the local Ethics Committee (EK166082004)
> [\[24,25\]](#_bookmark58). Briefly, these patients were treated
> between 2006 and 2013 at the DKTK partner site Dresden, had to be at
> least 18 years old with WHO performance status 0--2, were treated with
> primary RCTx and received a median dose of 72 Gy (range 69.0--72.0
> Gy). The remaining 49 patients were also treated at the Dresden site
> between 1999 and 2006 or 2009 and 2015 and received primary RCTx with
> a median dose of 70.6 Gy (range 70.0-- 76.8 Gy). None of the patients
> of the validation cohort were included in the training cohort. Further
> inclusion criteria were the histologically confirmation of the
> presence of squamous cell carcinoma arising from the oropharynx, oral
> cavity, hypopharynx or larynx. For all patients, formalin-fixed
> paraffin-embedded (FFPE) tumour material in terms of pre-treatment
> biopsies, radiotherapy treatment plans, computed tomography (CT),
> magnetic resonance imaging (MRI) or positron emission tomography--CT
> (PET/CT) images of the location of the recurrent tumours as well as
> follow-up data of patients had to be available. The composition of the
> cohorts is presented in [Fig. 1](#_bookmark48).

2.  *Segmentation and failure pattern analyses*

> The segmentations of the primary and of the nodal gross tumour volume
> (GTV) of all cases have retrospectively been per- formed in CT scans
> by one radiation oncologist (F.L.), who has expertise in the
> delineation of head and neck cancers. For segmen- tation, RayStation 6
> (Raysearch Laboratories, Stockholm Sweden) and an in-house software
> solution has been used [\[24\]](#_bookmark58). Disease sta- tus as
> well as the first site of relapse (e.g. loco-regional failure, dis-
> tant failure or combined failure) have been evaluated. For each
> loco-regional failure, the radiotherapy treatment plan and radio-
> logical images of the recurrence (CT, MRI or PET--CT) were centrally
> reviewed by one experienced radiation oncologist (F.L.) in order
> ensure that the failure occurred within the irradiated volume.

3.  *Preparation of biomaterials for biomarker analysis*

> The preparation of FFPE tissue material was performed as described in
> [\[1\]](#_bookmark56). Briefly, a fresh section of each FFPE block was
> first subjected to haematoxylin and eosin staining in order to
> histolog- ically confirm the presence of squamous cell carcinoma.
> After- wards, the FFPE material was further processed for
> immunohistochemistry or for preparation of genomic DNA or RNA under
> standardized conditions as described previously [\[1\]](#_bookmark56).
> Briefly, for p16 immunohistochemistry the CINtec Histology kit (Roche
> mtm laboratories AG, Basel, CH) was used according to the
> manufacturers' instructions [\[1\]](#_bookmark56). A moderate or
> strong overex- pression of p16 in at least 70% of the tumour cells was
> considered as a p16 positive tumour [\[1\]](#_bookmark56). For the
> immunohistochemical analy- sis of CD44 protein expression, the
> monoclonal mouse anti-human
>
> ![](media/image52.png)
>
> []{#_bookmark48 .anchor}Fig. 1. Study design. The number of patients
> initially available is presented along with the analyses, which were
> performed. IHC = immunohistochemical staining.
>
> ![](media/image56.png){width="1.1274529746281714in"
> height="0.10937445319335083in"}CD44 antibody (Clone DF1485; Dako) was
> used. Negative control slides were incubated with the corresponding
> IgG antibody control (Dako). CD44 staining intensity was considered as
> positive if speci- fic staining was observed in at least 5% of the
> tumour cells. Blinded samples were evaluated by two independent
> observers (AL and CvN) with an inter-observer variability of \<5% for
> all immunohisto- chemical analyses.
>
> DNA extraction and PCR-array based analyses of HPV status have been
> performed as described previously [\[26\]](#_bookmark60). Briefly,
> genomic DNA was extracted from 5-mm FFPE sections using the QIAamp DNA
> FFPE tissue kit (Qiagen). HPV DNA analyses including geno- typing were
> performed using the LCD-Array HPV 3.5 kit (CHIPRON GmbH, Berlin, DE)
> according to the manufacturer's instruction.
>
> Gene expression analyses were performed using nanoString Elements
> technology (nanoString Technologies, Seattle, WA, USA) as described
> previously [\[1\]](#_bookmark56) and included the potential CSC
> markers *CD44* and *SLC3A2* as well as the 15-, 26-, and 30-gene
> hypoxia- associated signatures [\[27--31\]](#_bookmark61). Briefly,
> the raw counts were loga- rithmised and then normalized to the mean of
> the internal level of reference genes *ACTR3*, *B2M*, *GNB2L1*,
> *NDFIP1*, *POLR2A*, *RPL11*, *RPL37A*. For the hypoxia-gene
> signatures, the corresponding refer- ence genes were used
> [\[27,30,31\]](#_bookmark61). Note that *DHX34* was not avail- able.
> Thus only 29 genes of the original 30-gene signature
> [\[27,30,31\]](#_bookmark61) were evaluated.

4.  *Clinical endpoints and statistical analyses*

> The primary endpoint was loco-regional tumour control (LRC). Overall
> survival (OS) was the secondary endpoint. The correspond- ing times
> were calculated starting from the first day of radiother- apy to the
> date of event or censoring. The Kaplan-Meier method was used to
> estimate survival curves. The endpoints were com- pared between
> stratified groups using log-rank tests. Univariable and multivariable
> Cox regression was applied to estimate the impact of potential
> prognostic variables on the endpoints. The con- cordance index (ci)
> was used to validate the performance of the multivariable Cox models
> defined in the training cohort [\[1\]](#_bookmark56). While a ci of
> 1.0 represents a perfect prediction, a ci of 0.5 is obtained for a
> non-informative model. The 95% confidence interval (95% CI) of the ci
> and the p-value of the corresponding model were estimated by 1000
> bootstrap samples of the particular cohort. The validation
>
> was considered successful when the lower boundary of the 95% CI was
> above 0.5. On the training cohort a multivariable logistic regression
> model was developed, predicting 2-year LRC [\[1\]](#_bookmark56). This
> model was validated using the area under the receiver operating
> characteristics curve (AUC). As for the ci, a non-informative model
> leads to an AUC of 0.5 and a perfect model to an AUC of 1.0
> [\[32\]](#_bookmark64). Differences between the cohorts were evaluated
> by Mann- Whitney-U tests (continuous variables) and chi-squared tests
> (cat- egorical variables). Hypoxia classification was performed on the
> validation cohort using two cluster centres for every gene signa-
> ture, which were determined on the training cohort by k-means
> clustering [\[1\]](#_bookmark56). The analyses were performed using
> SPSS 25 (IBM Corporation, Armonk, NY, USA), R-Statistics (R Foundation
> for Sta- tistical Computing [\[33\]](#_bookmark65)) and Python (Python
> Software Foundation. Python Language Reference, version 2.7). For all
> analyses, two- sided tests were performed and p-values \<0.05 were
> considered statistically significant.

3.  Results

> In comparison to the training cohort [\[1\]](#_bookmark56), the
> patients of the val- idation cohort had significantly larger primary
> tumour volumes (p = 0.003). The proportion of tumours arising from the
> oropharynx was much higher in the training cohort (50.6%) than in the
> valida- tion cohort (26.1%), while oral cavity tumours were more
> common in the validation cohort (32.6%) than in the training cohort
> (17.1%). The 2-year rates of LRC were similar between both cohorts
> (LRC: 62.6% vs 64.1%, p = 0.62), while, as a statistical trend, OS was
> higher in the training cohort (59.6% vs 53.2%, p = 0.084).
> Characteristics of both cohorts are compared and summarized in [Table
> 1](#_bookmark49). The corre- sponding Kaplan-Meier estimates are shown
> in Supplementary Fig. 1.
>
> Univariable Cox models were built on the validation cohort using the
> same parameters as were applied for the training cohort in
> [\[1\]](#_bookmark56) ([Table 2](#_bookmark50)). In contrast to the
> training cohort, the logarithmised primary tumour volume (training
> (t): HR = 1.44, p = 0.028; valida- tion (v): HR = 1.30, p = 0.24,
> [Fig. 2](#_bookmark51)C, D) could not be validated as a significant
> prognostic factor for LRC, while the total volume of pri- mary tumour
> and lymph nodes showed borderline significance for LRC in the
> validation set (t: HR = 1.57, p = 0.008; v: HR = 1.61, p = 0.050). N
> stage (t: HR = 1.86, p = 0.048; v: HR = 3.42,
>
> []{#_bookmark49 .anchor}Table 1
>
> Patient characteristics of the training and validation cohort. 95%
> confidence intervals (95% CI) are marked by ^\#^. The hypoxia
> signatures, marked by ^\*^, stratified the patients into groups with
> more and less hypoxic tumours based on the gene expression of the 15-,
> 26-, and 30-gene hypoxia-associated signatures
> [\[26--30\]](#_bookmark60). LN = lymph nodes; pts = patients.

+---+--------------------+------------+-----+------------+-----+------+
|   | Characteristics    | > Training |     | >          |     | >    |
|   |                    | > cohort   |     | Validation |     |  p-v |
|   |                    | > (158)    |     | > cohort   |     | alue |
|   |                    | > Median   |     | > (92)     |     |      |
|   |                    | > (range)  |     | > Median   |     |      |
|   |                    |            |     | > (range)  |     |      |
+===+====================+============+=====+============+=====+======+
|   | Follow-up (months) | > 54.4     |     | > 59.0     |     | > 0  |
|   |                    | > (10.9-   |     | > (7,7--   |     | .064 |
|   |                    | -81.1)^\#^ |     | 131.9)^\#^ |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Age (years)        | > 58.6     |     | > 56       |     | >    |
|   |                    | > (3       |     | > (3       |     | 0.30 |
|   |                    | 9.2--81.9) |     | 9.8--82.1) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Dose (Gy)          | > 72.0     |     | > 72.0     |     | > 0  |
|   |                    | > (6       |     | > (6       |     | .007 |
|   |                    | 8.4--74.0) |     | 9.0--76.6) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Volume Tumour      | > 26.8     |     | > 33.9     |     | > 0  |
|   | (cm^3^)            | > (4       |     | > (5       |     | .003 |
|   |                    | .4--175.8) |     | .1--322.6) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Volume LN (cm^3^)  | > 8.2      |     | > 7.2      |     | >    |
|   |                    | >          |     | >          |     | 0.94 |
|   |                    | (0--300.0) |     | (0--272.6) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Volume total       | > 41.0     |     | > 53.0     |     | > 0  |
|   | (Tumour + LN)      | > (5       |     | > (7       |     | .042 |
|   | (cm^3^)            | .6--351.7) |     | .9--344.7) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | *CD44*             | > 0.60     |     | > 0.38     |     | > 0  |
|   |                    | > (-0      |     | > (-1      |     | .008 |
|   |                    | .79--3.36) |     | .20--1.64) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | *SLC3A2*           | > !3.17    |     | > !2.56    |     | >    |
|   |                    | > (-5.8    |     | > (-4.1    |     | 0.83 |
|   |                    | 6-(-1.27)) |     | 9-(-1.26)) |     |      |
+---+--------------------+------------+-----+------------+-----+------+
|   |                    | > Number   | >   | > Number   | >   |      |
|   |                    | > of pts   | (%) | > of pts   | (%) |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | Gender Male        | > 133      | > 8 | > 76       | > 8 |      |
|   |                    |            | 4.2 |            | 2.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Female           | > 25       | > 1 | > 16       | > 1 | >    |
|   |                    |            | 5.8 |            | 7.4 | 0.75 |
+---+--------------------+------------+-----+------------+-----+------+
|   | Never smoker Yes   | > 21       | > 1 | > 7        | >   |      |
|   |                    |            | 3.3 |            | 7.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > No               | > 137      | > 8 | > 85       | > 9 | >    |
|   |                    |            | 6.7 |            | 2.4 | 0.17 |
+---+--------------------+------------+-----+------------+-----+------+
|   | Localization       | > 80       | > 5 | > 24       | > 2 |      |
|   | Oropharynx         |            | 0.6 |            | 6.1 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Oral cavity      | > 27       | > 1 | > 30       | > 3 |      |
|   |                    |            | 7.1 |            | 2.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Hypopharynx      | > 51       | > 3 | > 30       | > 3 |      |
|   |                    |            | 2.3 |            | 2.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > []{#_bookmark50  | > 0        | > 0 | > 8        | >   | >    |
|   | > .anchor}Larynx   |            |     |            | 8.7 |  \<0 |
|   |                    |            |     |            |     | .001 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > T stage 1        | > 0        | > 0 | > 1        | >   |      |
|   |                    |            |     |            | 1.1 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 2                | > 18       | > 1 | > 4        | >   |      |
|   |                    |            | 1.4 |            | 4.3 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 3                | > 41       | > 2 | > 22       | > 2 |      |
|   |                    |            | 5.9 |            | 3.9 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 4                | > 99       | > 6 | > 65       | > 7 | >    |
|   |                    |            | 2.7 |            | 0.7 | 0.13 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > N stage 0        | > 28       | > 1 | > 12       | > 1 |      |
|   |                    |            | 7.7 |            | 3.0 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 1                | > 7        | >   | > 7        | >   |      |
|   |                    |            | 4.4 |            | 7.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 2                | > 115      | > 7 | > 68       | > 7 |      |
|   |                    |            | 2.8 |            | 3.9 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 3                | > 8        | >   | > 5        | >   | >    |
|   |                    |            | 5.1 |            | 5.4 | 0.60 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Chemotherapy Yes | > 158      | >   | > 92       | >   |      |
|   |                    |            |  10 |            |  10 |      |
|   |                    |            | 0.0 |            | 0.0 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > No               | > 0        | > 0 |            |     | > -- |
+---+--------------------+------------+-----+------------+-----+------+
|   | > HPV16 DNA        | > 137      | > 8 | > 78       | > 8 |      |
|   | > Negative         |            | 6.7 |            | 9.7 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > status Positive  | > 20       | > 1 | > 9        | > 1 |      |
|   |                    |            | 2.7 |            | 0.3 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 1        | >   | > 5        | >   | >    |
|   |                    |            | 0.6 |            | 5.4 | 0.58 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > p16 protein      | > 125      | > 7 | > 80       | > 8 |      |
|   | > Negative         |            | 9.1 |            | 7.0 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Positive         | > 24       | > 1 | > 12       | > 1 |      |
|   |                    |            | 5.2 |            | 3.0 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 9        | >   | > 0        | > 0 | >    |
|   |                    |            | 5.7 |            |     | 0.52 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > CD44 protein     | > 28       | > 1 | > 5        | >   |      |
|   | > Negative         |            | 7.7 |            | 5.4 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Positive         | > 108      | > 6 | > 76       | > 8 |      |
|   |                    |            | 8.4 |            | 2.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 22       | > 1 | > 11       | > 1 | > 0  |
|   |                    |            | 3.9 |            | 2.0 | .004 |
+---+--------------------+------------+-----+------------+-----+------+
|   | > 15-gene hypoxia  | > 55       | > 3 | > 41       | > 4 |      |
|   | > Negative         |            | 4.8 |            | 4.6 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > signature^\*^    | > 83       | > 5 | > 51       | > 5 |      |
|   | > Positive         |            | 2.5 |            | 5.4 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 20       | > 1 | > 0        | > 0 | >    |
|   |                    |            | 2.7 |            |     | 0.48 |
+---+--------------------+------------+-----+------------+-----+------+
|   | 26-gene hypoxia    | > 47       | > 2 | > 24       | > 2 |      |
|   | Negative           |            | 9.7 |            | 6.1 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | signature^\*^      | > 91       | > 5 | > 68       | > 7 |      |
|   | Positive           |            | 7.6 |            | 3.9 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 20       | > 1 | > 0        | > 0 | >    |
|   |                    |            | 2.7 |            |     | 0.20 |
+---+--------------------+------------+-----+------------+-----+------+
|   | 30-gene hypoxia    | > 53       | > 3 | > 49       | > 5 |      |
|   | Negative           |            | 3.5 |            | 3.3 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | signature^\*^      | > 85       | > 5 | > 43       | > 4 |      |
|   | Positive           |            | 3.8 |            | 6.7 |      |
+---+--------------------+------------+-----+------------+-----+------+
|   | > Missing          | > 20       | > 1 | > 0        | > 0 | > 0  |
|   |                    |            | 2.7 |            |     | .026 |
+---+--------------------+------------+-----+------------+-----+------+

> Table 2
>
> Univariable Cox regression on the validation cohort. The hazard ratios
> (HR) and the corresponding 95% confidence interval (95% CI) are shown
> for the endpoints loco-regional tumour control and overall survival.
> The parameters which were significant in the training cohort are
> marked by a ^\*^. Not converging models are marked by ^y^. GTV = gross
> tumour volume; Ln = natural logarithm; LN = lymph nodes.

+--------------------+-------------+----+--------+----------+------+---+
| > Parameter        | Lo          |    |        | Overall  |      |   |
|                    | co-regional |    |        | survival |      |   |
|                    | tumour      |    |        |          |      |   |
|                    | control     |    |        |          |      |   |
+====================+=============+====+========+==========+======+===+
|                    | HR (95% CI) | >  |        | HR (95%  | >    |   |
|                    |             |  p |        | CI)      |  p-v |   |
|                    |             | -v |        |          | alue |   |
|                    |             | al |        |          |      |   |
|                    |             | ue |        |          |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > Oral cavity vs   | 1.36        | >  |        | 1.76     | > 0  |   |
| > others           | (           | 0. |        | (1.0     | .040 |   |
|                    | 0.64--2.92) | 43 |        | 2--3.03) |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > N stage (0,1 vs  | 3.63        | >  |        | 3.42     | > 0  |   |
| > 2,3) p16         | (1          |  0 |        | (1.4     | .005 |   |
|                    | .10--12.00) | .0 |        | 6--7.99) | ^\*^ |   |
|                    |             | 30 |        |          | >    |   |
|                    | 0.78        | >  |        | 0.85     | >    |   |
|                    | (           | >  |        | (0.3     | 0.71 |   |
|                    | 0.24--2.57) | 0. |        | 6--1.98) | ^\*^ |   |
|                    |             | 68 |        |          |      |   |
|                    |             | ^\ |        |          |      |   |
|                    |             | *^ |        |          |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > HPV16 DNA        | 1.13        | >  |        | 1.19     | >    |   |
|                    | (           | 0. |        | (0.5     | 0.68 |   |
|                    | 0.34--3.74) | 85 |        | 1--2.79) |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > Ln(GTV)          | 1.30        | >  |        | 1.53     | > 0  |   |
|                    | (           | 0. |        | (1.1     | .009 |   |
|                    | 0.84--2.03) | 24 |        | 1--2.10) |      |   |
|                    |             | ^\ |        |          |      |   |
|                    |             | *^ |        |          |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > Ln(LN)           | 1.38        | >  |        | 1.21     | > 0  |   |
|                    | (           |  0 |        | (1.0     | .040 |   |
|                    | 1.06--1.80) | .0 |        | 1--1.46) |      |   |
|                    |             | 20 |        |          |      |   |
+--------------------+-------------+----+--------+----------+------+---+
| > Ln(GTVtot)       | 1.61        | >  |        | 1.75     | > 0  |   |
|                    | (           |  0 |        | (1.2     | .001 |   |
|                    | 1.00--2.60) | .0 |        | 5--2.46) |      |   |
|                    |             | 50 |        |          |      |   |
|                    |             | ^\ |        |          |      |   |
|                    |             | *^ |        |          |      |   |
+--------------------+-------------+----+--------+----------+------+---+

> CD44 protein --^y^ --^\*^ 2.24 (0.54--9.25) 0.26^\*^

+----------------+---------------+-----------+---------------+--------+
| > *CD44*       | > 1.30        | > 0.38    | 1.80          | 0.0    |
|                | >             |           | (1.16--2.79)  | 09^\*^ |
|                |  (0.73--2.33) |           |               |        |
+================+===============+===========+===============+========+
| > *MET*        | > 1.05        | > 0.83    | 0.85          | 0.35   |
|                | >             |           | (0.60--1.20)  |        |
|                |  (0.66--1.68) |           |               |        |
+----------------+---------------+-----------+---------------+--------+
| > *SLC3A2*     | > 1.29        | >         | 1.41          | 0.17   |
|                | > (0.66--2.5) |  0.46^\*^ | (0.86--2.31)  |        |
+----------------+---------------+-----------+---------------+--------+
| > 15-gene      | > 1.89        | > 0.09    | 2.06          | 0.010  |
| > hypoxia      | >             |           | (1.19--3.58)  |        |
| > signature    |  (0.90--3.99) |           |               |        |
+----------------+---------------+-----------+---------------+--------+
| > 26-gene      | > 1.56        | > 0.33    | 1.13          | 0.69   |
| > hypoxia      | >             |           | (0.62--2.07)  |        |
| > signature    |  (0.64--3.81) |           |               |        |
+----------------+---------------+-----------+---------------+--------+
| > 30-gene      | > 1.68        | > 0.16    | 1.18          | 0.55   |
| > hypoxia      | >             |           | (0.70--1.99)  |        |
| > signature    |  (0.82--3.47) |           |               |        |
+----------------+---------------+-----------+---------------+--------+

> ![](media/image57.jpeg){width="5.067910104986876in"
> height="6.413333333333333in"}
>
> []{#_bookmark51 .anchor}Fig. 2. Kaplan-Meier estimates for
> loco-regional tumour control in the training cohort (A, C) and the
> validation cohort (B, D) stratified based on the CD44 protein status
> (A, B) and the tumour volume (C, D). The p-values are based on
> log-rank tests.
>
> p = 0.005), the primary tumour volume (t: HR = 1.63, p = 0.001; v: HR
> = 1.53, p = 0.009) and the gene expression of *CD44* (t: HR = 1.81, p
> = 0.006; v: HR = 1.80p = 0.009) were significantly asso- ciated with
> the secondary endpoint OS. Only 5 patients with a CD44 protein
> negative tumour were included in the validation cohort. None of them
> showed a loco-regional recurrence ([Fig. 2](#_bookmark51)A, B). While
> less hypoxic tumours showed higher LRC for all hypoxia-associated gene
> signatures (Supplementary Fig. 2), these differences were not
> significant.
>
> Similar results were obtained for the validation of the multi-
> variable Cox models ([Table 3](#_bookmark52)). The baseline model
> containing the logarithmised primary tumour volume, p16 status and N
> stage was derived from our multicentre study [\[1\]](#_bookmark56). It
> showed a validation ci of 0.59 (95% confidence interval (CI):
> 0.49--0.70) for LRC and of
>
> 0.63 (0.55--0.71) for OS, which was only slightly lower than the
> results on the training cohort. The performance of these baseline
>
> models could be slightly improved on the validation cohort by
> including the expression of *CD44* (LRC: 0.61 (0.50--0.72); OS: 0.69
> (0.61--0.75)), CD44 protein (LRC: 0.62 (0.50--0.72); OS: 0.65
>
> (0.56--0.73)) and *SLC3A2* (OS: 0.65 (0.57--0.73)). None of the
>
> hypoxia-associated gene signatures could improve the perfor- mance of
> the baseline model, which is similar to the training cohort.
>
> Two logistic regression models were developed previously to predict
> 2-year LRC [\[1\]](#_bookmark56). The first univariable regression
> included the primary tumour volume only, while the second
> multivariable regression combined the primary tumour volume with p16
> and CD44 protein status. The AUC of the univariable and multivariable
> regressions on the training cohort were 0.65 and 0.73, respectively.
> The same models were applied to the validation cohort leading to AUC =
> 0.58 (0.43--0.74) and to AUC = 0.64 (0.49--0.79), respectively.
>
> []{#_bookmark52 .anchor}Table 3
>
> Training and validation of different multivariable Cox models for the
> endpoints loco-regional tumour control and overall survival. The
> concordance index (ci) and its 95% confidence interval (95% CI) are
> shown for the trained models and their independent validation. Bold
> values present two sided p-values \<0.05 which were considered
> statistically significant. The baseline model (BL) consisting of N
> stage, p16 status and the logarithmised primary tumour volume (lnGTV)
> is supplemented by the additional putative CSC markers (*CD44* gene or
> CD44 protein; *SLC3A2*) or hypoxia classifiers based on the 15- and
> 30-gene hypoxia-associated signatures [\[26,27,30\]](#_bookmark60).

+---------------------------+---------+-----+------+---------+-----+---+
|                           | T       |     |      | Val     |     |   |
|                           | raining |     |      | idation |     |   |
|                           | cohort  |     |      | cohort  |     |   |
+===========================+=========+=====+======+=========+=====+===+
|                           | ci (95% | p   |      | ci (95% | > p |   |
|                           | CI)     | -va |      | CI)     | -va |   |
|                           |         | lue |      |         | lue |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > *Loco-regional tumour   |         |     |      |         |     |   |
| > control*                |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > Baseline (BL): N stage, | 0.64    | \<0 |      | 0.59    | 0   |   |
| > p16, lnGTV              | (0.56   | .01 |      | (0.49   | .09 |   |
|                           | --0.71) |     |      | --0.70) |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, CD44                | 0.66    | \<0 |      | 0.62    | 0.  |   |
|                           | (0.59   | .01 |      | (0.50   | 046 |   |
|                           | --0.75) |     |      | --0.73) |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, *CD44*              | 0.64    | \<0 |      | 0.61    | 0.  |   |
|                           | (0.58   | .01 |      | (0.50   | 046 |   |
|                           | --0.72) |     |      | --0.72) |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, *SLC3A2*            | 0.65    | \<0 |      | 0.58    | 0   |   |
|                           | (0.59   | .01 |      | (0.49   | .09 |   |
|                           | --0.73) |     |      | --0.67) |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 15-gene hypoxia     | 0.63    | \<0 |      | 0.54    | 0   |   |
| > signature,              | (0.58   | .01 |      | (0.43   | .50 |   |
|                           | --0.73) |     |      | --0.65) |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 15-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV      |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 30-gene hypoxia     | 0.62    | \<0 | >    |         | > 0 |   |
| > signature,              | (0.58   | .01 | 0.59 |         | .12 |   |
|                           | --0.73) |     | >    |         |     |   |
|                           |         |     | (0.4 |         |     |   |
|                           |         |     | 8--0 |         |     |   |
|                           |         |     | .69) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 30-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV      |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 15-gene hypoxia     | 0.66    | \<0 | >    |         | > 0 |   |
| > signature,              | (0.62   | .01 | 0.56 |         | .29 |   |
|                           | --0.75) |     | >    |         |     |   |
|                           |         |     | (0.4 |         |     |   |
|                           |         |     | 5--0 |         |     |   |
|                           |         |     | .66) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| 15-gene hypoxia signature |         |     |      |         |     |   |
| \* lnGTV, *SLC3A2*        |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 30-gene hypoxia     | 0.66    | \<0 | >    |         | > 0 |   |
| > signature,              | (0.61   | .01 | 0.60 |         | .07 |   |
|                           | --0.75) |     | >    |         |     |   |
|                           |         |     | (0.4 |         |     |   |
|                           |         |     | 9--0 |         |     |   |
|                           |         |     | .70) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| 30-gene hypoxia signature |         |     |      |         |     |   |
| \* lnGTV, *SLC3A2*        |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > *Overall survival*      |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > Baseline (BL): N stage, | 0.68    | \<0 | >    |         | >   |   |
| > p16, lnGTV              | (0.62   | .01 | 0.63 |         | \<0 |   |
|                           | --0.75) |     | >    |         | .01 |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 5--0 |         |     |   |
|                           |         |     | .71) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, CD44                | 0.71    | \<0 | >    |         | >   |   |
|                           | (0.65   | .01 | 0.65 |         | \<0 |   |
|                           | --0.78) |     | >    |         | .01 |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 6--0 |         |     |   |
|                           |         |     | .73) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, *CD44*              | 0.68    | \<0 | >    |         | >   |   |
|                           | (0.62   | .01 | 0.69 |         | \<0 |   |
|                           | --0.75) |     | >    |         | .01 |   |
|                           |         |     | (0.6 |         |     |   |
|                           |         |     | 1--0 |         |     |   |
|                           |         |     | .76) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, *SLC3A2*            | 0.67    | \<0 | >    |         | >   |   |
|                           | (0.62   | .01 | 0.65 |         | \<0 |   |
|                           | --0.74) |     | >    |         | .01 |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 7--0 |         |     |   |
|                           |         |     | .73) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 15-gene hypoxia     | 0.68    | \<0 | >    |         | > 0 |   |
| > signature,              | (0.62   | .01 | 0.60 |         | .02 |   |
|                           | --0.75) |     | >    |         |     |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 2--0 |         |     |   |
|                           |         |     | .68) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 15-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV      |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 30-gene hypoxia     | 0.69    | \<0 | >    |         | >   |   |
| > signature,              | (0.63   | .01 | 0.63 |         | \<0 |   |
|                           | --0.75) |     | >    |         | .01 |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 4--0 |         |     |   |
|                           |         |     | .71) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 30-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV      |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 15-gene hypoxia     | 0.68    | \<0 | >    |         | >   |   |
| > signature,              | (0.63   | .01 | 0.66 |         | \<0 |   |
|                           | --0.76) |     | >    |         | .01 |   |
|                           |         |     | (0.5 |         |     |   |
|                           |         |     | 8--0 |         |     |   |
|                           |         |     | .74) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 15-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV,     |         |     |      |         |     |   |
| > *CD44*                  |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > BL, 30-gene hypoxia     | 0.69    | \<0 | >    |         | >   |   |
| > signature,              | (0.64   | .01 | 0.70 |         | \<0 |   |
|                           | --0.76) |     | >    |         | .01 |   |
|                           |         |     | (0.6 |         |     |   |
|                           |         |     | 1--0 |         |     |   |
|                           |         |     | .77) |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+
| > 30-gene hypoxia         |         |     |      |         |     |   |
| > signature \* lnGTV,     |         |     |      |         |     |   |
| > *CD44*                  |         |     |      |         |     |   |
+---------------------------+---------+-----+------+---------+-----+---+

![](media/image58.jpeg){width="5.913172572178477in"
height="4.3666655730533686in"}

> []{#_bookmark53 .anchor}Fig. 3. Logistic regression regarding 2-year
> loco-regional tumour control. The results of the training cohort (A,
> C) and the validation cohort (B, D) are shown. A and B show the
> univariable logistic regression solely based on the primary tumour
> volume, while C and D show the multivariable logistic regression,
> which was additionally based on the p16 and CD44 protein status. Since
> none of the patients within the validation cohort presented with a p16
> positive and simultaneous CD44 negative tumour, the regression for the
> corresponding model is not shown in D.
>
> The logistic regressions are shown in [Fig. 3](#_bookmark53) for the
> training and the validation cohort.

4.  Discussion

> In our previous study we demonstrated the prognostic value of the
> primary tumour volume, the potential CSC markers CD44 pro- tein,
> *CD44* and *SLC3A2* gene expression as well as hypoxia- associated
> gene signatures [\[27,31\]](#_bookmark61) for patients with locally
> advanced HNSCC who were treated with primary RCTx. In the cur- rent
> study, an independent cohort including 92 patients with locally
> advanced HNSCC, who received primary RCTx between 1999 and 2015 was
> used to validate these results. The inclusion of the putative CSC
> marker *CD44* and CD44 protein resulted in slight improvements of the
> prognostic value of the baseline model containing primary tumour
> volume, p16 status and N stage for the endpoints LRC and OS. The
> logistic regression model for 2-year LRC based on the primary tumour
> volume could be improved by including the p16 status and the CD44
> protein status, which is in line with the previous training cohort
> [\[1\]](#_bookmark56).
>
> The volume of the primary tumour is a widely accepted prog- nostic
> biomarker for the outcome of radiotherapy in patients with HNSCC
> [\[20,22,34\]](#_bookmark59) and showed a significant correlation to
> LRC in the training cohort. In validation, the affected lymph node
> volume was significantly related to LRC, while the volume of the
> primary tumour showed no significant association. Since many patients
> presented with large tumours, the distinction between primary tumour
> and lymph nodes may be difficult, which may also lead to differences
> in delineation between radiation oncologists and thereby to
> differences in the dose prescription in the irradiated vol- ume,
> respectively. Considering the total volume of both primary tumour and
> lymph nodes, a significant association to LRC was obtained for the
> training cohort and a statistical trend (p = 0.050) was shown for the
> validation cohort, underlining the importance of the tumour volume as
> a biomarker.
>
> In contrast to the training cohort, the p16 status was not signif-
> icantly related to LRC in the validation cohort, which may be
> explained by a substantially smaller proportion of oropharyngeal
> tumours in the latter cohort (26% in the validation cohort vs. 51% in
> the training cohort) and by the cohort size as such. In the valida-
> tion cohort, 5 out of 24 patients with oropharyngeal tumours pre-
> sented with a p16 positive tumour with 2 of them being also positive
> for HPV16 DNA. In contrast, the training cohort included
>
> 16 patients with p16-positive oropharyngeal tumours with 11 being also
> positively tested for HPV16 DNA. Thus, the numbers of HPV-driven
> positive oropharyngeal tumours, as characterized by the simultaneous
> positivity for p16 and HPV16 DNA [\[35\]](#_bookmark67), were very
> low.
>
> In the validation cohort, CD44 protein slightly improved the
> performance of the multivariable Cox models for LRC and OS, and it
> improved the logistic regression model for the prognosis of 2- year
> LRC. Furthermore, the only two Cox models that could be suc- cessfully
> validated for LRC (p \< 0.05) were the models including CD44, either
> at its protein or at its gene expression level. This underlines the
> importance of CD44 for outcome prediction of HNSCC after
> radio(chemo)therapy and is in line with earlier find- ings, e.g. it
> was shown that the expression of *CD44* and the CD44 protein obtained
> from pre-treatment biopsies of laryngeal cancer patients were
> correlated with the endpoint local recurrence
> [\[36,37\]](#_bookmark68).
>
> In the training cohort, the impact of the hypoxia-associated gene
> signatures on LRC was significant for tumours smaller than 19 cm^3^
> [\[1\]](#_bookmark56). For the complete cohort, significant patient
> stratifica- tions based on the hypoxia status were not achieved, even
> though less hypoxic tumours showed a slightly better outcome
> [\[1\]](#_bookmark56). The
>
> results obtained in the validation cohort, comparing subgroups with
> more or less hypoxic tumours, were similar to the training cohort for
> all three considered gene-signatures (Supplementary Fig. 2). Due to
> the smaller validation cohort and significantly larger tumours,
> however, the power was not sufficient to allow for signif- icant
> stratifications. In addition, the ratio of more and less hypoxic
> tumours, identified by the 15- and 26-gene signatures, were simi- lar
> in training and validation, indicating that hypoxia classifica- tions
> by these gene signatures may be considered as robust biomarkers. The
> evaluation of the prognostic impact of hypoxia- associated gene
> expressions on LRC requires a larger cohort of patients undergoing
> primary radiochemotherapy, and will be per- formed in the prospective
> HNprädBio study of the DKTK-ROG, which has completed patient accrual.
>
> The prognostic performance of the multivariable Cox models defined in
> [\[1\]](#_bookmark56) was slightly higher in training than in
> validation, with a ci between 0.63 and 0.74 in training and 0.56--0.68
> in vali- dation. Interestingly, Cox models including the most features
> (6 features) showed a lower ci in validation than in training. This
> may be explained by overfitting, which occurs if the number of
> parameters in a multivariable model is too large, compared to the
> number of events [\[32\]](#_bookmark64).

5.  Conclusions

> In this study an independent validation of earlier identified
> potential biomarkers for treatment outcome of patients with locally
> advanced HNSCC treated with primary RCTx was per- formed. The
> importance of the clinical parameters tumour vol- ume and N stage, as
> well as the putative CSC markers CD44 and *SLC3A2* could be confirmed
> for OS as well as the importance of CD44 for LRC. However, the
> clinical utility of the observed slight improvements in prognostic
> power in this study has yet to be addressed in large data sets. While
> the stratification based on the hypoxia status was similar as in
> training, a significant impact on LRC and OS was not observed. Since
> several limitations are associated with the retrospective nature of
> this study, further prospective validation is required to assess the
> clinical relevance of the biomarkers. This will be performed in the
> currently ongo- ing prospective multicentre HNprädBio study
> ([www.clinicaltri-](http://www.clinicaltrials.gov/)
> [als.gov](http://www.clinicaltrials.gov/); NCT02059668) within the
> DKTK-ROG, which completed patient accrual in 2018.
>
> Conflict of interest
>
> Volker Gudziol is a member of the advisory board of Bristol, Myers
> Squibb and received speaking fees from Roche Company.
>
> In the past 5 years, Dr. Baumann attended an advisory board meeting of
> MERCK KGaA (Darmstadt), for which the University of Dresden received a
> travel grant. He further received funding for his research projects
> and for educational grants to the University of Dresden by Teutopharma
> GmbH (2011--2015), IBA (2016), Bayer AG (2016--2018), Merck KGaA
> (2016--2030), Medipan GmbH (2014--2018).
>
> Dr. Baumann, as former chair of OncoRay (Dresden) and present CEO and
> Scientific Chair of the German Cancer Research Center (DKFZ,
> Heidelberg), signed/s contracts for his institute(s) and for the staff
> for research funding and collaborations with a multitude of companies
> worldwide.
>
> For the German Cancer Research Center (DKFZ, Heidelberg) Dr. Baumann
> is on the supervisory boards of HI-STEM gGmbH (Heidelberg).
>
> For the present study, Dr. Baumann confirms that none of the above
> mentioned funding sources were involved in the
>
> [[[[[]{#_bookmark59 .anchor}]{#_bookmark58 .anchor}]{#_bookmark57
> .anchor}]{#_bookmark56 .anchor}]{#_bookmark54 .anchor}study design or
> materials used, nor in the collection, analysis []{#_bookmark55
> .anchor}and interpretation of data nor in the writing of the paper.
>
> The other authors have nothing to disclose.
>
> []{#_bookmark60 .anchor}Acknowledgements
>
> The authors gratefully acknowledge the excellent technical
> []{#_bookmark61 .anchor}assistance by Mrs. Liane Stolz-Kieslich, Mrs.
> Sigrid Balschukat and Mrs. Daniela Friede. The authors also wish to
> thank the Tumour and Normal Tissue Bank of the University Cancer
> Center of the University Hospital Dresden (TNTB, Prof. Dr. Daniela
> Aust) and the external pathologist Dr. Uwe Sturm (Dresden) for provid-
> ing the respective FFPE material.
>
> Appendix A. Supplementary data
>
> Supplementary data to this article can be found online at
> <https://doi.org/10.1016/j.ctro.2019.03.002>.
>
> References

1.  []{#_bookmark64 .anchor}Linge A, Lohaus F, Löck S, Nowak A, Gudziol
    V, Valentini C, et al. HPV status, []{#_bookmark62 .anchor}cancer
    stem cell marker expression, hypoxia gene signatures and tumour
    []{#_bookmark65 .anchor}volume identify good prognosis subgroups in
    patients with HNSCC after []{#_bookmark63 .anchor}primary
    radiochemotherapy: a multicentre retrospective study of the German
    Cancer Consortium Radiation. Radiother Oncol 2016;121:364--73.
    [[https://doi]{.underline}.](https://doi.org/10.1016/j.radonc.2016.11.008)
    [[org/10.1016/j.radonc.2016.11.008]{.underline}](https://doi.org/10.1016/j.radonc.2016.11.008).

2.  Duray A, Descamps G, Decaestecker C, Remmelink M, Sirtaine N,
    Lechien J, et al. Human papillomavirus DNA strongly correlates with
    a poorer prognosis in oral cavity carcinoma. Laryngoscope
    2012;122:1558--65.
    [[https://doi.org/]{.underline}](https://doi.org/10.1002/lary.23298)
    [[]{#_bookmark67 .anchor}]{#_bookmark66
    .anchor}[[10.1002/lary.23298]{.underline}](https://doi.org/10.1002/lary.23298).

3.  Serrano E, Daly-schveitzer N, Bachaud J, David J. Combined
    postoperative radiotherapy and weekly cisplatin infusion for locally
    advanced head and neck carcinoma: final report of a randomized
    trial. Int J Radiat Oncol Biol Phys 1996;36:999--1004.
    [[https://doi.org/10.1016/S0360-3016(96)00430-0]{.underline}](https://doi.org/10.1016/S0360-3016(96)00430-0).

4.  Al-Sarraf M, Pajak TF, Byhardt RW, Beitler JJ, Salter MM, Cooper JS.
    []{#_bookmark68 .anchor}Postoperative radiotherapy with concurrent
    cisplatin appears to improve locoregional control of advanced,
    resectable head and neck cancers: RTOG 88-

> 24\. Int J Radiat Oncol Biol Phys 1997;37:777--82.
> [[https://doi.org/10.1016/]{.underline}](https://doi.org/10.1016/S0360-3016(96)00614-1)
> [[S0360-3016(96)00614-1]{.underline}](https://doi.org/10.1016/S0360-3016(96)00614-1).

5.  Jeremic B, Shibamoto Y, Milicic B, Nikolic N, Dagovic A,
    Aleksandrovic J, et al. Hyperfractionated radiation therapy with or
    without concurrent low-dose daily cisplatin in locally advanced
    squamous cell carcinoma of the head and neck: a prospective
    randomized trial. J Clin Oncol 2000;18:1458--64.
    [[https://]{.underline}](https://doi.org/10.1200/JCO.2000.18.7.1458)
    [[doi.org/10.1200/JCO.2000.18.7.1458]{.underline}](https://doi.org/10.1200/JCO.2000.18.7.1458).

6.  Adelstein DJ, Li Y, Adams GL, Wagner H, Kish JA, Ensley JF, et al.
    An Intergroup phase III comparison of standard radiation therapy and
    two schedules of concurrent chemoradiotherapy in patients with
    unresectable squamous cell head and neck cancer. J Clin Oncol
    2003;21:92--8.
    [[https://doi.org/10.1200/]{.underline}](https://doi.org/10.1200/JCO.2003.01.008)
    [[JCO.2003.01.008]{.underline}](https://doi.org/10.1200/JCO.2003.01.008).

7.  Huguenin P, Beer KT, Allal A, Rufibach K, Friedli C, Davis JB, et
    al. Concomitant cisplatin significantly improves locoregional
    control in advanced head and neck cancers treated with
    hyperfractionated radiotherapy. J Clin Oncol 2004;22:4665--73.
    [[https://doi.org/10.1200/JCO.2004.12.193]{.underline}](https://doi.org/10.1200/JCO.2004.12.193).

8.  Bonner JA, Harari PM, Giralt J, Cohen RB, Jones CU, Sur RK, et al.
    Radiotherapy plus cetuximab for locoregionally advanced head and
    neck cancer: 5-year survival data from a phase 3 randomised trial,
    and relation between cetuximab-induced rash and survival. Lancet
    Oncol 2010;11:21--8.
    [[https://]{.underline}](https://doi.org/10.1016/S1470-2045(09)70311-0)
    [[doi.org/10.1016/S1470-2045(09)70311-0]{.underline}](https://doi.org/10.1016/S1470-2045(09)70311-0).

9.  Leemans CR, Braakhuis BJM, Brakenhoff RH. The molecular biology of
    head and neck cancer. Nat Rev Cancer 2011;11:9--22.
    [[https://doi.org/10.1038/nrc2982]{.underline}](https://doi.org/10.1038/nrc2982).

10. Baumann M, Krause M, Hill R. Exploring the role of cancer stem cells
    in radioresistance. Nat Rev Cancer 2008;8:545--54.
    [[https://doi.org/10.1038/]{.underline}](https://doi.org/10.1038/nrc2419)
    [[nrc2419]{.underline}](https://doi.org/10.1038/nrc2419).

11. Baumann M, Krause M, Overgaard J, Debus J, Bentzen SM, Daartz J, et
    al. Radiation oncology in the era of precision medicine. Nat Rev
    Cancer 2016;16:234--49.
    [[https://doi.org/10.1038/nrc.2016.18]{.underline}](https://doi.org/10.1038/nrc.2016.18).

12. [Gillison ML, Koch WM, Shah KV. Human papillomavirus in head and
    neck](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0060)
    [squamous cell carcinoma: are some head and neck cancers a
    sexually](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0060)
    [transmitted disease? Curr Opin Oncol
    1999;11](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0060).

13. [Gillison ML, Koch WM, Capone RB, Spafford M, Westra WH, Wu L, et
    al.](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0065)
    [Evidence for a causal association between human papillomavirus and
    a subset](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0065)
    [of head and neck cancers. J Natl Cancer Inst
    2000;92:709--20](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0065).

14. [Braakhuis BJM, Snijders PJF, Keune W-JH, Meijer CJLM,
    Ruijter-Schippers
    HJ,](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0070)
    [Leemans CR, et al. Genetic patterns in head and neck cancers that
    contain or](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0070)
    [lack transcriptionally active human papillomavirus. J Natl Cancer
    Inst](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0070)
    [2004;96:998--1006](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0070).

15. [Lindel K, Beer KT, Laissue J, Greiner RH, Aebersold DM. Human
    papillomavirus](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0075)
    [positive squamous cell carcinoma of the oropharynx: a
    radiosensitive](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0075)

> [subgroup of head and neck carcinoma. Cancer 2001;92:805--13.
> doi:](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0075)
> [10.1002/1097-0142(20010815)92:4\<805::AID-CNCR1386\>3.0.CO;2-9](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0075).

16. Lassen P, Eriksen JG, Hamilton-Dutoit S, Tramm T, Alsner J,
    Overgaard J. HPV- associated p16-expression and response to hypoxic
    modification of radiotherapy in head and neck cancer. Radiother
    Oncol 2010;94:30--5.
    [[https://doi.org/10.1016/j.radonc.2009.10.008]{.underline}](https://doi.org/10.1016/j.radonc.2009.10.008).

17. Lassen P, Eriksen JG, Krogdahl A, Therkildsen MH, Ulhøi BP,
    Overgaard M, et al. The influence of HPV-associated p16-expression
    on accelerated fractionated radiotherapy in head and neck cancer:
    evaluation of the randomised DAHANCA 6&7 trial. Radiother Oncol
    2011;100:49--55.
    [[https://doi.org/]{.underline}](https://doi.org/10.1016/j.radonc.2011.02.010)
    [[10.1016/j.radonc.2011.02.010]{.underline}](https://doi.org/10.1016/j.radonc.2011.02.010).

18. Tehrany N, Kitz J, Rave-Fränk M, Lorenzen S, Li L, Küffer S, et al.
    High-grade acute organ toxicity and p16INK4A expression as positive
    prognostic factors in primary radio(chemo)therapy for patients with
    head and neck squamous cell carcinoma. Strahlentherapie Und Onkol
    2015;191:566--72.
    [[https://doi.org/]{.underline}](https://doi.org/10.1007/s00066-014-0801-3)
    [[10.1007/s00066-014-0801-3]{.underline}](https://doi.org/10.1007/s00066-014-0801-3).

19. Bentzen SM, Thames HD. Tumor volume and local control probability:
    clinical data and radiobiological interpretations. Int J Radiat
    Oncol 1996;36:247--51.
    [[https://doi.org/10.1016/S0360-3016(96)00242-8]{.underline}](https://doi.org/10.1016/S0360-3016(96)00242-8).

20. Dubben H-H, Thames HD, Beck-Bornholdt H-P. Tumor volume: a basic and
    specific response predictor in radiotherapy. Radiother Oncol
    1998;47:167--74.
    [[https://doi.org/10.1016/S0167-8140(97)00215-6]{.underline}](https://doi.org/10.1016/S0167-8140(97)00215-6).

21. Jensen AR, Nellemann HM, Overgaard J. Tumor progression in waiting
    time for radiotherapy in head and neck cancer. Radiother Oncol
    2007;84:5--10.
    [[https://]{.underline}](https://doi.org/10.1016/j.radonc.2007.04.001)
    [[doi.org/10.1016/j.radonc.2007.04.001]{.underline}](https://doi.org/10.1016/j.radonc.2007.04.001).

22. Krause M, Dubrovska A, Linge A, Baumann M. Cancer stem cells:
    radioresistance, prediction of radiotherapy outcome and specific
    targets for combined treatments. Adv Drug Deliv Rev 2017;109:63--73.
    [[https://doi.org/]{.underline}](https://doi.org/10.1016/j.addr.2016.02.002)
    [[10.1016/j.addr.2016.02.002]{.underline}](https://doi.org/10.1016/j.addr.2016.02.002).

23. Zwanenburg A, Löck S. Why validation of prognostic models matters?
    Radiother Oncol 2018;127:370--3.
    [[https://doi.org/10.1016/j]{.underline}.](https://doi.org/10.1016/j.radonc.2018.03.004)
    [[radonc.2018.03.004]{.underline}](https://doi.org/10.1016/j.radonc.2018.03.004).

24. Zips D, Zöphel K, Abolmaali N, Perrin R, Abramyuk A, Haase R, et al.
    Exploratory prospective trial of hypoxia-specific PET imaging during
    radiochemotherapy in patients with locally advanced head-and-neck
    cancer. Radiother Oncol 2012;105:21--8.
    [[https://doi.org/10.1016/j]{.underline}.](https://doi.org/10.1016/j.radonc.2012.08.019)
    [[radonc.2012.08.019]{.underline}](https://doi.org/10.1016/j.radonc.2012.08.019).

25. Löck S, Perrin R, Seidlitz A, Bandurska-Luque A, Zschaeck S, Zöphel
    K, et al. Residual tumour hypoxia in head-and-neck cancer patients
    undergoing primary radiochemotherapy, final results of a prospective
    trial on repeat FMISO-PET imaging. Radiother Oncol J Eur Soc Ther
    Radiol Oncol 2017.
    [[https://]{.underline}](https://doi.org/10.1016/j.radonc.2017.08.010)
    [[doi.org/10.1016/j.radonc.2017.08.010]{.underline}](https://doi.org/10.1016/j.radonc.2017.08.010).

26. Lassen P, Eriksen JG, Hamilton-Dutoit S, Tramm T, Alsner J,
    Overgaard J. Effect of HPV-associated p16INK4A expression on
    response to radiotherapy and survival in squamous cell carcinoma of
    the head and neck. J Clin Oncol 2009;27:1992--8.
    [[https://doi.org/10.1200/JCO.2008.20.2853]{.underline}](https://doi.org/10.1200/JCO.2008.20.2853).

27. Toustrup K, Sørensen BS, Nordsmark M, Busk M, Wiuf C, Alsner J, et
    al. Development of a hypoxia gene expression classifier with
    predictive impact for hypoxic modification of radiotherapy in head
    and neck cancer. Cancer Res 2011;71:5923--31.
    [[https://doi.org/10.1158/0008-5472.CAN-11-1182]{.underline}](https://doi.org/10.1158/0008-5472.CAN-11-1182).

28. Toustrup K, Sørensen BS, Metwally MAH, Tramm T, Mortensen LS,
    Overgaard J, et al. Validation of a 15-gene hypoxia classifier in
    head and neck cancer for prospective use in clinical trials. Acta
    Oncol 2016:1--8.
    [[https://doi.org/]{.underline}](https://doi.org/10.3109/0284186X.2016.1167959)
    [[10.3109/0284186X.2016.1167959]{.underline}](https://doi.org/10.3109/0284186X.2016.1167959).

29. Buffa FM, Harris AL, West CM, Miller CJ. Large meta-analysis of
    multiple cancers reveals a common, compact and highly prognostic
    hypoxia metagene. Br J Cancer 2010;102:428--35.
    [[https://doi.org/10.1038/sj.bjc.6605450]{.underline}](https://doi.org/10.1038/sj.bjc.6605450).

30. Eustace A, Mani N, Span PN, Irlam JJ, Taylor J, Betts GNJ, et al. A
    26-gene hypoxia signature predicts benefit from hypoxia-modifying
    therapy in laryngeal cancer but not bladder. Cancer 2013;9:19--21.
    [[https://doi.org/]{.underline}](https://doi.org/10.1158/1078-0432.CCR-13-0542)
    [[10.1158/1078-0432.CCR-13-0542]{.underline}](https://doi.org/10.1158/1078-0432.CCR-13-0542).

31. Lendahl U, Lee KL, Yang H, Poellinger L. Generating specificity and
    diversity in the transcriptional response to hypoxia. Nat Rev Genet
    2009;10:821--32.
    [[https://doi.org/10.1038/nrg2665]{.underline}](https://doi.org/10.1038/nrg2665).

32. [James G, Witten D, Hastie T, Tibshirani R. An introduction to
    statistical](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0160)
    [learning: with applications in R. 1st ed. New York: Springer; 2013.
    10.1007/](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0160)
    [978-1-4614-7138-7](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0160).

33. [Development Core R, Team R. A Language and Environment. for
    Statistical](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0165)
    [Computing;
    2008](http://refhub.elsevier.com/S2405-6308(18)30115-0/h0165).

34. Soliman M, Yaromina A, Appold S, Zips D, Reiffenstuhl C, Schreiber
    A, et al. GTV differentially impacts locoregional control of
    non-small cell lung cancer (NSCLC) after different fractionation
    schedules: subgroup analysis of the prospective randomized CHARTWEL
    trial. Radiother Oncol 2013;106:299--304.
    [[https://doi.org/10.1016/j.radonc.2012.12.008]{.underline}](https://doi.org/10.1016/j.radonc.2012.12.008).

35. Linge A, Schötz U, Löck S, Lohaus F, von Neubeck C, Gudziol V, et
    al. Comparison of detection methods for HPV status as a prognostic
    marker for loco-regional control after radiochemotherapy in patients
    with HNSCC. Radiother Oncol 2018;127:27--35.
    [[https://doi.org/10.1016/j]{.underline}.](https://doi.org/10.1016/j.radonc.2017.12.007)
    [[radonc.2017.12.007]{.underline}](https://doi.org/10.1016/j.radonc.2017.12.007).

36. de Jong MC, Pramana J, van der Wal JE, Lacko M, Peutz-Kootstra CJ,
    de Jong JM, et al. CD44 expression predicts local recurrence after
    radiotherapy in larynx cancer. Clin Cancer Res 2010;16:5329--38.
    [[https://doi.org/10.1158/1078-0432]{.underline}.](https://doi.org/10.1158/1078-0432.CCR-10-0799)
    [[CCR-10-0799]{.underline}](https://doi.org/10.1158/1078-0432.CCR-10-0799).

37. Baumann M, Krause M. CD44: a cancer stem cell-related biomarker with
    predictive potential for radiotherapy. Clin Cancer Res
    2010;16:5091--3.
    [[https://doi.org/10.1158/1078-0432.CCR-10-2244]{.underline}](https://doi.org/10.1158/1078-0432.CCR-10-2244).

> ![](media/image1.jpeg)
> ![](media/image59.png){width="0.7896095800524935in"
> height="0.9895833333333334in"}
>
> HPV in postoperative RT of oropharynx

## ![](media/image60.png)HPV16 DNA status is a strong prognosticator of loco-regional control after postoperative radiochemotherapy of locally advanced oropharyngeal carcinoma: Results from a multicentre explorative study of the German Cancer Consortium Radiation Oncology Group (DKTK-ROG) {#hpv16-dna-status-is-a-strong-prognosticator-of-loco-regional-control-after-postoperative-radiochemotherapy-of-locally-advanced-oropharyngeal-carcinoma-results-from-a-multicentre-explorative-study-of-the-german-cancer-consortium-radiation-oncology-group-dktk-rog .unnumbered}

> Fabian Lohaus
> ^[a](#_bookmark69),[i](#_bookmark70),[j](#_bookmark70),[1](#_bookmark71)^,
> Annett Linge
> ^[a](#_bookmark69),[i](#_bookmark70),[j](#_bookmark70),[1](#_bookmark71)^,
> Inge Tinhofer ^[b](#_bookmark69),[k](#_bookmark72)^, Volker Budach
> ^[b](#_bookmark69),[k](#_bookmark72)^, Eleni Gkika
> ^[c](#_bookmark69),[l](#_bookmark73)^, Martin Stuschke
> ^[c](#_bookmark69),[l](#_bookmark73)^, Panagiotis Balermpas
> [^m^](#_bookmark73), Claus Rödel
> ^[d](#_bookmark69),[m](#_bookmark73)^, Melanie Avlar
> ^[e](#_bookmark69),[n](#_bookmark74)^, Anca-Ligia Grosu
> ^[e](#_bookmark69),[o](#_bookmark74)^, Amir Abdollahi
> ^[f](#_bookmark69),[p](#_bookmark74),[q](#_bookmark75),[r](#_bookmark76)^,
> Jürgen Debus
> ^[f](#_bookmark69),[p](#_bookmark74),[q](#_bookmark75),[s](#_bookmark76)^,
> Christine Bayer [^g^](#_bookmark69), Claus Belka
> ^[g](#_bookmark69),[t](#_bookmark77)^, Steffi Pigorsch
> ^[g](#_bookmark69),[u](#_bookmark78)^, Stephanie E. Combs
> ^[g](#_bookmark69),[u](#_bookmark78)^,
>
> David Mönnich ^[h](#_bookmark70),[v](#_bookmark78)^, Daniel Zips
> ^[h](#_bookmark70),[v](#_bookmark78)^, Cläre von Neubeck
> ^[a](#_bookmark69),[j](#_bookmark70)^, Gustavo B. Baretton
> ^[a](#_bookmark69),[w](#_bookmark79),[x](#_bookmark80)^, Steffen Löck
> [^j^](#_bookmark70),
>
> []{#_bookmark69 .anchor}Howard D. Thames [^z^](#_bookmark81),
> Mechthild Krause
> ^[a](#_bookmark69),[i](#_bookmark70),[j](#_bookmark70),[y](#_bookmark80)^,
> Michael Baumann
> ^[a](#_bookmark69),[i](#_bookmark70),[j](#_bookmark70),[y](#_bookmark80),^[⇑](#_bookmark82),
> for the DKTK-ROG
>
> []{#_bookmark70 .anchor}^a^ *German Cancer Research Center (DKFZ),
> Heidelberg, Germany and German Cancer Consortium (DKTK) partner sites:
> Dresden;* ^b^ *Berlin;* ^c^ *Essen;* ^d^ *Frankfurt;* ^e^ *Freiburg;*
> ^f^ *Heidelberg;* []{#_bookmark72 .anchor}^g^ *Munich;* ^h^
> *Tübingen;* ^i^ *Department of Radiation Oncology, Faculty of Medicine
> and University Hospital Carl Gustav Carus, Technische Universität
> Dresden;* ^j^ *OncoRay -- National* []{#_bookmark73 .anchor}*Center
> for Radiation Research in Oncology, Faculty of Medicine and University
> Hospital Carl Gustav Carus, Technische Universität Dresden;* ^k^
> *Department of Radiooncology and* []{#_bookmark74
> .anchor}*Radiotherapy, Charité University Hospital, Berlin;* ^l^
> *Department of Radiotherapy, Medical Faculty, University of
> Duisburg-Essen, Essen;* ^m^ *Department of Radiotherapy and Oncology,
> Goethe-University Frankfurt;* ^n^ *Department of Radiation Oncology,
> Clinical Study Section, University of Freiburg;* ^o^ *Department of
> Radiation Oncology, University of Freiburg;* []{#_bookmark75
> .anchor}^p^ *Department of Radiation Oncology, Heidelberg Ion Therapy
> Center (HIT), Heidelberg Institute of Radiation Oncology (HIRO),
> National Center for Radiation Research in Oncology* []{#_bookmark76
> .anchor}*(NCRO), University of Heidelberg Medical School and German
> Cancer Research Center (DKFZ);* ^q^ *National Center for Tumor
> Diseases (NCT), University of Heidelberg Medical School and*
> []{#_bookmark77 .anchor}*German Cancer Research Center (DKFZ);* ^r^
> *Translational Radiation Oncology, University of Heidelberg Medical
> School and German Cancer Research Center (DKFZ);* ^s^ *Clinical*
> []{#_bookmark78 .anchor}*Cooperation Unit Radiation Oncology,
> University of Heidelberg Medical School and German Cancer Research
> Center (DKFZ);* ^t^ *Department of Radiotherapy and Radiation
> Oncology,* []{#_bookmark79 .anchor}*Ludwig-Maximilians-Universität,
> Munich;* ^u^ *Department of Radiation Oncology, Technische Universität
> München;* ^v^ *Department of Radiation Oncology, Faculty of Medicine
> and* [[]{#_bookmark80 .anchor}]{#_bookmark71 .anchor}*University
> Hospital Tübingen, Eberhard Karls Universität Tübingen;* ^w^
> *Institute of Pathology, Faculty of Medicine and University Hospital
> Carl Gustav Carus, Technische Universität* []{#_bookmark81
> .anchor}*Dresden;* ^x^ *Tumor- and Normal Tissue Bank, Universitäts
> KrebsCentrum (UCC), University Hospital Carl Gustav Carus, Technische
> Universität Dresden;* ^y^ *Institute of Radiooncology,
> Helmholtz-Zentrum Dresden -- Rossendorf, Germany;* ^z^ *Department of
> Biostatistics, The University of Texas MD Anderson Cancer Center,
> Houston, USA*
>
> a r t i c l e i n f o
>
> *Article history:*
>
> Received 30 August 2014
>
> Received in revised form 29 October 2014 Accepted 8 November 2014
>
> Available online 2 December 2014
>
> *Keywords:*
>
> DKTK-ROG HNSCC HPV
>
> Postoperative radiochemotherapy p16
>
> p53
>
> a b s t r a c t
>
> *Objective:* To investigate the impact of HPV status in patients with
> locally advanced head and neck squa- mous cell carcinoma (HNSCC), who
> received surgery and cisplatin-based postoperative radiochemother-
> apy.
>
> *Materials and methods:* For 221 patients with locally advanced
> squamous cell carcinoma of the hypophar- ynx, oropharynx or oral
> cavity treated at the 8 partner sites of the German Cancer Consortium,
> the impact of HPV DNA, p16 overexpression and p53 expression on
> outcome were retrospectively analysed. The primary endpoint was
> loco-regional tumour control; secondary endpoints were distant
> metastases and overall survival.
>
> *Results:* In the total patient population, univariate analyses
> revealed a significant impact of HPV16 DNA positivity, p16
> overexpression, p53 positivity and tumour site on loco-regional tumour
> control. Multivar- iate analysis stratified for tumour site showed
> that positive HPV 16 DNA status correlated with loco- regional tumour
> control in patients with oropharyngeal carcinoma (*p* = 0.02) but not
> in the oral cavity carcinoma group. Multivariate evaluation of the
> secondary endpoints in the total population revealed a significant
> association of HPV16 DNA positivity with overall survival (*p* \<
> 0.01) but not with distant metastases.
>
> []{#_bookmark82 .anchor}⇑ Corresponding author at: Department of
> Radiation Oncology and OncoRay -- National Center for Radiation
> Research in Oncology, Faculty of Medicine and University Hospital Carl
> Gustav Carus, Technische Universität Dresden, Fetscherstraße 74, 01307
> Dresden, Germany.
>
> *E-mail address:* <Michael.Baumann@uniklinikum-dresden.de> (M.
> Baumann).
>
> ^1^ Both authors contributed equally to this study.
>
> <http://dx.doi.org/10.1016/j.radonc.2014.11.011> 0167-8140/© 2014
> Elsevier Ireland Ltd.
>
> This is an open access article under the CC BY-NC-ND license
> (<http://creativecommons.org/licenses/by-nc-nd/3.0/>).
>
> *Conclusions:* HPV16 DNA status appears to be a strong prognosticator
> of loco-regional tumour control after postoperative cisplatin-based
> radiochemotherapy of locally advanced oropharyngeal carcinoma and is
> now being explored in a prospective validation trial.
>
> © 2014 Elsevier Ireland Ltd. Radiotherapy and Oncology 113 (2014)
> 317--323 This is an open access article under the CC BY-NC-ND license
> (<http://creativecommons.org/licenses/by-nc-nd/3.0/>).
>
> Head and neck squamous cell carcinoma (HNSCC) patients have a 5-year
> survival of approximately 50% [\[1,2\]](#_bookmark93). Within the last
> years, the number of oropharyngeal cancers has increased
> [\[3--5\]](#_bookmark94). Postop- erative radiotherapy (PORT) with
> concurrent cisplatin (PORT-C) has become standard for the adjuvant
> treatment of patients with resected locally advanced HNSCC after three
> randomised trials showing superior loco-regional control and improved
> survival rates compared to PORT alone [\[6--8\]](#_bookmark94). A
> meta-analysis of two of the above mentioned trials, the EORTC and the
> RTOG trials, con- cluded that patients with positive microscopic
> resection margins and/or extracapsular extension (ECE) from neck nodes
> benefit most from this combined treatment [\[9\]](#_bookmark94).
> However, all three trials also report increased toxicity after PORT-C
> vs. PORT [\[6--8\]](#_bookmark94).
>
> In recent years, infection with high-risk human papillomavirus (HPV)
> has been implicated in the pathogenesis of HNSCC, indepen- dently of
> the risk factors smoking and alcohol consumption
> [\[10\]](#_bookmark94). On the molecular level, the HPV oncogenes E6
> and E7 have been shown to drive carcinogenesis, which is accompanied
> by degrada- tion of p53 and pRB and overexpression of p16 due to loss
> of its transcriptional repression. In addition, HPV-related HNSCC show
> less p53 mutations than smoking and alcohol induced HNSCC
> [\[11\]](#_bookmark94). HPV-positive tumours most commonly originate
> from the oropharynx [\[12,13\]](#_bookmark95); high-grade histology
> and positive lymph nodes are frequent [\[14\]](#_bookmark96).
> Interestingly, a recent study observed that patients with HPV16 DNA
> positive oropharyngeal carcinoma and lower levels of comorbidity show
> a significantly better overall sur- vival compared to patients with
> HPV16 DNA negative tumours [\[15\]](#_bookmark97). Specifically for
> the field of Radiation Oncology, it has been shown that HPV-positivity
> is a strong prognostic marker for overall survival and/or local tumour
> control in patients treated with pri- mary radiochemotherapy
> [\[16--20\]](#_bookmark100). However, its impact on out- come after
> PORT-C is not well evaluated so far.
>
> In an ongoing multicentre retrospective -- prospective trial con-
> ducted by the Radiation Oncology Group of the German Cancer Consortium
> (DKTK-ROG), biomarkers for stratification of patients for dose of
> primary or postoperative radiochemotherapy of HNSCC are being
> evaluated. The present publication reports the impact of HPV infection
> on loco-regional tumour control and survival after PORT-C in the
> multicentre explorative cohort.
>
> Material and methods
>
> *Patients, treatment and tissue samples*
>
> Patients meeting the following criteria were eligible for inclu- sion
> in this retrospective study: histologically proven squamous cell
> carcinoma arising from the hypopharynx, oropharynx or oral cavity,
> treatment between 2005 and 2010 with a curatively intended
> cisplatin-based PORT-C according to standard protocols covering the
> former tumour region and the neck nodes. All patients had to be judged
> as being at high risk for loco-regional recurrence due to locally
> advanced disease with a tumour stage pT4 and/or \>3 positive lymph
> nodes and/or due to postoperative residual disease (positive
> microscopic resection margins and/or extracapsular spread). Minimum
> follow-up of patients without progressive dis- ease had to be 24
> months. Additionally, formalin-fixed paraffin- embedded (FFPE)
> material, radiotherapy treatment plans, CT, MRI or PET--CT images of
> the localisation of the recurrent tumours as well as follow-up data of
> patients had to be available. Smoking sta-
>
> tus and alcohol consumption were not consistently recorded for all
> patients and therefore could not be analysed. It was aimed to include
> 40 patients per DKTK partner centre (i.e., 320 patients in total). To
> enhance the proportion of HPV-positivity, patients were included
> consecutively backwards from 2010 towards 2005 in all centres, as HPV
> prevalence in HNSCC is increasing in recent years
> [\[21\]](#_bookmark105). Finally, 221 patients were found to meet all
> requirements. Those patients were evaluated in this study ([Table
> 2](#_bookmark83)). Pathological specimens, radiotherapy treatment
> plans, radiological images of recurrent tumours and follow-up data of
> patients were centrally collected in the DKTK RadPlanBio Platform
> \[Skripcak et al., manu- script in preparation\] at the DKTK partner
> site Dresden.
>
> Ethical approval for multicentre retrospective analyses of clini- cal
> and biological data was obtained by the Ethics Committees of all DKTK
> partner sites.
>
> *Failure pattern analysis*
>
> Disease status and first site of relapse were evaluated by the
> treating institution (loco-regional failure, distant failure or com-
> bined failure). When loco-regional recurrence and distant metasta- ses
> occurred at the same time (maximally 6 weeks difference), the patient
> was counted as combined failure. For every reported loco- regional
> failure, the radiotherapy treatment plan and radiological images of
> the recurrence (CT, MRI or PET--CT) were centrally reviewed to ensure
> that failures originated from the irradiated volume.
>
> *Preparation of biomaterials for biomarker analyses*
>
> FFPE blocks of the primary tumours were centrally collected at the
> DKTK partner site Dresden where slides for immunohisto- chemistry were
> prepared and genomic DNA was extracted (vide infra). In parallel
> tissue microarrays, RNA isolates and cDNA were generated for further
> investigations of biomarkers, which are cur- rently ongoing at the
> different partner sites. HPV DNA, p16 and p53 reported in this article
> were evaluated at the DKTK partner site Dresden.
>
> *Immunohistochemical staining of p16*
>
> In all FFPE samples, the squamous cell carcinoma content was estimated
> from haematoxylin and eosin stained tissue sections and FFPE samples
> with \<10% tumour content were excluded from p16 analysis. Two hundred
> and fourteen of the 221 tumour sam- ples (60 oral cavity, 121
> oropharynx, 33 hypopharynx) were evalu- able for p16.
> Immunohistochemical staining was performed using the CINtec^®^
> Histology Kit (Roche mtm laboratories AG, Basel, CH) according to the
> manufacturer's instruction. Overexpression of p16 was defined as P70%
> intense tumour staining [\[16\]](#_bookmark100). Blinded samples were
> evaluated semi-quantitatively by two independent observers (A.L. and
> C.v.N.) with an inter-observer variability of \<5%.
>
> *Immunohistochemical staining of p53*
>
> FFPE material from all 221 patients was available for p53 anal- ysis.
> Following deparaffinisation and antigen retrieval in target retrieval
> solution (pH 9; Dako, Glostrup, DK) for 35 min at 630 W,
> immunohistochemical staining was performed. Endoge- nous peroxidase
> activity was blocked (Peroxidase Block, Dako).
>
> Sections were then incubated with the monoclonal mouse anti- human p53
> antibody (Clone DO-7; Dako) in Dako REAL Antibody Diluent for 30 min.
> Negative control slides were incubated with corresponding IgG antibody
> control (Dako). The staining was visu- alised by DAB immunostaining
> (Dako REAL EnVision Detection System, Peroxidase/DAB, Rabbit/Mouse).
> Blinded samples were evaluated semi-quantitatively by two independent
> observers (A.L. and C.v.N.) with an inter-observer variability of
> \<5%. Percent- age of p53 staining [\[22\]](#_bookmark106) and
> staining intensity were scored (0, +, ++,
>
> +++). Tumours with P70% positive nuclei and moderate (++) or strong
> (+++) staining intensities were considered as p53 positive.
>
> *DNA extraction and PCR array-based analysis of HPV status*
>
> Genomic DNA was extracted from 5 lm FFPE-sections using the QIAamp DNA
> FFPE tissue kit (Qiagen GmbH, Hilden, DE) according to the
> manufacturer's instruction and stored at -20 °C until required. HPV
> DNA analyses including genotyping were carried out using the LCD-Array
> HPV 3.5 kit (CHIPRON GmbH, Berlin, DE) according to the manufacturer's
> instruction. Briefly, PCR was per- formed using the provided Primer
> Mix A (My 11/09) and B ('125') and the HotStarTaq Plus Master Mix
> (Qiagen GmbH).
>
> Hybridisation mix including 5 ll of each amplified PCR product *A*
>
> and *B* were added to each field of the LCD-Array. After staining and
> washing, the hybridisation spots were scanned and analysed using the
> SlideReader Software (CHIPRON GmbH). For internal quality control
> purposes, a positive control (HPV33 DNA, UT-SCC- 45 xenograft) and a
> negative control (RNase free water; Qiagen GmbH) were included in each
> array. Six tissue samples had to be omitted from HPV DNA analysis due
> to too low DNA yield, thus 215 of the 221 tumour samples (58 oral
> cavity, 123 oropharynx, 34 hypopharynx) were evaluable for HPV-PCR
> array.
>
> *Statistics*
>
> The primary endpoint was loco-regional tumour control; free- dom from
> distant metastases and overall survival were evaluated as secondary
> endpoints. Loco-regional tumour control, distant metastases and
> overall survival were calculated from the first day of radiotherapy to
> the date of local or regional recurrence, date of metastases and date
> of death or last follow-up, respectively. All endpoints were estimated
> with the Kaplan--Meier method. The impact of potential prognosticators
> on the endpoints was evalu- ated using the Cox-regression model.
> Parameters found to be sig- nificant in univariate analysis were
> included in a multivariate Cox model. Statistical analyses were
> performed for all patients and for the subgroups of patients with oral
> cavity cancers as well as oropharyngeal cancers. Patients diagnosed
> with hypopharyn- geal cancers were excluded from this subgroup
> analysis due to the low number of cases. Sensitivity and specificity
> of p16 and HPV16 DNA for predicting loco-regional recurrence were
> deter- mined by cross tabulation. For all analyses, two-sided tests
> were used and *p*-values \<0.05 were considered statistically
> significant. SPSS 21 software (IBM Corporation, Armonk, NY, USA) was
> used for the generation of Kaplan--Meier plots. STATA 11 (StataCorp
> LP, College Station, TX, USA) was used for Cox analyses.
>
> Results
>
> In total, 221 patients treated with PORT-C for locally advanced HNSCC
> were evaluated in this multicentre retrospective study. Patient
> characteristics, treatment parameters and the number of patients
> included at each of the 8 DKTK partner sites are summa- rised in
> [Tables 1 and 2](#_bookmark84).
>
> Isolated loco-regional failure occurred in 21 patients, isolated
> distant failure in 31 patients and combined failures were observed
>
> in 8 patients. In 2 patients loco-regional recurrence occurred after
> dis- tant progression and 4 patients developed distant progression
> after loco-regional recurrence. All loco-regional recurrences occurred
> in the treatment volume. Actuarial rates at two years for
> loco-regional control, freedom from distant metastases and overall
> survival for the total patient population were 89.6%, 85.1% and 83.4%.
>
> The results of the biomarker analyses of HPV DNA, p16 and p53 and
> their occurrence at the different tumour sites are shown in [Table
> 3](#_bookmark87). According to the International Agency for Research
> in Cancer (IARC), HPV16 DNA positive HNSCC are currently being
> considered as HPV associated [\[23\]](#_bookmark98) and only this
> parameter was used for further analysis. HPV16 DNA positivity was
> observed in 34% of the tumours. Overexpression of p16 was found in 37%
> of all tumours, and 53% of the oral cavity tumours were positive for
> p53. The majority (86%) of HPV16 DNA positive tumours were found to be
> p53 negative.
>
> Only two loco-regional recurrences occurred in patients suffer- ing
> from HPV16 DNA positive tumours: the first in a R0-resected, ECE
> positive, pT2 pN2b oral cavity carcinoma (floor of mouth) after 23
> months in the boost volume (66 Gy), the second in a R0 resected, ECE
> positive pT2 pN2b oropharyngeal cancer (tonsil) after 26 months at the
> margin of the boost volume (64 Gy) to the adju- vant volume (54 Gy).
> Univariate analyses revealed a significant impact on loco-regional
> tumour control for HPV16 DNA positivity (HR 0.13; *p* \< 0.01; [Fig.
> 1](#all-patients)A, Supplementary Table S1). This effect was seen at
> all 8 treatment centres ([Fig. 2](#_bookmark90)). Overexpression of
> p16 (HR 0.24; *p* \< 0.01), p53 positivity (HR 3.36; *p* \< 0.01) and
> tumour site (oral cavity vs. all other tumour sites, HR 3.86; *p* \<
> 0.01; oropharynx vs. all other tumour sites, HR 0.38; *p* = 0.01) also
> showed a significant impact on loco-regional tumour control. No
> significant impact was found for sex, UICC stage, R status and ECE
> status (Supplementary Table S1). Specificity and sensitivity of HPV16
> DNA positivity in the total patient population to predict
> loco-regional tumour control were 93% and 38%, the corresponding
> values for overexpression of p16 were 86% and 41%. In oropharyn- geal
> cancer specificity and sensitivity of HPV16 DNA positivity were 91%
> and 53%, for p16 overexpression 73% and 56% were obtained (1 out of 11
> loco-regional recurrences occurred within the HPV16 DNA positive
> group, 3 out of 11 were tested positive for p16). Stratified for
> tumour site, univariate analyses in oropha- ryngeal cancer showed that
> HPV16 DNA (HR 0.09; *p* = 0.02; [Fig. 1](#all-patients)B,
> Supplementary Table S1) but not p16 overexpression or p53 positivity
> have a significant impact on loco-regional tumour control, whereas p53
> showed a significant impact on loco-regional tumour control in oral
> cavity cancer (HR 3.61; *p* \< 0.05; Supple- mentary Table S1).
>
> [Table 4](#_bookmark91) summarises the results of the multivariate
> analyses,
>
> including the significant parameters of the univariate analyses plus
> ECE status, which had shown a significant impact on the secondary
> endpoints (Supplementary Table S1). For the total patient popula-
> tion, positive HPV16 DNA status was significantly associated with a
> high chance of loco-regional tumour control (HR 0.20; *p* = 0.04).
> Oral cavity cancer showed significantly poorer loco-regional tumour
> control than oropharyngeal cancers (HR 2.30; *p* = 0.04). Multivariate
> analysis stratified for tumour site showed that positive HPV16 DNA
> status correlated with loco-regional tumour control in patients with
> oropharyngeal carcinoma (HR 0.09; *p* = 0.03) but not in the oral
> cavity carcinoma group ([Table 4](#_bookmark91)). As HPV16 DNA and p16
> are strongly correlated, a second multivariate Cox model assessing p16
> overexpression was performed. The results of this Cox model showed
> that the HPV status was a borderline independent prognos- tic marker
> for loco-regional tumour control in the total patient pop- ulation (HR
> 0.36; *p* = 0.07; Supplementary Table S2).
>
> Multivariate evaluation of the secondary endpoints in the total
> population revealed a significant association of HPV16 DNA positivity
> with overall survival (HR 0.36; *p* \< 0.01) but not with
>
> [[[]{#_bookmark85 .anchor}]{#_bookmark84 .anchor}]{#_bookmark83
> .anchor}Table 1
>
> Patient characteristics and treatment parameter.

+---+-------------------------+--------+-----+----+-----+------+------+
|   | Patient characteristic  |        |     |    |     | >    |      |
|   |                         |        |     |    |     |  *n* |      |
|   |                         |        |     |    |     | > %  |      |
+===+=========================+========+=====+====+=====+======+======+
|   | Number of patients      |        |     |    |     | >    |      |
|   |                         |        |     |    |     |  221 |      |
+---+-------------------------+--------+-----+----+-----+------+------+
|   | Sex                     | > Male |     |    |     | >    | >    |
|   |                         |        |     |    |     |  180 | 81.4 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | >      |     |    |     | > 41 | >    |
|   |                         | Female |     |    |     |      | 18.6 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   | UICC stage              | > II   |     |    |     | > 8  | >    |
|   |                         |        |     |    |     |      |  3.6 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > III  |     |    |     | > 33 | >    |
|   |                         |        |     |    |     |      | 18.6 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > IV   |     |    |     | >    | >    |
|   |                         |        |     |    |     |  180 | 81.4 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   | Tumour localisation     | > Oral |     |    |     | > 60 | >    |
|   |                         | >      |     |    |     |      | 27.1 |
|   |                         | cavity |     |    |     |      |      |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > Orop |     |    |     | >    | >    |
|   |                         | harynx |     |    |     |  126 | 57.0 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | >      |     |    |     | > 35 | >    |
|   |                         |  Hypop |     |    |     |      | 15.8 |
|   |                         | harynx |     |    |     |      |      |
+---+-------------------------+--------+-----+----+-----+------+------+
|   | R                       | > Ne   |     |    |     | >    | >    |
|   | status[⇑](#_bookmark85) | gative |     |    |     |  125 | 56.6 |
|   |                         | >      |     |    |     | >    | >    |
|   |                         | > []{  |     |    |     | > 94 | >    |
|   |                         | #_book |     |    |     |      | 42.5 |
|   |                         | mark86 |     |    |     |      |      |
|   |                         | > .anc |     |    |     |      |      |
|   |                         | hor}Po |     |    |     |      |      |
|   |                         | sitive |     |    |     |      |      |
+---+-------------------------+--------+-----+----+-----+------+------+
|   | ECE status              | > Ne   |     |    |     | >    | >    |
|   |                         | gative |     |    |     |  102 | 46.2 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > Po   |     |    |     | >    | >    |
|   |                         | sitive |     |    |     |  119 | 53.8 |
+---+-------------------------+--------+-----+----+-----+------+------+
| > |                         | >      | >   | 2  | >   | 90%  | > R  |
|   |                         | Median |  Pe | 5% | 75% |      | ange |
| T |                         |        | rce |    |     |      |      |
| r |                         |        | nti |    |     |      |      |
| e |                         |        | les |    |     |      |      |
| a |                         |        | >   |    |     |      |      |
| t |                         |        | >   |    |     |      |      |
| m |                         |        | []{ |    |     |      |      |
| e |                         |        | #_b |    |     |      |      |
| n |                         |        | ook |    |     |      |      |
| t |                         |        | mar |    |     |      |      |
| > |                         |        | k87 |    |     |      |      |
|   |                         |        | >   |    |     |      |      |
| p |                         |        |  .a |    |     |      |      |
| a |                         |        | nch |    |     |      |      |
| r |                         |        | or} |    |     |      |      |
| a |                         |        | 10% |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
+---+-------------------------+--------+-----+----+-----+------+------+
| > |                         | > 200  | >   | 2  | >   | 240  | >    |
|   |                         |        | 100 | 00 | 200 |      | 100- |
| A |                         |        |     |    |     |      | -300 |
| p |                         |        |     |    |     |      |      |
| p |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| c |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| p |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| - |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| ( |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| g |                         |        |     |    |     |      |      |
| / |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| ^ |                         |        |     |    |     |      |      |
| 2 |                         |        |     |    |     |      |      |
| ^ |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| b |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| y |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| f |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| c |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| ) |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| R |                         |        |     |    |     |      |      |
| T |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| ( |                         |        |     |    |     |      |      |
| G |                         |        |     |    |     |      |      |
| y |                         |        |     |    |     |      |      |
| ) |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| B |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| v |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| P |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| f |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| c |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| A |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| j |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| v |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| v |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| P |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| f |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| c |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| T |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| b |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| w |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| g |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| y |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| f |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| h |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| p |                         |        |     |    |     |      |      |
| y |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| ( |                         |        |     |    |     |      |      |
| w |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| k |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| ) |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| O |                         |        |     |    |     |      |      |
| v |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| r |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| f |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| P |                         |        |     |    |     |      |      |
| O |                         |        |     |    |     |      |      |
| R |                         |        |     |    |     |      |      |
| T |                         |        |     |    |     |      |      |
| - |                         |        |     |    |     |      |      |
| C |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| ( |                         |        |     |    |     |      |      |
| d |                         |        |     |    |     |      |      |
| a |                         |        |     |    |     |      |      |
| y |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| ) |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| F |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| l |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| w |                         |        |     |    |     |      |      |
| - |                         |        |     |    |     |      |      |
| u |                         |        |     |    |     |      |      |
| p |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| i |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| e |                         |        |     |    |     |      |      |
| > |                         |        |     |    |     |      |      |
|   |                         |        |     |    |     |      |      |
| ( |                         |        |     |    |     |      |      |
| m |                         |        |     |    |     |      |      |
| o |                         |        |     |    |     |      |      |
| n |                         |        |     |    |     |      |      |
| t |                         |        |     |    |     |      |      |
| h |                         |        |     |    |     |      |      |
| s |                         |        |     |    |     |      |      |
| ) |                         |        |     |    |     |      |      |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 64.0 | > 6 | 63 | > 6 | 66.0 | 57   |
|   |                         |        | 0.0 | .9 | 6.0 |      | .2-- |
|   |                         |        |     |    |     |      | 68.4 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 2.0  | >   | 1  | >   | 2.0  | >    |
|   |                         |        | 1.8 | .8 | 2.0 |      | 1.8- |
|   |                         |        |     |    |     |      | -2.1 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 50.4 | > 5 | 50 | > 5 | 60.0 | 46   |
|   |                         |        | 0.0 | .0 | 5.9 |      | .8-- |
|   |                         |        |     |    |     |      | 66.0 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 2.0  | >   | 2  | >   | 2.1  | >    |
|   |                         |        | 1.8 | .0 | 2.0 |      | 1.8- |
|   |                         |        |     |    |     |      | -2.2 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 6.0  | >   | 5  | >   | 9.6  | 1    |
|   |                         |        | 4.1 | .0 | 7.5 |      | .0-- |
|   |                         |        |     |    |     |      | 23.0 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 44.0 | > 4 | 43 | > 4 | 50.0 | 31   |
|   |                         |        | 1.0 | .0 | 6.5 |      | .0-- |
|   |                         |        |     |    |     |      | 57.0 |
+---+-------------------------+--------+-----+----+-----+------+------+
|   |                         | > 47.3 | > 1 | 30 | > 6 | 71.7 | 2    |
|   |                         |        | 1.1 | .7 | 1.2 |      | .5-- |
|   |                         |        |     |    |     |      | 98.6 |
+---+-------------------------+--------+-----+----+-----+------+------+

> []{#_bookmark88 .anchor}\* Two patients were not evaluable.
>
> Table 2
>
> Number of patients per treatment centre and tumour localisation.

+----------+------+------+-----+------+-------------+---------------+
| > T      | > ce | >    | > O | > ca | >           | > Hypopharynx |
| reatment | ntre |  *n* | ral | vity |  Oropharynx |               |
+==========+======+======+=====+======+=============+===============+
| >        | 42   |      | >   | > 14 |             | > 7           |
|  Dresden |      |      |  21 |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| > F      | 27   |      | >   | > 14 |             | > 1           |
| rankfurt |      |      |  12 |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| >        | 33   |      | > 9 | > 19 |             | > 5           |
| Tübingen |      |      |     |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| >        | 30   |      | > 5 | > 20 |             | > 5           |
| Freiburg |      |      |     |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| > Essen  | 32   |      | > 2 | > 22 |             | > 9           |
+----------+------+------+-----+------+-------------+---------------+
| > Berlin | 24   |      | > 9 | > 11 |             | > 4           |
+----------+------+------+-----+------+-------------+---------------+
| > M      | 17   |      | > 0 | > 16 |             | > 1           |
| unich[^a |      |      |     |      |             |               |
| ^](#_boo |      |      |     |      |             |               |
| kmark86) |      |      |     |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| > He     | 15   |      | > 2 | > 10 |             | > 3           |
| idelberg |      |      |     |      |             |               |
+----------+------+------+-----+------+-------------+---------------+
| > Total  | >    |      | >   | >    |             | > 35          |
|          |  221 |      |  60 |  126 |             |               |
+----------+------+------+-----+------+-------------+---------------+

> ^a^ Partner site Munich consists of Technische Universität and Ludwig-
> Maximilians-Universität.
>
> Table 3
>
> Number of tumours with positive biomarkers per tumour localisation.

+-----------------+-----------+-----------+------------+--------------+
| > Biomarker     | > Overall | > Oral    | >          | >            |
|                 |           | > cavity  | Oropharynx |  Hypopharynx |
+=================+===========+===========+============+==============+
| > HPV16 DNA     | > 72      | > 7       | > 59       | > 5 (14.7%)  |
|                 | > (33.5%) | > (12.1%) | > (48.0%)  |              |
+-----------------+-----------+-----------+------------+--------------+
| > HPV16/18      | > 1       | > --      | > 1        | > --         |
| > DNA[^a^       |           |           |            |              |
| ](#_bookmark88) |           |           |            |              |
+-----------------+-----------+-----------+------------+--------------+
| > HPV18 DNA     | > 1       | > 1       | > --       | > --         |
+-----------------+-----------+-----------+------------+--------------+
| > HPV33 DNA     | > 1       | > --      | > 1        | > --         |
+-----------------+-----------+-----------+------------+--------------+
| > p16           | > 79      | > 11      | > 65       | > 3 (9.1%)   |
|                 | > (36.9%) | > (18.3%) | > (53.7%)  |              |
+-----------------+-----------+-----------+------------+--------------+
| > p53           | > 85      | > 32      | > 40       | > 13 (37.1%) |
|                 | > (38.5%) | > (53.3%) | > (31.7%)  |              |
+-----------------+-----------+-----------+------------+--------------+

> ^a^ Patient was included in HPV16 DNA positive group.
>
> distant metastases. ECE status showed significant association with
> distant metastases (HR 2.55; *p* \< 0.01) and borderline significance
> with overall survival ([Table 4](#_bookmark91) and Supplementary Table
> S1). Overexpression of p16 showed a significant association with
> distant metastases (HR 0.31; *p* = 0.02) and on overall survival (HR
> 0.44; *p* = 0.01) (Supplementary Table S2).
>
> Discussion
>
> While several previous studies provided strong evidence that the HPV
> status is a significant prognostic marker of loco-regional
>
> tumour control and/or survival in patients treated with primary
> radiotherapy or radiochemotherapy for locally advanced HNSCC
> [\[14,16,18,19\]](#_bookmark96), the impact of HPV status on outcome
> of postopera- tive radio(chemo)therapy is less well investigated. The
> results of the present multicentre retrospective study of the DKTK-ROG
> show that HPV16 DNA positivity is a significant prognosticator of
> loco- regional tumour control and survival of patients treated with
> cis- platin-based postoperative radiochemotherapy after surgical
> resection of locally advanced HNSCC. The effect appears to be robust
> over all treatment centres and is driven by the results in
> oropharyngeal cancers. Our findings are in line with a study by
> Snietura et al. who investigated the influence of HPV infection on the
> clinical outcome in a posthoc analysis of a randomised clinical trial
> of two different schedules of PORT without chemotherapy in 279 HNSCC
> patients. HPV analysis was conducted in tumours of 131 patients. From
> the 66 patients with oral cavity or oropharyn- geal carcinoma, 9 were
> found to be positive for HPV16 DNA and were locally controlled after 5
> years, whereas the loco-regional tumour control rate in the whole HPV
> DNA negative group was only 58% [\[24\]](#_bookmark98). Taken
> together, HPV16 DNA appears to be a poten- tially promising biomarker
> for stratification and individualised prescription of postoperative
> radiotherapy. HPV-positivity seems to be sufficient to define a
> patient cohort that is highly unlikely to develop loco-regional
> recurrences after PORT-C, which is in con- trast to primary
> radiochemotherapy where more stratification parameters are necessary
> [\[25\]](#_bookmark98). This difference between the two treatment
> approaches may be caused by the fact that such addi- tional factors,
> most obviously tumour volume, play a lesser role when the tumour is
> resected. Other patient-related risk factors like smoking status could
> not be evaluated in our dataset but might be relevant as well. We are
> currently performing similar analyses in a patient cohort that has
> been treated by the same centres and within the same period of time
> with primary radiochemotherapy to further evaluate such differential
> prognosticators using a multi- dimensional statistical approach
> including radiobiological esti- mates e.g., on tumour cell number. For
> the group of HPV- negative patients, the situation is largely
> different. Here, HPV can-

###### All patients

> **100**

######  Pts. with oropharyngeal carcinoma

> **100**
>
> **80 80**
>
> []{#_bookmark90 .anchor}**60 60**
>
> **0**
>
> []{#_bookmark91 .anchor}Fig. 1. Kaplan--Meier estimates of
> loco-regional tumour control. (A) Patients with HPV16 DNA positive
> HNSCC had significantly higher loco-regional tumour control compared
> to HPV16 DNA negative tumours. (B) In subgroup analysis, patients with
> HPV16 DNA positive oropharyngeal tumours showed significantly higher
> loco-regional tumour control rates than those with HPV16 DNA negative
> tumours confined to the oropharynx.
>
> Treatment centre
>
> Loco-regional control, HR (95% CI)
>
> HPV16 DNA positive HPV16 DNA negative Events Total Events Total
>
> []{#_bookmark92 .anchor}\* No CI was calculated in case of no event in
> the HPV16 DNA positive group
>
> 0 1 2 3 4 5
>
> HR (95% CI)
>
> Fig. 2. Forest plot demonstrating the impact of HPV16 DNA status on
> loco-regional tumour control at the different treatment centres and
> the pooled estimate (univariate analyses; Supplementary Table S1).
> HPV16 DNA status is a prognostic parameter for loco-regional tumour
> control at all treatment centres.
>
> Table 4
>
> Multivariate analyses of prognostic factors for loco-regional control,
> distant metastases and overall survival. HR = hazard ratio; 95% CI =
> 95 percent confidence interval.

<table>
<colgroup>
<col style="width: 15%" />
<col style="width: 4%" />
<col style="width: 10%" />
<col style="width: 11%" />
<col style="width: 7%" />
<col style="width: 10%" />
<col style="width: 11%" />
<col style="width: 7%" />
<col style="width: 10%" />
<col style="width: 9%" />
</colgroup>
<thead>
<tr class="header">
<th colspan="10"><blockquote>
<p>All sites Oral cavity Oropharynx</p>
<p>HR (95% CI) <em>p</em>-value HR (95% CI) <em>p</em>-value HR (95% CI)
<em>p</em>-value</p>
</blockquote></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><blockquote>
<p><em>Loco-regional control</em></p>
</blockquote></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>HPV16 DNA</p>
</blockquote></td>
<td>0.20</td>
<td><blockquote>
<p>(0.04–0.92)</p>
</blockquote></td>
<td><blockquote>
<p>0.04</p>
</blockquote></td>
<td><blockquote>
<p>0.83</p>
</blockquote></td>
<td><blockquote>
<p>(0.10–6.75)</p>
</blockquote></td>
<td><blockquote>
<p>0.87</p>
</blockquote></td>
<td><blockquote>
<p>0.09</p>
</blockquote></td>
<td><blockquote>
<p>(0.01–0.74)</p>
</blockquote></td>
<td><blockquote>
<p>0.03</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>p53 positivity</p>
</blockquote></td>
<td>1.81</td>
<td><blockquote>
<p>(0.81–4.06)</p>
</blockquote></td>
<td><blockquote>
<p>0.15</p>
</blockquote></td>
<td><blockquote>
<p>3.21</p>
</blockquote></td>
<td><blockquote>
<p>(0.87–11.8)</p>
</blockquote></td>
<td><blockquote>
<p>0.08</p>
</blockquote></td>
<td><blockquote>
<p>0.89</p>
</blockquote></td>
<td><blockquote>
<p>(0.23–3.50)</p>
</blockquote></td>
<td><blockquote>
<p>0.87</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>ECE status</p>
</blockquote></td>
<td>1.35</td>
<td><blockquote>
<p>(0.62–2.93)</p>
</blockquote></td>
<td><blockquote>
<p>0.45</p>
</blockquote></td>
<td><blockquote>
<p>3.68</p>
</blockquote></td>
<td><blockquote>
<p>(0.83–16.4)</p>
</blockquote></td>
<td><blockquote>
<p>0.09</p>
</blockquote></td>
<td><blockquote>
<p>1.09</p>
</blockquote></td>
<td><blockquote>
<p>(0.29–4.14)</p>
</blockquote></td>
<td><blockquote>
<p>0.90</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Oral cavity<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>2.30</td>
<td><blockquote>
<p>(1.02–5.16)</p>
</blockquote></td>
<td><blockquote>
<p>0.04</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Hypopharynx<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>0.71</td>
<td><blockquote>
<p>(0.19–2.58)</p>
</blockquote></td>
<td><blockquote>
<p>0.60</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>Distant metastases</em></p>
</blockquote></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>HPV16 DNA</p>
</blockquote></td>
<td>0.56</td>
<td><blockquote>
<p>(0.22–1.42)</p>
</blockquote></td>
<td><blockquote>
<p>0.23</p>
</blockquote></td>
<td><blockquote>
<p>0.97</p>
</blockquote></td>
<td><blockquote>
<p>(0.21–4.38)</p>
</blockquote></td>
<td><blockquote>
<p>0.97</p>
</blockquote></td>
<td><blockquote>
<p>0.40</p>
</blockquote></td>
<td><blockquote>
<p>(0.11–1.49)</p>
</blockquote></td>
<td><blockquote>
<p>0.17</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>p53 positivity</p>
</blockquote></td>
<td>1.39</td>
<td><blockquote>
<p>(0.72–2.70)</p>
</blockquote></td>
<td><blockquote>
<p>0.32</p>
</blockquote></td>
<td><blockquote>
<p>0.93</p>
</blockquote></td>
<td><blockquote>
<p>(0.35–2.45)</p>
</blockquote></td>
<td><blockquote>
<p>0.88</p>
</blockquote></td>
<td><blockquote>
<p>1.46</p>
</blockquote></td>
<td><blockquote>
<p>(0.42–5.12)</p>
</blockquote></td>
<td><blockquote>
<p>0.55</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>ECE status</p>
</blockquote></td>
<td>2.55</td>
<td><blockquote>
<p>(1.26–5.15)</p>
</blockquote></td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td><blockquote>
<p>9.10</p>
</blockquote></td>
<td><blockquote>
<p>(1.21–68.8)</p>
</blockquote></td>
<td><blockquote>
<p>0.03</p>
</blockquote></td>
<td><blockquote>
<p>1.36</p>
</blockquote></td>
<td><blockquote>
<p>(0.43–4.34)</p>
</blockquote></td>
<td><blockquote>
<p>0.60</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Oral cavity<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>2.37</td>
<td><blockquote>
<p>(1.11–5.07)</p>
</blockquote></td>
<td><blockquote>
<p>0.03</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Hypopharynx<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>2.73</td>
<td><blockquote>
<p>(1.15–6.47)</p>
</blockquote></td>
<td><blockquote>
<p>0.02</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p><em>Overall survival</em></p>
</blockquote></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td><blockquote>
<p>HPV16 DNA</p>
</blockquote></td>
<td>0.36</td>
<td><blockquote>
<p>(0.17–0.73)</p>
</blockquote></td>
<td><blockquote>
<p>&lt;0.01</p>
</blockquote></td>
<td><blockquote>
<p>0.30</p>
</blockquote></td>
<td><blockquote>
<p>(0.04–2.23)</p>
</blockquote></td>
<td><blockquote>
<p>0.24</p>
</blockquote></td>
<td><blockquote>
<p>0.36</p>
</blockquote></td>
<td><blockquote>
<p>(0.15–0.82)</p>
</blockquote></td>
<td><blockquote>
<p>0.02</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>p53 positivity</p>
</blockquote></td>
<td>1.07</td>
<td><blockquote>
<p>(0.65–1.79)</p>
</blockquote></td>
<td><blockquote>
<p>0.78</p>
</blockquote></td>
<td><blockquote>
<p>1.27</p>
</blockquote></td>
<td><blockquote>
<p>(0.58–2.79)</p>
</blockquote></td>
<td><blockquote>
<p>0.55</p>
</blockquote></td>
<td><blockquote>
<p>1.03</p>
</blockquote></td>
<td><blockquote>
<p>(0.46–2.30)</p>
</blockquote></td>
<td><blockquote>
<p>0.94</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>ECE status</p>
</blockquote></td>
<td>1.63</td>
<td><blockquote>
<p>(0.98–2.70)</p>
</blockquote></td>
<td><blockquote>
<p>0.06</p>
</blockquote></td>
<td><blockquote>
<p>3.60</p>
</blockquote></td>
<td><blockquote>
<p>(1.24–10.5)</p>
</blockquote></td>
<td><blockquote>
<p>0.02</p>
</blockquote></td>
<td><blockquote>
<p>0.99</p>
</blockquote></td>
<td><blockquote>
<p>(0.47–2.13)</p>
</blockquote></td>
<td><blockquote>
<p>1.00</p>
</blockquote></td>
</tr>
<tr class="odd">
<td><blockquote>
<p>Oral cavity<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>1.73</td>
<td><blockquote>
<p>(1.00–2.96)</p>
</blockquote></td>
<td><blockquote>
<p>&lt;0.05</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
<tr class="even">
<td><blockquote>
<p>Hypopharynx<a href="#_bookmark92">⇑</a></p>
</blockquote></td>
<td>0.66</td>
<td><blockquote>
<p>(0.30–1.46)</p>
</blockquote></td>
<td><blockquote>
<p>0.31</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
<td></td>
<td><blockquote>
<p>–</p>
</blockquote></td>
</tr>
</tbody>
</table>

> \* Baseline oropharynx.
>
> not be used as a sole biomarker to predict tumour recurrences, as
> shown by the low sensitivity of 38% or 41% for HPV16 DNA or p16
> positivity. Thus, the HPV-negative group needs further investiga-
> tions into potential biomarkers to stratify for patients who may need
> treatment intensification and for patients for whom local recurrences
> are not to be expected.
>
> Currently a prospective multicentre study of the DKTK-ROG is ongoing
> to validate the prognostic value of HPV16 DNA positivity on
> loco-regional tumour control after PORT-C in 240 HNSCC patients. If
> the results of the present retrospective cohort are con- firmed, an
> interventional trial to de-escalate PORT-C radiation doses in HPV16
> DNA positive, clinically suitable oropharyngeal cancer patients will
> be initiated. Specificity assessment of HPV16 DNA positivity for
> loco-regional tumour control from the present investigation suggests
> that very few if any recurrences should be expected from a moderate
> decrease of radiation dose in these patients, therefore strict
> stopping rules for patient safety against the risk of inferior
> treatment may be applied in such trial, using e.g., a Pocock boundary
> approach [\[26\]](#_bookmark98). Further refinement of risk
> stratification specifically for the HPV-negative group may result from
> prospective assessment of clinical parameters [\[27\]](#_bookmark98)
> in the val- idation trial, and from ongoing efforts to identify
> further biomark- ers in the current retrospective and in the
> validation patient cohort.
>
> The low risk of loco-regional recurrence in HPV16 DNA positive
> oropharyngeal cancers after curatively intended resection and PORT-C
> suggests that either less tumour stem cells are present at start of
> PORT-C, that the remaining HPV-positive tumour cells are more
> radio(chemo)sensitive, or a combination of both. Recently it was
> reported that HPV-positive oropharyngeal cancers show low expression
> of stem cell markers such as CD44 and CD98 compared to HPV-negative
> oropharyngeal cancers [\[28\]](#_bookmark98). Furthermore, patients
> []{#_bookmark93 .anchor}with HPV-positive and low CD98 expressing
> tumours showed bet- ter overall survival and progression-free survival
> compared to patients with high CD98 expressing HPV-positive tumours.
> Increased radiosensitivity of HPV-positive tumour cells is sup- ported
> by a number of investigations. HPV-positive HNSCC cell lines (all
> positive for HPV DNA, HPV RNA and p16) assessed by col- ony formation
> assay in vitro showed a higher cellular radiosensi- tivity when
> compared to HPV-negative cell lines due to compromised DNA repair
> capacity [\[29\]](#_bookmark98). Similar observations have been
> reported by others [\[30,31\]](#_bookmark98). Further observations
> using both in vitro and in vivo approaches suggest that overexpressed
> p16 impairs the recruitment of RAD51 to the DNA damage site in
> HPV-positive HNSCC by down-regulation of cyclin D1, thereby affecting
> the cell cycle and homologous recombination-mediated DNA repair
> response [\[32\]](#_bookmark98).
>
> There is currently no generally agreed consensus for the assess-
>
> ment of the HPV infection status as a potential biomarker; general
> methods used for assessment of HPV infection include HPV DNA, HPV RNA,
> and p16 overexpression [\[33--35\]](#_bookmark99). The vast majority
> of HPV-positive HNSCC has been shown to be positive for HPV16 DNA
> [\[21,36\]](#_bookmark105), which is in line with the results reported
> here. HPV16 DNA showed stronger correlations with outcome parame- ters
> as compared to p16 immunohistochemistry in a cohort of 50 patients
> with oropharyngeal tumours who received primary radio- chemotherapy
> [\[37\]](#_bookmark101). Also in our study HPV16 DNA appears as a
> stronger prognosticator for loco-regional tumour control compared to
> p16 expression ([Table 4](#_bookmark91) vs. Supplementary Table S2),
> however this needs to be validated in a larger cohort.
>
> In the present study HPV16 DNA positivity was a strong inde- pendent
> prognosticator for loco-regional tumour control in oropha- ryngeal but
> not in oral cavity tumours. In contrast, increased p53 positivity was
> observed in oral cavity tumours, which suggests an alternative pathway
> for tumour development, e.g. life style factors.
>
> The tumour suppressor gene TP53 is known to be involved in carci-
> nogenesis of HNSCC [\[38\]](#_bookmark102) and its overexpression is
> reported in heavy smokers and heavy drinkers
> [\[39,40\]](#_bookmark103). Increased positivity has been linked to
> TP53 gene mutations, which can cause stabilisation and nuclear
> accumulation of p53 proteins [\[41\]](#_bookmark104). It has been
> demon- strated that the HPV oncoprotein E6 inactivates and inhibits
> p53 [\[42,43\]](#_bookmark107), which is in line with the fact that
> the majority of our HPV-positive study cohort was negative for p53.
>
> In our study, extracapsular extension of lymph nodes is a prog- nostic
> factor for overall survival in patients with oral cavity carci- nomas
> but not in the total patient population. This seems to be in contrast
> to the results of the meta-analysis by Bernier et al., show- ing that
> positive margins and/or extracapsular extension are the most
> significant prognosticators for poor outcome/overall survival
> [\[9\]](#_bookmark94). However this meta-analysis did not stratify
> between oral cav- ity and oropharyngeal cancer and molecular
> biomarkers were not investigated as confounding factors in the trials
> included in the meta-analysis (EORTC and RTOG trials). Furthermore
> patients included in the EORTC and RTOG trials might reflect a
> different population relative to the patient cohort analysed in the
> present study, underlining the necessity of constant marker adaptation
> for patient stratification. Further efforts to investigate different
> biomarkers specifically for the HPV-negative group receiving post-
> operative radiochemotherapy as well as for primary radiochemo- therapy
> are currently ongoing in the DKTK-ROG using material of the cohort
> reported here and of the validation trial.
>
> In conclusion, our results of this retrospective explorative mul-
> ticentre study show that HPV16 DNA seems to be a strong prog-
> nosticator of loco-regional tumour control after postoperative
> cisplatin-based radiochemotherapy of locally advanced oropharyn- geal
> carcinoma and is therefore a promising biomarker for patient
> stratification. The effect appeared robust over the 8 treatment cen-
> tres. For patients with HPV16 DNA positive oropharyngeal carci- noma
> treatment de-intensification may be a valid interventional option for
> a prospective trial that is currently prepared.
>
> Conflict of interest statement
>
> The authors have nothing to disclose.
>
> Acknowledgements
>
> The authors acknowledge the excellent technical assistance by Mrs.
> Liane Stolz-Kieslich and Mrs. Sigrid Balschukat. The authors wish to
> thank all pathologists, head and neck surgeons, and maxil- lofacial
> surgeons at the 8 treatment centres that provided materi- als and data
> for this study. The study was financed by a Joint Funding Grant within
> the German Cancer Consortium (DKTK), which is funded as one of the
> National German Health Centres by the Federal German Ministry of
> Education and Research.
>
> Appendix A. Supplementary data
>
> Supplementary data associated with this article can be found, in the
> online version, at
> [http://dx.doi.org/10.1016/j.radonc.201](http://dx.doi.org/10.1016/j.radonc.2014.11.011)
> [4.11.011](http://dx.doi.org/10.1016/j.radonc.2014.11.011).
>
> References

1.  [Ferlay J, Shin HR, Bray F, Forman D, Mathers C, Parkin DM.
    Estimates
    of](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0005)
    [worldwide burden of cancer in 2008: GLOBOCAN 2008. Int J
    Cancer](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0005)
    [2010;127:2893--917](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0005).

2.  [Leemans CR, Braakhuis BJ, Brakenhoff RH. The molecular biology of
    head and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0010)
    [neck cancer. Nat Rev Cancer
    2011;11:9--22](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0010).

3.  [[[[]{#_bookmark98 .anchor}]{#_bookmark96 .anchor}]{#_bookmark95
    .anchor}]{#_bookmark94 .anchor}[[]{#_bookmark99 .anchor}Chaturvedi
    AK, Anderson WF, Lortet-Tieulent J, et al. Worldwide trends
    in](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0015)
    [incidence rates for oral cavity and oropharyngeal cancers. J Clin
    Oncol](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0015)
    []{#_bookmark97
    .anchor}[2013;31:4550--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0015).

4.  [Sturgis EM, Cinciripini PM. Trends in head and neck cancer
    incidence
    in](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0020)
    [relation to smoking prevalence: an emerging epidemic of
    human](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0020)
    [[]{#_bookmark101 .anchor}]{#_bookmark100
    .anchor}[papillomavirus-associated cancers? Cancer
    2007;110:1429--35](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0020).

5.  [Habbous S, Chu KP, Qiu X, et al. The changing incidence of
    human](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0025)
    [papillomavirus-associated oropharyngeal cancer using multiple
    imputation](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0025)
    [from 2000 to 2010 at a Comprehensive Cancer Centre. Cancer
    Epidemiol](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0025)
    []{#_bookmark102
    .anchor}[2013;37:820--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0025).

6.  [Bernier J, Domenge C, Ozsahin M, et al. Postoperative irradiation
    with or](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0030)
    [without concomitant chemotherapy for locally advanced head and
    neck](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0030)
    []{#_bookmark103 .anchor}[cancer. N Engl J Med
    2004;350:1945--52](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0030).

7.  [Cooper JS, Pajak TF, Forastiere AA, et al. Postoperative concurrent
    radiotherapy](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0035)
    [and chemotherapy for high-risk squamous-cell carcinoma of the head
    and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0035) [neck.
    N Engl J Med
    2004;350:1937--44](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0035).

8.  [Fietkau R, Lautenschläger C, Sauer R, et al. Postoperative
    concurrent](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0040)
    []{#_bookmark104 .anchor}[radiochemotherapy versus radiotherapy in
    high-risk SCCA of the head
    and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0040) [neck:
    results of the German phase III trial ARO 96--3. J Clin
    Oncol](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0040)
    [2006;24:5507](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0040).

9.  [[]{#_bookmark107 .anchor}Bernier J, Cooper JS, Pajak TF, et al.
    Defining risk levels in locally advanced
    head](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0045)
    []{#_bookmark105 .anchor}[and neck cancers: a comparative analysis
    of concurrent
    postoperative](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0045)
    [radiation plus chemotherapy trials of the EORTC (#22931) and
    RTOG](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0045)
    []{#_bookmark106 .anchor}[(#9501). Head Neck
    2005;27:843--50](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0045).

10. [Fakhry C, Gillison ML. Clinical implications of human
    papillomavirus in
    head](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0050) [and
    neck cancers. J Clin Oncol
    2006;24:2606--11](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0050).

11. [Agrawal N, Frederick MJ, Pickering CR, et al. Exome sequencing of
    head and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0055)
    [neck squamous cell carcinoma reveals inactivating mutations in
    NOTCH1.](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0055)
    [Science
    2011;333:1154--7](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0055).

12. [Kawakami H, Okamoto I, Terao K, et al. Human papillomavirus DNA and
    p16](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0060)
    [expression in Japanese patients with oropharyngeal squamous cell
    carcinoma.](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0060)
    [Cancer Med
    2013;2:933--41](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0060).

13. [Salazar CR, Smith RV, Garg MK, et al. Human
    papillomavirus-associated
    head](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0065) [and
    neck squamous cell carcinoma survival: a comparison by tumor site
    and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0065)
    [initial treatment. Head Neck Pathol
    2014;8:77--87](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0065).

14. [Fakhry C, Westra WH, Li S, et al. Improved survival of patients
    with human](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0070)
    [papillomavirus-positive head and neck squamous cell carcinoma in
    a](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0070)
    [prospective clinical trial. J Natl Cancer Inst
    2008;100:261--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0070).

15. [Ankola AA, Smith RV, Burk RD, Prystowsky MB, Sarta C, Schlecht
    NF.](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0075)
    [Comorbidity, human papillomavirus infection and head and neck
    cancer](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0075)
    [survival in an ethnically diverse population. Oral Oncol
    2013;49:911--7](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0075).

16. [Ang KK, Harris J, Wheeler R, et al. Human papillomavirus and
    survival of](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0080)
    [patients with oropharyngeal cancer. N Engl J Med
    2010;363:24--35](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0080).

17. [Habbous S, Harland LT, La Delfa A, et al. Comorbidity and prognosis
    in head](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0085)
    [and neck cancers: differences by subsite, stage, and human
    papillomavirus](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0085)
    [status. Head Neck
    2014;36:802--10](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0085).

18. [Lassen P, Eriksen JG, Hamilton-Dutoit S, Tramm T, Alsner J,
    Overgaard J.
    Effect](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0090) [of
    HPV-associated p16INK4A expression on response to radiotherapy
    and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0090)
    [survival in squamous cell carcinoma of the head and neck. J Clin
    Oncol](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0090)
    [2009;27:1992--8](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0090).

19. [Lassen P, Overgaard J, Eriksen JG. Expression of EGFR and
    HPV-associated
    p16](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0095) [in
    oropharyngeal carcinoma: correlation and influence on prognosis
    after](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0095)
    [radiotherapy in the randomized DAHANCA 5 and 7 trials. Radiother
    Oncol](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0095)
    [2013;108:489--94](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0095).

20. [Rischin D, Young RJ, Fisher R, et al. Prognostic significance of
    p16INK4A
    and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0100) [human
    papillomavirus in patients with oropharyngeal cancer treated on
    TROG](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0100)

> [02.02 phase III trial. J Clin Oncol
> 2010;28:4142--8](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0100).

21. [Marur S, D'Souza G, Westra WH, Forastiere AA. HPV-associated head
    and neck](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0105)
    [cancer: a virus-related cancer epidemic. Lancet Oncol
    2010;11:781--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0105).

22. [Mineta H, Borg A, Dictor M, Wahlberg P, Akervall J, Wennerberg J.
    P53](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0110)
    [mutation, but not p53 overexpression, correlates with survival in
    head and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0110)
    [neck squamous cell carcinoma. Br J Cancer
    1998;78:1084--90](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0110).

23. [Bouvard V, Baan R, Straif K, et al. A review of human carcinogens
    -- part B:](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0115)
    [biological agents. Lancet Oncol
    2009;10:321--2](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0115).

24. [Snietura M, Piglowski W, Jaworska M, et al. Impact of HPV infection
    on the](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0120)
    [clinical outcome of p-CAIR trial in head and neck cancer. Eur
    Arch](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0120)
    [Otorhinolaryngol
    2011;268:721--6](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0120).

25. [Rios-Velazquez E, Hoebers F, Aerts HJWL, et al. Externally
    validated](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0125)
    [HPV-based prognostic nomogram for oropharyngeal carcinoma
    patients](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0125)
    [yields more accurate predictions than TNM staging. Radiother Oncol
    2014;](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0125)
    [113:317--23](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0125).

26. [Ivanova A, Qaqish BF, Schell MJ. Continuous toxicity monitoring in
    phase II](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0130)
    [trials in oncology. Biometrics
    2005;61:540--5](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0130).

27. [Garden AS, Kies MS, Morrison WH, et al. Outcomes and patterns of
    care of](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0135)
    [patients with locally advanced oropharyngeal carcinoma treated in
    the early](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0135)
    [21st century. Radiat Oncol
    2013;8:21](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0135).

28. [Rietbergen MM, Martens-deKemp SR, Bloemena E, et al. Cancer stem
    cell](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0140)
    [enrichment marker CD98: a prognostic factor for survival in
    patients
    with](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0140) [human
    papillomavirus-positive oropharyngeal cancer. Eur J
    Cancer](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0140)
    [2014;50:765--73](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0140).

29. [Rieckmann T, Tribius S, Grob TJ, et al. HNSCC cell lines positive
    for HPV and](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0145)
    [p16 possess higher cellular radiosensitivity due to an impaired DSB
    repair](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0145)
    [capacity. Radiother Oncol
    2013;107:242--6](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0145).

30. [Gupta AK, Lee JH, Wilke WW, et al. Radiation response in two
    HPV-infected](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0150)
    [head-and-neck cancer cell lines in comparison to a non-HPV-infected
    cell line](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0150)
    [and relationship to signaling through AKT. Int J Radiat Oncol Biol
    Phys](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0150)
    [2009;74:928--33](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0150).

31. [Kimple RJ, Smith MA, Blitzer GC, et al. Enhanced radiation
    sensitivity in
    HPV-](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0155)
    [positive head and neck cancer. Cancer Res
    2013;73:4791--800](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0155).

32. [Dok R, Kalev P, Van Limbergen EJ, et al. P16INK4a impairs
    homologous](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0160)
    [recombination-mediated DNA repair in human papillomavirus-positive
    head](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0160) [and
    neck tumors. Cancer Res
    2014;74:1739--51](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0160).

33. [Snow AN, Laudadio J. Human papillomavirus detection in head and
    neck](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0165)
    [squamous cell carcinomas. Adv Anat Pathol
    2010;17:394--403](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0165).

34. [Westra WH. Detection of human papillomavirus (HPV) in clinical
    samples:](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0170)
    [Evolving methods and strategies for the accurate determination of
    HPV status](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0170)
    [of head and neck carcinomas. Oral Oncol
    2014;50:771--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0170).

35. [Smeets SJ, Hesselink AT, Speel EJ, et al. A novel algorithm for
    reliable
    detection](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0175)
    [of human papillomavirus in paraffin embedded head and neck
    cancer](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0175)
    [specimen. Int J Cancer
    2007;121:2465--72](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0175).

36. [Mineta H, Ogino T, Amano HM, et al. Human papilloma virus (HPV)
    type 16](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0180)
    [and 18 detected in head and neck squamous cell carcinoma.
    Anticancer
    Res](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0180)
    [1998;18:4765--8](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0180).

37. [Bussu F, Sali M, Gallus R, et al. Human papillomavirus (HPV)
    infection
    in](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0185)
    [squamous cell carcinomas arising from the oropharynx: detection of
    HPV DNA](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0185)
    [and p16 immunohistochemistry as diagnostic and prognostic
    indicators --
    A](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0185) [pilot
    study. Int J Radiat Oncol Biol Phys
    2014;89:1115--20](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0185).

38. [Nylander K, Dabelsteen E, Hall PA. The p53 molecule and its
    prognostic role
    in](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0190)
    [squamous cell carcinomas of the head and neck. J Oral Pathol
    Med](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0190)
    [2000;29:413--25](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0190).

39. [Field JK, Spandidos DA, Malliri A, Gosney JR, Yiagnisis M, Stell
    PM. Elevated
    P53](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0195)
    [expression correlates with a history of heavy smoking in squamous
    cell](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0195)
    [carcinoma of the head and neck. Br J Cancer
    1991;64:573--7](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0195).

40. [Field JK, Spandidos DA, Stell PM. Overexpression of p53 gene in
    head-and-neck](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0200)
    [cancer, linked with heavy smoking and drinking. Lancet
    1992;339:502--3](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0200).

41. [Maestro R, Dolcetti R, Gasparotto D, et al. High frequency of p53
    gene](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0205)
    [alterations associated with protein overexpression in human
    squamous
    cell](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0205)
    [carcinoma of the larynx. Oncogene
    1992;7:1159--66](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0205).

42. [Scheffner M, Werness BA, Huibregtse JM, Levine AJ, Howley PM. The
    E6](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0210)
    [oncoprotein encoded by human papillomavirus types 16 and 18
    promotes
    the](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0210)
    [degradation of p53. Cell
    1990;63:1129--36](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0210).

43. [Werness BA, Levine AJ, Howley PM. Association of human
    papillomavirus](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0215)
    [types 16 and 18 E6 proteins with p53. Science
    1990;248:76--9](http://refhub.elsevier.com/S0167-8140(14)00489-7/h0215).

> **Letter to the Editor**

###### Page 1 of 3 {#page-1-of-3 .unnumbered}

> **HPV and beyond---looking out for biomarkers for distinguishing the
> good prognosis from the bad prognosis group in locally advanced and
> clinically high risk HNSCC**
>
> **Fabian Lohaus^1,2,3^, Annett Linge^1,2,3^, Michael
> Baumann^1,2,3,4^**
>
> ^1^German Cancer Research Center (DKFZ), Heidelberg and German Cancer
> Consortium (DKTK) partner site Dresden, Dresden, Germany;
> ^2^Department of Radiation Oncology, ^3^OncoRay -- National Center for
> Radiation Research in Oncology, Faculty of Medicine and University
> Hospital Carl Gustav Carus, Technische Universität, Dresden, Germany;
> ^4^Institute of Radiooncology, Helmholtz-Zentrum Dresden --
> Rossendorf, Dresden, Germany
>
> *Correspondence to:* Prof. Dr. med. Michael Baumann. DKTK partner site
> Dresden, Department of Radiation Oncology and OncoRay - National
> Center for Radiation Research in Oncology, Medical Faculty and
> University Hospital Carl Gustav Carus, Technische Universität Dresden,
> Fetscherstraße 74, 01307 Dresden, Germany. Email:
> [Michael.Baumann@uniklinikum-dresden.de.](mailto:Michael.Baumann@uniklinikum-dresden.de)
>
> Submitted Sep 16, 2015. Accepted for publication Sep 16, 2015. doi:
> 10.3978/j.issn.2305-5839.2015.09.35
>
> **View this article at:**
> <http://dx.doi.org/10.3978/j.issn.2305-5839.2015.09.35>
>
> In their recent editorial Kimple and Harari (1) reviewed the current
> knowledge of the importance of HPV status of head and neck squamous
> cell carcinoma (HNSCC) for outcome of radiotherapy. The editorial
> summarized, among others, a retrospective multicenter study by the
> German Cancer Consortium Radiation Oncology Group (DKTK-ROG) which
> demonstrated significant prognostic value of the HPV status in
> patients with locally advanced HNSCC on the outcome of postoperative
> radiochemotherapy (PORT-C) (2). The overall aim of the DKTK-ROG head
> and neck program is to identify and validate biomarkers including
> biological and clinical parameters as well as imaging data for patient
> stratification in terms of individualization of radiotherapy by
> multimodal treatment. The study design includes retrospective
> explorative analyses in patients who received PORT-C or primary
> radiochemotherapy (RCT) for biomarker identification. The most
> promising biomarkers will then be validated in a prospective
> validation cohort, which is currently recruiting at all eight DKTK-ROG
> partner sites. Based on these results, interventional studies are
> under preparation. Biomarker analyses are being performed at all eight
> DKTK partner sites and are addressing different topics such as HPV
> status, hypoxia, cancer stem cells, tumor infiltrating lymphocytes,
> tumor volume, targeted next generation sequencing, transcriptomics and
> methylome analyses.
>
> With a 2-year overall survival rate of about 83% in the retrospective
> PORT-C study, the patients treated at the
>
> DKTK-ROG sites seem to have a favorable prognosis compared to the two
> landmark studies of the RTOG (#9501) (63%) and EORTC (#22931) (75%)
> (3,4). This difference may be due to differences in staging (in the
> DKTK-ROG study staging was performed with contrast-enhanced MRI, CT or
> PET/CT) and the broad availability of intensity modulated radiotherapy
> (IMRT). However, also formal differences exist between these three
> studies, with the DKTK-ROG study evaluating contemporary retrospective
> data from a limited number of tertiary high volume centers, while the
> randomized RTOG and EORTC trials have prospectively accrued patients
> in a larger number of centers. This limits the validity of direct
> comparison of the survival data of these studies. It is also not
> straightforward to compare the PORT-C data reported by the DKTK- ROG
> with the outcome of studies testing primary RCT, as the patient
> cohorts submitted to these different treatment strategies may vary in
> important parameters, including stage and co-morbidity. Therefore it
> appears most rationale to compare the relative impact of biological
> stratification parameters on outcome separately in different trials.
>
> Impact of HPV in non-oropharyngeal cancer

##### In our DKTK-ROG study, the HPV status was analyzed in patients with locally advanced HNSCC of the oral cavity, oropharynx and hypopharynx. However, 35 patients with {#in-our-dktk-rog-study-the-hpv-status-was-analyzed-in-patients-with-locally-advanced-hnscc-of-the-oral-cavity-oropharynx-and-hypopharynx.-however-35-patients-with .unnumbered}

> © Annals of Translational Medicine. All rights reserved.
>
> [www.atmjournal.org](http://www.atmjournal.org/)
>
> *Ann Transl Med* 2015;3(17):255
>
> **Page 2 of 3 Lohaus et al. HPV and beyond---biomarkers for
> radiochemotherapy in head and neck cancer**

##### hypopharyngeal carcinoma were included in this study with five of them being HPV16 DNA positive. Because of this low number, we did not draw any conclusion of the impact of the HPV status in this tumor subsite. Kimple and Harari pointed out that in a study by Chung *et al*. (5), patients with p16-positive non-oropharyngeal cancer treated with different RCT schedules had a significantly better overall survival than patients with p16-negative tumors. Whether this applies also to patients in Germany and Europe needs to be addressed in larger studies, as today's HPV infection rate is lower in Germany than in North America. {#hypopharyngeal-carcinoma-were-included-in-this-study-with-five-of-them-being-hpv16-dna-positive.-because-of-this-low-number-we-did-not-draw-any-conclusion-of-the-impact-of-the-hpv-status-in-this-tumor-subsite.-kimple-and-harari-pointed-out-that-in-a-study-by-chung-et-al.-5-patients-with-p16-positive-non-oropharyngeal-cancer-treated-with-different-rct-schedules-had-a-significantly-better-overall-survival-than-patients-with-p16-negative-tumors.-whether-this-applies-also-to-patients-in-germany-and-europe-needs-to-be-addressed-in-larger-studies-as-todays-hpv-infection-rate-is-lower-in-germany-than-in-north-america. .unnumbered}

> Impact of HPV in primary RCT ***vs***. PORT-C

##### In our retrospective DKTK-ROG PORT-C cohort, a very high locoregional tumor control rate was achieved in HPV16 DNA positive (oropharyngeal) tumors. If confirmed in our ongoing multicenter validation trial, this observation is of substantial importance for the design of future clinical studies. Local control rates close to 100% should allow to testing the value of dose de-escalation in HPV positive tumors without compromising locoregional control. This is in contrast to primary RCT of macroscopic tumors (e.g., RTOG0129, RTOG 0522, DAHANCA 5, DAHANCA 6, {#in-our-retrospective-dktk-rog-port-c-cohort-a-very-high-locoregional-tumor-control-rate-was-achieved-in-hpv16-dna-positive-oropharyngeal-tumors.-if-confirmed-in-our-ongoing-multicenter-validation-trial-this-observation-is-of-substantial-importance-for-the-design-of-future-clinical-studies.-local-control-rates-close-to-100-should-allow-to-testing-the-value-of-dose-de-escalation-in-hpv-positive-tumors-without-compromising-locoregional-control.-this-is-in-contrast-to-primary-rct-of-macroscopic-tumors-e.g.-rtog0129-rtog-0522-dahanca-5-dahanca-6 .unnumbered}

> DAHANCA 7) where long term local control rates of HPV positive
> oropharyngeal cancers are in the range between 70% and 80%. Further
> clinical and biological parameters are needed in this situation to
> safely stratify patients to less or more aggressive treatments. This
> can for example be demonstrated in the primary RCT DKTK-ROG cohort,
> which shows that tumor volume, hypoxia and stem cell markers add
> significantly to the prognostic power of HPV (Lohaus, Linge, Löck,
> Baumann *et al*., to be submitted).
>
> Beyond HPV infection

##### To further stratify patients with HPV negative tumors undergoing PORT-C, further biomarkers are currently under investigation. Recently, Balermpas *et al*. showed in the retrospective PORT-C cohort of the DKTK-ROG, the prognostic value of tumor-infiltrating CD8 positive lymphocytes on overall survival and local progression- free survival independent of the HPV status (6). In terms of potential treatment modulation, the question remains to be resolved whether HPV positive HNSCC are more radiosensitive *per se* or if the improved response is due to the immune response. Furthermore, targeted next generation sequencing in the retrospective DKTK PORT-C cohort {#to-further-stratify-patients-with-hpv-negative-tumors-undergoing-port-c-further-biomarkers-are-currently-under-investigation.-recently-balermpas-et-al.-showed-in-the-retrospective-port-c-cohort-of-the-dktk-rog-the-prognostic-value-of-tumor-infiltrating-cd8-positive-lymphocytes-on-overall-survival-and-local-progression--free-survival-independent-of-the-hpv-status-6.-in-terms-of-potential-treatment-modulation-the-question-remains-to-be-resolved-whether-hpv-positive-hnscc-are-more-radiosensitive-per-se-or-if-the-improved-response-is-due-to-the-immune-response.-furthermore-targeted-next-generation-sequencing-in-the-retrospective-dktk-port-c-cohort .unnumbered}

> revealed that loss-of-function alterations in tumor suppressor genes
> are related to an increased risk of loco- regional recurrence, distant
> metastases and death, mainly in HPV negative tumors (7). Also hypoxia
> and expression of cancer stem cell markers, independently of HPV,
> correlate with outcome of PORT-C (Linge *et al*., submitted for
> publication). Thus clinical applicable biomarkers beyond HPV are
> emerging which, if validated in our ongoing prospective trial, may
> significantly enhance our arsenal for stratification of patients for
> clinical trials and individualized treatment approaches, offering new
> opportunities on the avenue towards personalized precision radiation
> oncology in head and neck cancer.
>
> Acknowledgements

##### None. {#none. .unnumbered}

> Footnote
>
> *Conflicts of Interest:* The authors have no conflicts of interest

##### to declare. {#to-declare. .unnumbered}

> References

1.  Kimple RJ, Harari PM. The prognostic value of HPV in head and neck
    cancer patients undergoing postoperative chemoradiotherapy. Ann
    Transl Med 2015;3:S14.

2.  Lohaus F, Linge A, Tinhofer I, et al. HPV16 DNA status is a strong
    prognosticator of loco-regional control after postoperative
    radiochemotherapy of locally advanced oropharyngeal carcinoma:
    results from a multicentre explorative study of the German Cancer
    Consortium Radiation Oncology Group (DKTK-ROG). Radiother Oncol
    2014;113:317-23.

3.  Bernier J, Domenge C, Ozsahin M, et al. Postoperative irradiation
    with or without concomitant chemotherapy for locally advanced head
    and neck cancer. N Engl J Med 2004;350:1945-52.

4.  Cooper JS, Pajak TF, Forastiere AA, et al. Postoperative concurrent
    radiotherapy and chemotherapy for high-risk squamous-cell carcinoma
    of the head and neck. N Engl J Med 2004;350:1937-44.

5.  Chung CH, Zhang Q, Kong CS, et al. p16 protein expression and human
    papillomavirus status as prognostic biomarkers of nonoropharyngeal
    head and neck squamous cell carcinoma. J Clin Oncol 2014;32:3930-8.

6.  Balermpas P, Rödel F, Rödel C, et al. CD8+ tumour-

> © Annals of Translational Medicine. All rights reserved.
>
> [www.atmjournal.org](http://www.atmjournal.org/)
>
> *Ann Transl Med* 2015;3(17):255
>
> **?33\"/- \'( @:\"3-/\".8\'3\"/ A\$48\<83\$B 2\'/ )B C\' &D E\<.\'5\$:
> \>F&G !\"#\$ ) \'( )**
>
> infiltrating lymphocytes in relation to HPV status and
>
> 1&-\$-1%& (C,1(A/ -\$ N%,-/\$,\' @-,4 4/%0 %\$0 \$/1\^ 1%\$1/+
>
> %),/+ N(\',(N/+%,-5/ 14/A(+%0-(,4/+%NO? \# AC&,-1/\$,+/
>
> \',C0O () ,4/ D/+A%\$ 1%\$1/+ 1(\$\'(+,-CA +%0-%,-(\$ (\$1(&(3O 3+(CN
> \<GH\*HXFPD\>2 V\$, m E%\$1/+ 6789:8;=?6\`UUX\`;2
>
> =2 \*-\$4()/+ VK ZC0%14 eK k-\$3/ #K /, %&2 .C,%,-(\$%& N%,,/+\$\'
>
> () LYeo %\$0 LYeX \'fC%A(C\' 1/&& 1%+1-\$(A%\' () ,4/ 4/%0
>
> %\$0 \$/1\^ \<\_EELR\> %\$0 ,4/-+ -\$,/+)/+/\$1/ @-,4 (C,1(A/
>
> %),/+ %0BC5%\$, 14/A(+%0-%,-(\$? \# AC&,-1/\$,/+ M-(A%+\^/+ \',C0O ()
> ,4/ D/+A%\$ E%\$1/+ E(\$\'(+,-CA F%0-%,-(\$ P\$1(&(3O D+(CN2 m E&-\$
> P\$1(& 6789:;;?%M\',+ \`77\`2
>
> ! #\$\$%&\' () \*+%\$\'&%,-(\$%& ./0-1-\$/2 #&& +-34,\' +/\'/+5/02
>
> @@@2%,AB(C+\$%&2(+3
>
> *!\"\" \$%&\"\'( )\*+* 6789:;\<8=\>?699

# Comparison of detection methods for HPV status as a prognostic marker for loco- regional control after radiochemotherapy in patients with HNSCC {#comparison-of-detection-methods-for-hpv-status-as-a-prognostic-marker-for-loco--regional-control-after-radiochemotherapy-in-patients-with-hnscc .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** A. Linge **Autor** U. Schotz **Autor** S. Lock **Autor** F.
> Lohaus
>
> **Autor** C. von Neubeck **Autor** V. Gudziol **Autor** A. Nowak
> **Autor** I. Tinhofer **Autor** V. Budach **Autor** A. Sak
>
> **Autor** M. Stuschke **Autor** P. Balermpas **Autor** C. Rodel
> **Autor** H. Bunea **Autor** A. L. Grosu **Autor** A. Abdollahi
> **Autor** J. Debus **Autor** U. Ganswindt **Autor** K. Lauber
> **Autor** S. Pigorsch **Autor** S. E. Combs **Autor** D. Monnich
> **Autor** D. Zips
>
> **Autor** G. B. Baretton **Autor** F. Buchholz **Autor** M. Krause
> **Autor** C. Belka **Autor** M. Baumann **Autor** R. O. G. Dktk
>
> **Zusammenfassung** OBJECTIVE: To compare six HPV detection methods in
> pre-treatment FFPE tumour samples from patients with locally advanced
> head and neck squamous cell carcinoma (HNSCC) who received
> postoperative (N=175) or primary (N=90) radiochemotherapy. MATERIALS
> AND METHODS: HPV analyses included detection of (i) HPV16 E6/E7 RNA,
> (ii) HPV16 DNA (PCR-based arrays, A-PCR),
>
> \(iii\) HPV DNA (GP5+/GP6+ qPCR, (GP-PCR)), (iv) p16
> (immunohistochemistry, p16 IHC), (v) combining p16 IHC and the A-PCR
> result and (vi) combining p16
>
> IHC and the GP-PCR result. Differences between HPV positive and
> negative subgroups were evaluated for the primary endpoint
> loco-regional control (LRC) using Cox regression. RESULTS: Correlation
> between the HPV detection methods was high (chi-squared test,
> p\<0.001). While p16 IHC analysis resulted in several false positive
> classifications, A-PCR, GP-PCR and the combination of p16 IHC and
> A-PCR or GP-PCR led to results comparable to RNA analysis. In both
> cohorts, Cox regression analyses revealed significantly prolonged LRC
> for patients with HPV positive tumours irrespective of the detection
> method. CONCLUSIONS: The most stringent classification was obtained by
> detection of HPV16 RNA, or combining p16 IHC with A-PCR or GP-PCR.
> This approach revealed the lowest rate of recurrence in patients with
> tumours classified as HPV positive and therefore appears most suited
> for patient stratification in HPV-based clinical studies.
>
> **Datum** 2018
>
> **URL** <https://www.ncbi.nlm.nih.gov/pubmed/29295747>
>
> **Extra** Type: Journal Article
>
> **Band** 127
>
> **Seiten** 27-35
>
> **Publikation** Radiother Oncol
>
> **DOI**
> [10.1016/j.radonc.2017.12.007](http://doi.org/10.1016/j.radonc.2017.12.007)

#### Ausgabe 1 {#ausgabe-1 .unnumbered}

> **ISSN** 1879-0887 (Electronic) 0167-8140 (Linking)

**Hinzugefügt am** 25.9.2023, 04:40:01

**Geändert am** 25.9.2023, 04:40:01

### Tags: {#tags .unnumbered}

> Adult Aged Carcinoma Local/drug
> therapy/pathology/radiotherapy/\*virology Oncogene Proteins Squamous
> Cell/drug therapy/pathology/radiotherapy/\*virology Chemoradiotherapy
> Female Head and Neck Neoplasms/drug
> therapy/pathology/radiotherapy/\*virology Human papillomavirus
> 16/genetics
>
> /\*isolation & purification/metabolism Humans Immunohistochemistry
> Male Middle Aged Neoplasm Recurrence Viral/genetics Real-Time
> Polymerase Chain Reaction Repressor Proteins/genetics/metabolism
> Retrospective Studies Squamous Cell Carcinoma of Head and Neck
> \*dktk-rog \*hnscc \*hpv \*Loco- regional control \*Radiochemotherapy
> \*p16 Viral/genetics/metabolism Papillomavirus E7
> Proteins/genetics/metabolism Papillomavirus
> Infections/metabolism/\*virology Prognosis RNA

### Notizen: {#notizen .unnumbered}

> Linge, Annett Schotz, Ulrike Lock, Steffen Lohaus, Fabian von Neubeck,
> Clare Gudziol, Volker Nowak, Alexander Tinhofer, Inge Budach, Volker
> Sak, Ali Stuschke, Martin Balermpas, Panagiotis Rodel, Claus Bunea,
> Hatice Grosu, Anca-Ligia Abdollahi, Amir Debus, Jurgen Ganswindt, Ute
> Lauber, Kirsten Pigorsch, Steffi Combs, Stephanie E Monnich, David
> Zips, Daniel Baretton, Gustavo B Buchholz, Frank Krause, Mechthild
> Belka, Claus Baumann, Michael eng Comparative Study Multicenter Study
> Research Support, Non-U.S. Gov\'t Ireland Radiother Oncol. 2018
> Apr;127(1):27-35. doi: 10.1016/j.radonc.2017.12.007. Epub 2017 Dec 30.

# HPV and beyond-looking out for biomarkers for distinguishing the good prognosis from the bad prognosis group in locally advanced and clinically high risk HNSCC {#hpv-and-beyond-looking-out-for-biomarkers-for-distinguishing-the-good-prognosis-from-the-bad-prognosis-group-in-locally-advanced-and-clinically-high-risk-hnscc .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** F. Lohaus **Autor** A. Linge **Autor** M. Baumann **Datum**
> 2015
>
> **URL**
> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4620098/pdf/atm-03-17-255.pdf>
>
> **Extra** Type: Journal Article

#### Band 3 {#band-3 .unnumbered}

> **Seiten** 255
>
> **Publikation** Ann Transl Med
>
> **DOI**
> [10.3978/j.issn.2305-5839.2015.09.35](http://doi.org/10.3978/j.issn.2305-5839.2015.09.35)

#### Ausgabe 17 {#ausgabe-17 .unnumbered}

> **ISSN** 2305-5839 (Print) 2305-5839 (Linking)

**Hinzugefügt am** 25.9.2023, 04:39:59

**Geändert am** 25.9.2023, 04:39:59

### Notizen: {#notizen-1 .unnumbered}

> Lohaus, Fabian Linge, Annett Baumann, Michael eng China Ann Transl
> Med. 2015 Oct;3(17):255. doi: 10.3978/j.issn.2305-5839.2015.09.35.

### Anhänge {#anhänge .unnumbered}

-   Volltext

# HPV status, cancer stem cell marker expression, hypoxia gene signatures and tumour volume identify good prognosis subgroups in patients with HNSCC after primary radiochemotherapy: A multicentre retrospective study of the German Cancer Consortium Radiation Oncology Group (DKTK-ROG) {#hpv-status-cancer-stem-cell-marker-expression-hypoxia-gene-signatures-and-tumour-volume-identify-good-prognosis-subgroups-in-patients-with-hnscc-after-primary-radiochemotherapy-a-multicentre-retrospective-study-of-the-german-cancer-consortium-radiation-oncology-group-dktk-rog .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** A. Linge **Autor** F. Lohaus **Autor** S. Lock **Autor** A.
> Nowak **Autor** V. Gudziol **Autor** C. Valentini
>
> **Autor** C. von Neubeck
>
> **Autor** M. Jutz **Autor** I. Tinhofer **Autor** V. Budach **Autor**
> A. Sak **Autor** M. Stuschke **Autor** P. Balermpas **Autor** C. Rodel
> **Autor** A. L. Grosu **Autor** A. Abdollahi **Autor** J. Debus
>
> **Autor** U. Ganswindt **Autor** C. Belka **Autor** S. Pigorsch
> **Autor** S. E. Combs **Autor** D. Monnich **Autor** D. Zips **Autor**
> F. Buchholz **Autor** D. E. Aust
>
> **Autor** G. B. Baretton **Autor** H. D. Thames **Autor** A. Dubrovska
> **Autor** J. Alsner **Autor** J. Overgaard **Autor** M. Krause
> **Autor** M. Baumann **Autor** R. O. G. Dktk
>
> **Zusammenfassung** OBJECTIVE: To investigate the impact of the tumour
> volume, HPV status, cancer stem cell (CSC) marker expression and
> hypoxia gene signatures, as potential markers of radiobiological
> mechanisms of radioresistance, in a contemporary cohort of patients
> with locally advanced head and neck squamous cell carcinoma (HNSCC),
> who received primary radiochemotherapy (RCTx). MATERIALS AND METHODS:
> For 158 patients with locally advanced HNSCC of the oral cavity,
> oropharynx or hypopharynx who were treated at six DKTK partner sites,
> the impact of tumour volume, HPV DNA, p16 overexpression, p53
> expression, CSC marker expression and hypoxia-associated gene
> signatures on outcome of primary RCTx was retrospectively analyzed.
> The primary endpoint of this study was loco-regional control (LRC).
> RESULTS: Univariate Cox regression revealed a significant impact of
> tumour volume, p16 overexpression, and SLC3A2 and CD44 protein
> expression on LRC. The tumour hypoxia classification showed a
> significant impact only for small tumours. In multivariate analyses an
> independent correlation of tumour volume, SLC3A2 expression, and the
> 15-gene hypoxia signature with LRC was identified (CD44 protein n/a
> because of no event in the CD44-negative group).
>
> Logistic modelling showed that inclusion of CD44 protein expression
> and p16 overexpression significantly improved the performance to
> predict LRC at 2years compared to the model with tumour volume alone.
> CONCLUSIONS: Tumour volume, HPV status, CSC marker expression and
> hypoxia gene signatures are potential prognostic biomarkers for
> patients with locally advanced HNSCC, who were treated by primary
> RCTx. The study also supports that the individual tumour volumes
> should generally be included in biomarker studies and that panels of
> biomarkers are superior to individual parameters.
>
> **Datum** 2016
>
> **URL** <https://www.ncbi.nlm.nih.gov/pubmed/27913065>
>
> **Extra** Type: Journal Article
>
> **Band** 121
>
> **Seiten** 364-373
>
> **Publikation** Radiother Oncol
>
> **DOI**
> [10.1016/j.radonc.2016.11.008](http://doi.org/10.1016/j.radonc.2016.11.008)

#### Ausgabe 3 {#ausgabe-3 .unnumbered}

> **ISSN** 1879-0887 (Electronic) 0167-8140 (Linking)

**Hinzugefügt am** 25.9.2023, 04:40:02

**Geändert am** 25.9.2023, 04:40:02

### Tags: {#tags-1 .unnumbered}

> Tumor/metabolism Carcinoma Adult Aged Biomarkers Squamous
> Cell/genetics/pathology/\*therapy
>
> /virology Cell Hypoxia/genetics Chemoradiotherapy Female Gene
> Expression Profiling/methods Head and Neck
> Neoplasms/genetics/pathology/\*therapy/virology Humans Hyaluronan
> Receptors/metabolism Male Middle Aged Neoplastic Stem
> Cells/\*metabolism Papillomaviridae/\*isolation & purification
> Prognosis Radiation Tolerance/genetics Retrospective Studies Squamous
> Cell Carcinoma of Head and Neck Tumor Burden \*Biomarkers for
> radiotherapy \*Cancer stem cells \*hnscc \*hpv \*Hypoxia \*Primary
> radiochemotherapy

### Notizen: {#notizen-2 .unnumbered}

> Linge, Annett Lohaus, Fabian Lock, Steffen Nowak, Alexander Gudziol,
> Volker Valentini, Chiara von Neubeck, Clare Jutz, Martin Tinhofer,
> Inge Budach, Volker Sak, Ali Stuschke, Martin Balermpas, Panagiotis
> Rodel, Claus Grosu, Anca-Ligia Abdollahi, Amir Debus, Jurgen
> Ganswindt, Ute Belka, Claus Pigorsch, Steffi Combs, Stephanie E
> Monnich, David Zips, Daniel Buchholz, Frank Aust, Daniela E Baretton,
> Gustavo B Thames, Howard D Dubrovska, Anna Alsner, Jan Overgaard, Jens
> Krause, Mechthild Baumann, Michael eng Multicenter Study Ireland
> Radiother Oncol. 2016 Dec;121(3):364-373. doi:
> 10.1016/j.radonc.2016.11.008. Epub 2016 Nov 29.

# HPV16 DNA status is a strong prognosticator of loco-regional control after postoperative radiochemotherapy of locally advanced oropharyngeal carcinoma: results from a multicentre explorative study of the German Cancer Consortium Radiation Oncology Group (DKTK-ROG) {#hpv16-dna-status-is-a-strong-prognosticator-of-loco-regional-control-after-postoperative-radiochemotherapy-of-locally-advanced-oropharyngeal-carcinoma-results-from-a-multicentre-explorative-study-of-the-german-cancer-consortium-radiation-oncology-group-dktk-rog-1 .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** F. Lohaus **Autor** A. Linge **Autor** I. Tinhofer **Autor**
> V. Budach **Autor** E. Gkika **Autor** M. Stuschke **Autor** P.
> Balermpas **Autor** C. Rodel **Autor** M. Avlar **Autor** A. L. Grosu
> **Autor** A. Abdollahi **Autor** J. Debus **Autor** C. Bayer **Autor**
> C. Belka **Autor** S. Pigorsch
>
> **Autor** S. E. Combs **Autor** D. Monnich **Autor** D. Zips
>
> **Autor** C. von Neubeck **Autor** G. B. Baretton **Autor** S. Lock
>
> **Autor** H. D. Thames **Autor** M. Krause **Autor** M. Baumann
> **Autor** R. O. G. Dktk
>
> **Zusammenfassung** OBJECTIVE: To investigate the impact of HPV status
> in patients with locally advanced head and neck squamous cell
> carcinoma (HNSCC), who received surgery and cisplatin-based
> postoperative radiochemotherapy. MATERIALS AND METHODS: For 221
> patients with locally advanced squamous cell carcinoma of the
> hypopharynx, oropharynx or oral cavity treated at the 8 partner sites
> of the German Cancer Consortium, the impact of HPV DNA, p16
> overexpression and p53 expression on outcome were retrospectively
> analysed. The primary endpoint was loco-regional tumour control;
> secondary endpoints were distant metastases and overall survival.
> RESULTS: In the total patient population, univariate analyses revealed
> a significant impact of HPV16 DNA positivity, p16 overexpression, p53
> positivity and tumour site on loco-regional tumour control.
> Multivariate analysis stratified for tumour site showed that positive
> HPV 16 DNA status correlated with loco-regional tumour control in
> patients with oropharyngeal carcinoma (p=0.02) but not in the oral
> cavity carcinoma group. Multivariate evaluation of the secondary
> endpoints in the total population revealed a significant association
> of HPV16 DNA positivity with overall survival (p\<0.01) but not with
> distant metastases.
>
> CONCLUSIONS: HPV16 DNA status appears to be a strong prognosticator of
> loco- regional tumour control after postoperative cisplatin-based
> radiochemotherapy of locally advanced oropharyngeal carcinoma and is
> now being explored in a prospective validation trial.
>
> **Datum** 2014
>
> **URL**
> <https://www.thegreenjournal.com/article/S0167-8140(14)00489-7/pdf>
>
> **Extra** Type: Journal Article
>
> **Band** 113
>
> **Seiten** 317-23
>
> **Publikation** Radiother Oncol
>
> **DOI**
> [10.1016/j.radonc.2014.11.011](http://doi.org/10.1016/j.radonc.2014.11.011)

#### Ausgabe 3 {#ausgabe-3-1 .unnumbered}

> **ISSN** 1879-0887 (Electronic) 0167-8140 (Linking)

**Hinzugefügt am** 25.9.2023, 04:40:02

**Geändert am** 25.9.2023, 04:40:02

### Tags: {#tags-2 .unnumbered}

> Carcinoma Squamous Cell/\*genetics/\*therapy/virology
> Chemoradiotherapy/methods Cyclin-Dependent Kinase Inhibitor
> p16/\*genetics DNA/genetics Female Head and Neck
> Neoplasms/\*genetics/\*therapy
>
> /virology Humans Male Oropharyngeal
> Neoplasms/\*genetics/\*therapy/virology Papillomaviridae/genetics
> Postoperative Period Prognosis Prospective Studies Retrospective
> Studies Squamous Cell Carcinoma of Head and Neck Survival Analysis
> Treatment Outcome Dktk-rog Hnscc Hpv Postoperative radiochemotherapy
> p16 p53

### Notizen: {#notizen-3 .unnumbered}

> Lohaus, Fabian Linge, Annett Tinhofer, Inge Budach, Volker Gkika,
> Eleni Stuschke, Martin Balermpas, Panagiotis Rodel, Claus Avlar,
> Melanie Grosu, Anca-Ligia Abdollahi, Amir Debus, Jurgen Bayer,
> Christine Belka, Claus Pigorsch, Steffi Combs, Stephanie E Monnich,
> David Zips, Daniel von Neubeck, Clare Baretton, Gustavo B Lock,
> Steffen Thames, Howard D Krause, Mechthild Baumann, Michael eng
> Multicenter Study Research Support, Non-U.S. Gov\'t Ireland Radiother
> Oncol. 2014 Dec;113(3):317-23. doi: 10.1016/j.radonc.2014.11.011. Epub
> 2014 Dec 2.

### Anhänge {#anhänge-1 .unnumbered}

-   Volltext

# Independent validation of the prognostic value of cancer stem cell marker expression and hypoxia-induced gene expression for patients with locally advanced HNSCC after postoperative radiotherapy {#independent-validation-of-the-prognostic-value-of-cancer-stem-cell-marker-expression-and-hypoxia-induced-gene-expression-for-patients-with-locally-advanced-hnscc-after-postoperative-radiotherapy-1 .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** A. Linge **Autor** S. Lock **Autor** C. Krenn **Autor** S.
> Appold **Autor** F. Lohaus **Autor** A. Nowak **Autor** V. Gudziol
> **Autor** G. B. Baretton **Autor** F. Buchholz **Autor** M. Baumann
> **Autor** M. Krause
>
> **Zusammenfassung** Objective: To validate the impact of HPV status,
> cancer stem cell (CSC) marker expression and tumour hypoxia status in
> patients with locally advanced head and neck squamous cell carcinoma
> (HNSCC), who received postoperative radiotherapy. The results of the
> exploration cohort have previously been reported by the German Cancer
> Consortium Radiation Oncology Group (DKTK-ROG; Lohaus et al., 2014;
>
> Linge et al., 2016). Materials and methods: For 152 patients with
> locally advanced HNSCC the impact of HPV16 DNA status, CSC marker
> expression and hypoxia- associated gene signatures on outcome of
> postoperative radiotherapy were retrospectively analysed. Out of them,
> 40 patients received postoperative radiochemotherapy. Cox models
> presented in a previous study were validated using the concordance
> index as a performance measure. The primary endpoint of this study was
> loco-regional control. Results were compared to those previously
> reported by DKTK-ROG. Results: Loco-regional control, freedom from
> distant metastases and overall survival were inferior to the
> previously reported cohort. Despite of this, the prognostic value of
> the combination of HPV infection status, CSC marker expression
> (SLC3A2) and tumour hypoxia status could be validated in univariate
> analyses using an independent validation cohort. For multivariate
> models, the concordance index was between 0.58 and 0.69 in validation,
> indicating a good prognostic performance of the models. The inclusion
> of CD44 and the 15-gene hypoxia signature moderately improved the
> performance compared to a baseline model without CSC markers or
> hypoxia classifiers. Conclusions: The HPV status, CSC marker
> expression of CD44 and SLC3A2 as well as hypoxia status are potential
> prognostic biomarkers for patients with locally advanced HNSCC treated
> by postoperative radiotherapy.
>
> **Datum** 2016
>
> **URL**
> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5893482/pdf/main.pdf>
>
> **Extra** Type: Journal Article

#### Band 1 {#band-1 .unnumbered}

> **Seiten** 19-26
>
> **Publikation** Clin Transl Radiat Oncol
>
> **DOI**
> [10.1016/j.ctro.2016.10.002](http://doi.org/10.1016/j.ctro.2016.10.002)
>
> **ISSN** 2405-6308 (Electronic) 2405-6308 (Linking)

**Hinzugefügt am** 25.9.2023, 04:39:58

**Geändert am** 25.9.2023, 04:39:58

### Tags: {#tags-3 .unnumbered}

> Biomarker Cancer stem cells Hnscc Hpv Hypoxia Postoperative
> radiochemotherapy Validation

### Notizen: {#notizen-4 .unnumbered}

> Linge, Annett Lock, Steffen Krenn, Constanze Appold, Steffen Lohaus,
> Fabian Nowak, Alexander Gudziol, Volker Baretton, Gustavo B Buchholz,
> Frank Baumann, Michael Krause, Mechthild eng Ireland Clin Transl
> Radiat Oncol. 2016 Dec 22;1:19-26. doi: 10.1016/j.ctro.2016.10.002.
> eCollection 2016 Dec.

### Anhänge {#anhänge-2 .unnumbered}

-   Volltext

# Independent validation of tumour volume, cancer stem cell markers and hypoxia- associated gene expressions for HNSCC after primary radiochemotherapy {#independent-validation-of-tumour-volume-cancer-stem-cell-markers-and-hypoxia--associated-gene-expressions-for-hnscc-after-primary-radiochemotherapy .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** A. Linge **Autor** S. Schmidt **Autor** F. Lohaus **Autor**
> C. Krenn
>
> **Autor** A. Bandurska-Luque
>
> **Autor** I. Platzek **Autor** C. von Neubeck **Autor** S. Appold
> **Autor** A. Nowak **Autor** V. Gudziol **Autor** F. Buchholz
> **Autor** G. B. Baretton **Autor** M. Baumann **Autor** S. Lock
>
> **Autor** M. Krause
>
> **Zusammenfassung** Objective: To independently validate the impact of
> tumour volume, p16 status, cancer stem cell (CSC) marker expression
> and hypoxia-associated gene signatures as potential prognostic
> biomarkers for patients with locally advanced head and neck squamous
> cell carcinoma (HNSCC), who underwent primary radiotherapy or
> radiochemotherapy (RCTx). These markers have previously been reported
> in a study of the German Cancer Consortium Radiation Oncology Group
> (DKTK-ROG) (Linge et al., 2016). Materials and methods: In this
> retrospective monocentric study, 92 patients with locally advanced
> HNSCC were included. Univariable and multivariable logistic
> regressions and Cox models presented in the study of the DKTK-ROG were
> validated using the area under the curve (AUC) and the concordance
> index (ci), respectively. The primary endpoint of this study was loco-
> regional tumour control (LRC) after primary RCTx. Results: Although
> both cohorts significantly differed in the proportion of the tumour
> subsites, the parameters tumour volume, p16 status and N stage could
> be validated regarding LRC and overall survival (OS) using
> multivariable Cox regression (LRC ci: 0.59, OS ci: 0.63). These models
> were slightly improved by combination with the putative CSC marker
> CD44 (LRC ci: 0.61, OS ci: 0.69). The logistic regression model for
> 2-year LRC based on tumour volume, p16 status and CD44 protein was
> validated with an AUC of 0.64. The patient stratification based on
> hypoxia-associated gene signatures status was similar to the original
> study but without significant differences in LRC and OS. Conclusions:
> In this validation study, the inclusion of the putative CSC marker
> CD44 slightly improved the prognostic performance of the baseline
> parameters tumour volume, p16 status and N stage. No improvement was
> observed when including expressions of the hypoxia-associated gene
> signatures. Prospective validation on a larger cohort is warranted to
> assess the clinical relevance of these markers.
>
> **Datum** 2019
>
> **URL**
> <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6449705/pdf/main.pdf>
>
> **Extra** Type: Journal Article
>
> **Band** 16
>
> **Seiten** 40-47
>
> **Publikation** Clin Transl Radiat Oncol
>
> **DOI**
> [10.1016/j.ctro.2019.03.002](http://doi.org/10.1016/j.ctro.2019.03.002)
>
> **ISSN** 2405-6308 (Electronic) 2405-6308 (Linking)

**Hinzugefügt am** 25.9.2023, 04:40:01

**Geändert am** 25.9.2023, 04:40:01

### Tags: {#tags-4 .unnumbered}

> Biomarker Cancer stem cells Hnscc Hpv Hypoxia Primary
> radiochemotherapy

### Notizen: {#notizen-5 .unnumbered}

> Linge, Annett Schmidt, Stefan Lohaus, Fabian Krenn, Constanze
> Bandurska-Luque, Anna Platzek, Ivan von Neubeck, Clare Appold, Steffen
> Nowak, Alexander Gudziol, Volker Buchholz, Frank Baretton, Gustavo B
> Baumann, Michael Lock, Steffen Krause, Mechthild eng Ireland Clin
> Transl Radiat Oncol. 2019 Mar 18;16:40-47. doi:
> 10.1016/j.ctro.2019.03.002. eCollection 2019 May.

### Anhänge {#anhänge-3 .unnumbered}

-   Volltext

# Low Cancer Stem Cell Marker Expression and Low Hypoxia Identify Good Prognosis Subgroups in HPV(-) HNSCC after Postoperative Radiochemotherapy: A Multicenter Study of the DKTK-ROG {#low-cancer-stem-cell-marker-expression-and-low-hypoxia-identify-good-prognosis-subgroups-in-hpv--hnscc-after-postoperative-radiochemotherapy-a-multicenter-study-of-the-dktk-rog .unnumbered}

> **Eintragsart** Zeitschriftenartikel
>
> **Autor** A. Linge **Autor** S. Lock **Autor** V. Gudziol **Autor** A.
> Nowak **Autor** F. Lohaus
>
> **Autor** C. von Neubeck
>
> **Autor** M. Jutz **Autor** A. Abdollahi **Autor** J. Debus **Autor**
> I. Tinhofer
>
> **Autor** V. Budach **Autor** A. Sak **Autor** M. Stuschke **Autor**
> P. Balermpas **Autor** C. Rodel **Autor** M. Avlar **Autor** A. L.
> Grosu **Autor** C. Bayer **Autor** C. Belka **Autor** S. Pigorsch
> **Autor** S. E. Combs **Autor** S. Welz **Autor** D. Zips **Autor** F.
> Buchholz **Autor** D. E. Aust
>
> **Autor** G. B. Baretton **Autor** H. D. Thames **Autor** A. Dubrovska
> **Autor** J. Alsner **Autor** J. Overgaard **Autor** M. Baumann
> **Autor** M. Krause **Autor** R. O. G. Dktk
>
> **Zusammenfassung** PURPOSE: To investigate the impact of
> hypoxia-induced gene expression and cancer stem cell (CSC) marker
> expression on outcome of postoperative cisplatin- based
> radiochemotherapy (PORT-C) in patients with locally advanced head and
> neck squamous cell carcinoma (HNSCC). EXPERIMENTAL DESIGN: Expression
> of the CSC markers CD44, MET, and SLC3A2, and hypoxia gene signatures
> were analyzed in the resected primary tumors using RT-PCR and
> nanoString technology in a multicenter retrospective cohort of 195
> patients. CD44 protein expression was further analyzed in tissue
> microarrays. Primary endpoint was locoregional tumor control. RESULTS:
> Univariate analysis showed that hypoxia-induced gene expression was
> significantly associated with a high risk of locoregional recurrence
> using the 15-gene signature (P = 0.010) or the 26-gene signature (P =
> 0.002). In multivariate analyses, in patients with HPV16 DNA-negative
> but not with HPV16 DNA-positive tumors the effect of hypoxia-induced
> genes on locoregional control was apparent (15-gene signature: HR
> 4.54, P = 0.006; 26-gene signature: HR 10.27, P = 0.024). Furthermore,
> MET, SLC3A2, CD44, and CD44 protein showed an association with
> locoregional tumor control in multivariate analyses (MET: HR 3.71, P =
> 0.016; SLC3A2: HR 8.54, P = 0.037; CD44: HR 3.36, P = 0.054; CD44
>
> protein n/a because of no event in the CD44-negative group) in the
> HPV16 DNA- negative subgroup. CONCLUSIONS: We have shown for the first
> time that high hypoxia-induced gene expression and high CSC marker
> expression levels correlate with tumor recurrence after PORT-C in
> patients with HPV16 DNA-negative
>
> HNSCC. After validation in a currently ongoing prospective trial,
> these parameters may help to further stratify patients for
> individualized treatment de-escalation or intensification strategies.
> Clin Cancer Res; 22(11); 2639-49. (c)2016 AACR.
>
> **Datum** 2016
>
> **URL** <https://www.ncbi.nlm.nih.gov/pubmed/26755529>
>
> **Extra** Type: Journal Article
>
> **Band** 22
>
> **Seiten** 2639-49
>
> **Publikation** Clin Cancer Res
>
> **DOI**
> [10.1158/1078-0432.CCR-15-1990](http://doi.org/10.1158/1078-0432.CCR-15-1990)

#### Ausgabe 11 {#ausgabe-11 .unnumbered}

> **ISSN** 1557-3265 (Electronic) 1078-0432 (Linking)

**Hinzugefügt am** 25.9.2023, 04:40:02

**Geändert am** 25.9.2023, 04:40:02

### Tags: {#tags-5 .unnumbered}

> Tumor/metabolism Carcinoma Antineoplastic Agents/therapeutic use
> Biomarkers Heavy Chain/metabolism Human papillomavirus 16/genetics
> Humans Hyaluronan Receptors/metabolism Kaplan- Meier Estimate Mouth
> Neoplasms/\*metabolism/mortality/pathology/therapy Multivariate
> Analysis Neoplastic Stem Cells/\*metabolism Papillomavirus
> Infections/diagnosis Prognosis Prospective Studies Transcriptome
> Treatment Outcome Squamous
> Cell/\*metabolism/mortality/pathology/therapy Cell Hypoxia
> Chemoradiotherapy Cisplatin/therapeutic use Fusion Regulatory Protein
> 1

### Notizen: {#notizen-6 .unnumbered}

> Linge, Annett Lock, Steffen Gudziol, Volker Nowak, Alexander Lohaus,
> Fabian von Neubeck, Clare Jutz, Martin Abdollahi, Amir Debus, Jurgen
> Tinhofer, Inge Budach, Volker Sak, Ali Stuschke, Martin Balermpas,
> Panagiotis Rodel, Claus Avlar, Melanie Grosu, Anca-Ligia Bayer,
> Christine Belka, Claus Pigorsch, Steffi Combs, Stephanie E Welz,
> Stefan Zips, Daniel Buchholz, Frank Aust, Daniela E Baretton, Gustavo
> B Thames, Howard D Dubrovska, Anna Alsner, Jan Overgaard, Jens
> Baumann, Michael Krause, Mechthild eng Multicenter Study Research
> Support, Non-U.S. Gov\'t Clin Cancer Res. 2016 Jun 1;22(11):2639-49.
> doi: 10.1158/1078-0432.CCR-15-1990. Epub 2016 Jan 11.

### Anhänge {#anhänge-4 .unnumbered}

-   Volltext
